<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 7&nbsp; Singalling Pathways and Telomeres in Aging</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week8.html" rel="next">
<link href="../chapters/week6.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Alzheimer’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Parkinson’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Secondary Metabolism in Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Plants as Medicine</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#signalling-pathways" id="toc-signalling-pathways" class="nav-link active" data-scroll-target="#signalling-pathways"><span class="toc-section-number">7.1</span>  Signalling Pathways</a>
  <ul class="collapse">
  <li><a href="#mtorc1-and-mtorc2" id="toc-mtorc1-and-mtorc2" class="nav-link" data-scroll-target="#mtorc1-and-mtorc2"><span class="toc-section-number">7.1.1</span>  mTORC1 and mTORC2</a></li>
  <li><a href="#mtorc1-inhibits-autophagy" id="toc-mtorc1-inhibits-autophagy" class="nav-link" data-scroll-target="#mtorc1-inhibits-autophagy"><span class="toc-section-number">7.1.2</span>  mTORC1 Inhibits Autophagy</a></li>
  <li><a href="#activation-at-the-lysosomes" id="toc-activation-at-the-lysosomes" class="nav-link" data-scroll-target="#activation-at-the-lysosomes"><span class="toc-section-number">7.1.3</span>  Activation at the Lysosomes</a></li>
  <li><a href="#receptor-tyroskine-kinase-activation" id="toc-receptor-tyroskine-kinase-activation" class="nav-link" data-scroll-target="#receptor-tyroskine-kinase-activation"><span class="toc-section-number">7.1.4</span>  Receptor Tyroskine Kinase Activation</a></li>
  <li><a href="#pi3k-akt-mtor-signalling-pathway" id="toc-pi3k-akt-mtor-signalling-pathway" class="nav-link" data-scroll-target="#pi3k-akt-mtor-signalling-pathway"><span class="toc-section-number">7.1.5</span>  PI3K-AKT-mTOR Signalling Pathway</a></li>
  <li><a href="#insulin-and-glucose-uptake" id="toc-insulin-and-glucose-uptake" class="nav-link" data-scroll-target="#insulin-and-glucose-uptake"><span class="toc-section-number">7.1.6</span>  Insulin and Glucose Uptake</a></li>
  </ul></li>
  <li><a href="#telomeres-and-aging" id="toc-telomeres-and-aging" class="nav-link" data-scroll-target="#telomeres-and-aging"><span class="toc-section-number">7.2</span>  Telomeres and Aging</a>
  <ul class="collapse">
  <li><a href="#end-replication-by-telomerase" id="toc-end-replication-by-telomerase" class="nav-link" data-scroll-target="#end-replication-by-telomerase"><span class="toc-section-number">7.2.1</span>  End Replication by Telomerase</a></li>
  <li><a href="#dna-damage-response" id="toc-dna-damage-response" class="nav-link" data-scroll-target="#dna-damage-response"><span class="toc-section-number">7.2.2</span>  DNA Damage Response</a></li>
  <li><a href="#shelterin" id="toc-shelterin" class="nav-link" data-scroll-target="#shelterin"><span class="toc-section-number">7.2.3</span>  Shelterin</a></li>
  <li><a href="#cellular-senescence" id="toc-cellular-senescence" class="nav-link" data-scroll-target="#cellular-senescence"><span class="toc-section-number">7.2.4</span>  Cellular Senescence</a></li>
  <li><a href="#shelterin-and-non-homologous-end-joining" id="toc-shelterin-and-non-homologous-end-joining" class="nav-link" data-scroll-target="#shelterin-and-non-homologous-end-joining"><span class="toc-section-number">7.2.5</span>  Shelterin and Non-Homologous End Joining</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week7.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<section id="signalling-pathways" class="level2" data-number="7.1">
<h2 data-number="7.1" class="anchored" data-anchor-id="signalling-pathways"><span class="header-section-number">7.1</span> Signalling Pathways</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/mtorc1.png" class="img-fluid figure-img" width="404"></p>
<p></p><figcaption class="figure-caption">Mechanism of mTORC1</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>mTORC1 (mechanistic target of rapamycin complex 1)</strong> is a central regulator of cell growth, metabolism, and proliferation in response to various environmental cues, including nutrient availability, energy status, growth factors, and stress signals. Here’s how mTORC1 regulates cell growth:</p>
<ol type="1">
<li><p><strong>Nutrient Sensing:</strong> mTORC1 acts as a nutrient sensor, integrating signals from amino acids, glucose, lipids, and other metabolites to regulate cellular metabolism and growth. Amino acids, particularly leucine and arginine, are potent activators of mTORC1, promoting its translocation to the lysosomal membrane where it becomes activated.</p></li>
<li><p><strong>Activation by Growth Factors:</strong> Growth factors such as insulin, insulin-like growth factor 1 (IGF-1), and epidermal growth factor (EGF) activate mTORC1 signaling through the phosphoinositide 3-kinase (PI3K)/Akt pathway. Upon growth factor stimulation, Akt phosphorylates and inhibits tuberous sclerosis complex 2 (TSC2), relieving its inhibitory effect on mTORC1 and promoting its activation.</p></li>
<li><p><strong>Cellular Energy Status:</strong> mTORC1 activity is also influenced by cellular energy levels, particularly through the AMP-activated protein kinase (AMPK) pathway. AMPK, a sensor of cellular energy depletion, inhibits mTORC1 by phosphorylating TSC2 and raptor, leading to mTORC1 inhibition and energy conservation during metabolic stress.</p></li>
<li><p><strong>Regulation of Protein Synthesis:</strong> mTORC1 promotes protein synthesis by phosphorylating and activating key downstream effectors, including ribosomal protein S6 kinase 1 (S6K1) and eukaryotic initiation factor 4E-binding protein 1 (4E-BP1). Activation of S6K1 leads to the phosphorylation of ribosomal proteins and the initiation factors involved in mRNA translation, while phosphorylation of 4E-BP1 releases it from eukaryotic initiation factor 4E (eIF4E), allowing it to initiate cap-dependent translation.</p></li>
<li><p><strong>Lipid Biosynthesis and Lipogenesis:</strong> mTORC1 promotes lipid biosynthesis and lipogenesis by activating sterol regulatory element-binding proteins (SREBPs) and promoting the transcription of genes involved in fatty acid and cholesterol synthesis. This supports the increased demand for membrane biogenesis and lipid storage during cell growth and proliferation.</p></li>
<li><p><strong>Cell Cycle Progression:</strong> mTORC1 signaling promotes cell cycle progression by regulating the expression of cyclins, cyclin-dependent kinases (CDKs), and cell cycle inhibitors. Activation of mTORC1 promotes entry into the cell cycle and progression through the G1 phase, facilitating cell growth and proliferation.</p></li>
</ol>
<section id="mtorc1-and-mtorc2" class="level3" data-number="7.1.1">
<h3 data-number="7.1.1" class="anchored" data-anchor-id="mtorc1-and-mtorc2"><span class="header-section-number">7.1.1</span> mTORC1 and mTORC2</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/mtorc1and2.png" class="img-fluid figure-img" width="479"></p>
<p></p><figcaption class="figure-caption">Differences Between mTORC1 and mTORC2</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>mTOR (mechanistic target of rapamycin) exists in two distinct complexes: mTORC1 and mTORC2. While both complexes share the mTOR kinase as a core component, they have different compositions, substrates, and cellular functions. Here’s an overview of mTORC1 and mTORC2:</p>
<ol type="1">
<li><strong>mTORC1 (mechanistic target of rapamycin complex 1):</strong>
<ul>
<li>Composition: mTORC1 consists of mTOR, regulatory-associated protein of mTOR (RAPTOR), mammalian lethal with SEC13 protein 8 (mLST8), proline-rich Akt substrate 40 kDa (PRAS40), and DEP domain-containing mTOR-interacting protein (DEPTOR).</li>
<li>Activation: mTORC1 is primarily activated by growth factors, amino acids, energy status, and stress signals. Growth factors such as insulin and insulin-like growth factor 1 (IGF-1) activate mTORC1 through the PI3K/Akt pathway, while amino acids, particularly leucine, activate mTORC1 by promoting its translocation to the lysosomal membrane.</li>
<li>Substrates and Functions: mTORC1 regulates protein synthesis, lipid metabolism, autophagy, and cell growth by phosphorylating downstream effectors such as ribosomal protein S6 kinase 1 (S6K1) and eukaryotic initiation factor 4E-binding protein 1 (4E-BP1). Activation of mTORC1 promotes protein synthesis, lipid biosynthesis, and cell proliferation, while inhibiting autophagy.</li>
</ul></li>
<li><strong>mTORC2 (mechanistic target of rapamycin complex 2):</strong>
<ul>
<li>Composition: mTORC2 consists of mTOR, rapamycin-insensitive companion of mTOR (RICTOR), mammalian stress-activated protein kinase-interacting protein 1 (mSIN1), protein observed with RICTOR 1 and 2 (PROTOR1/2), and mLST8.</li>
<li>Activation: mTORC2 is less well understood than mTORC1, but it is activated by growth factors and regulates cell survival, cytoskeletal organization, and metabolism. Growth factor stimulation activates mTORC2 through PI3K/Akt pathway-dependent mechanisms.</li>
<li>Substrates and Functions: mTORC2 phosphorylates downstream effectors such as Akt at Serine 473 (Ser473), protein kinase C (PKC), and serum- and glucocorticoid-inducible kinase 1 (SGK1). Akt phosphorylation by mTORC2 at Ser473 enhances its kinase activity and promotes cell survival, proliferation, and metabolism. Additionally, mTORC2 regulates cytoskeletal dynamics and cell migration through its effects on PKC and SGK1.</li>
</ul></li>
</ol>
</section>
<section id="mtorc1-inhibits-autophagy" class="level3" data-number="7.1.2">
<h3 data-number="7.1.2" class="anchored" data-anchor-id="mtorc1-inhibits-autophagy"><span class="header-section-number">7.1.2</span> mTORC1 Inhibits Autophagy</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/mtorc1_auto.png" class="img-fluid figure-img" width="494"></p>
<p></p><figcaption class="figure-caption">mTORC1 Inhibiting Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Yes, mTORC1 (mechanistic target of rapamycin complex 1) is a potent inhibitor of autophagy, a cellular process responsible for the degradation and recycling of damaged or unnecessary cellular components. Here’s how mTORC1 inhibits autophagy:</p>
<ol type="1">
<li><p><strong>Regulation of Autophagy Initiation:</strong> mTORC1 negatively regulates the initiation of autophagy by phosphorylating and inhibiting key components of the autophagy machinery, including the Unc-51-like kinase 1 (ULK1) complex. ULK1 is a key initiator of autophagy, and its activity is inhibited by phosphorylation by mTORC1. When mTORC1 is active, it phosphorylates ULK1 and inhibits its kinase activity, thereby suppressing the initiation of autophagy.</p></li>
<li><p><strong>Inhibition of Autophagy Induction:</strong> mTORC1 also inhibits autophagy induction by regulating the activity of transcription factors involved in the expression of autophagy-related genes. For example, mTORC1 phosphorylates and inhibits the transcription factor EB (TFEB), a master regulator of lysosomal and autophagy gene expression. Inhibition of TFEB by mTORC1 prevents the transcriptional upregulation of genes involved in lysosome biogenesis and autophagy, thereby suppressing autophagy induction.</p></li>
<li><p><strong>Regulation of Autophagosome Formation:</strong> In addition to its effects on autophagy initiation and induction, mTORC1 also regulates the formation of autophagosomes, the double-membraned vesicles responsible for sequestering cellular cargo for degradation. mTORC1 inhibits the formation of autophagosomes by phosphorylating and inhibiting components of the phosphatidylinositol 3-kinase (PI3K) complex, which is required for the nucleation of the autophagosomal membrane.</p></li>
</ol>
</section>
<section id="activation-at-the-lysosomes" class="level3" data-number="7.1.3">
<h3 data-number="7.1.3" class="anchored" data-anchor-id="activation-at-the-lysosomes"><span class="header-section-number">7.1.3</span> Activation at the Lysosomes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/lysosomes.png" class="img-fluid figure-img" width="365"></p>
<p></p><figcaption class="figure-caption">Activation at Lysosomes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>mTORC1 (mechanistic target of rapamycin complex 1) activation at lysosomes is a critical step in the regulation of cell growth, metabolism, and autophagy. Lysosomes serve as signaling platforms for mTORC1 activation in response to various environmental cues, including amino acids, growth factors, and cellular energy status. Here’s how mTORC1 is activated at lysosomes:</p>
<ol type="1">
<li><p><strong>Amino Acid Sensing:</strong> Amino acids, particularly essential amino acids such as leucine, arginine, and glutamine, are potent activators of mTORC1 signaling. Upon amino acid stimulation, a series of events occur at the lysosomal surface to promote mTORC1 activation.</p></li>
<li><p><strong>Rag GTPases:</strong> The Rag GTPases, consisting of RagA/B and RagC/D heterodimers, play a central role in amino acid sensing and mTORC1 activation at lysosomes. Rag GTPases are anchored to the lysosomal membrane and function as molecular switches that regulate the recruitment of mTORC1 to lysosomes in response to amino acid availability.</p></li>
<li><p><strong>Ragulator Complex:</strong> The Rag GTPases are regulated by the Ragulator complex, a pentameric protein complex associated with the lysosomal surface. The Ragulator complex acts as a guanine nucleotide exchange factor (GEF) for RagA/B, promoting their activation in the presence of amino acids.</p></li>
<li><p><strong>Rag-mTORC1 Interaction:</strong> Activated Rag GTPases recruit mTORC1 to the lysosomal surface by interacting with the RAPTOR subunit of mTORC1. The interaction between Rag GTPases and mTORC1 is facilitated by the cytosolic scaffold protein folliculin-interacting protein 1 (FNIP1) and its homolog FNIP2.</p></li>
<li><p><strong>Rheb-GTP Loading:</strong> Once recruited to the lysosomes, mTORC1 becomes activated by the small GTPase Ras homolog enriched in brain (Rheb). Rheb is a potent activator of mTORC1 and is localized to the lysosomal membrane in its active, GTP-bound state. Rheb-GTP directly binds to and activates mTORC1, leading to its phosphorylation and activation.</p></li>
<li><p><strong>Downstream Signaling:</strong> Activated mTORC1 phosphorylates a multitude of downstream targets involved in protein synthesis, lipid metabolism, and autophagy regulation, thereby coordinating cellular responses to nutrient availability and growth signals.</p></li>
</ol>
</section>
<section id="receptor-tyroskine-kinase-activation" class="level3" data-number="7.1.4">
<h3 data-number="7.1.4" class="anchored" data-anchor-id="receptor-tyroskine-kinase-activation"><span class="header-section-number">7.1.4</span> Receptor Tyroskine Kinase Activation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/rtk_activation.png" class="img-fluid figure-img" width="416"></p>
<p></p><figcaption class="figure-caption">RTK Activation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Receptor tyrosine kinases (RTKs)</strong> are cell surface receptors that play crucial roles in signal transduction by initiating intracellular signaling cascades in response to extracellular stimuli such as growth factors, hormones, and cytokines. RTK activation typically occurs through dimerization, a process where two receptor molecules come together to form a functional complex. Here’s how RTK activation by dimerization works:</p>
<ol type="1">
<li><p><strong>Ligand Binding:</strong> RTKs have an extracellular ligand-binding domain that interacts with specific ligands, such as growth factors or hormones, in the extracellular space. Ligand binding induces a conformational change in the receptor, leading to the formation of an active signaling state.</p></li>
<li><p><strong>Receptor Dimerization:</strong> Ligand binding promotes the association of two RTK monomers, either by bringing pre-existing receptors into close proximity or by inducing receptor clustering on the cell surface. This dimerization event can occur in various configurations, including homodimerization (two identical receptor monomers) or heterodimerization (two different receptor monomers).</p></li>
<li><p><strong>Transphosphorylation:</strong> Upon dimerization, the intracellular tyrosine kinase domains of the RTK monomers become juxtaposed, allowing for cross-phosphorylation of specific tyrosine residues within the kinase activation loop of the opposing receptor. This transphosphorylation event leads to the activation of the RTK’s intrinsic kinase activity.</p></li>
<li><p><strong>Recruitment of Signaling Molecules:</strong> Phosphorylated tyrosine residues on the activated RTK serve as docking sites for various cytoplasmic signaling molecules containing Src homology 2 (SH2) or phosphotyrosine-binding (PTB) domains. These signaling molecules, such as adaptor proteins and enzymes, are recruited to the phosphorylated RTKs, where they become activated or modulate downstream signaling cascades.</p></li>
<li><p><strong>Initiation of Downstream Signaling:</strong> The recruitment and activation of downstream signaling molecules at the activated RTKs initiate intracellular signaling cascades that propagate the signal from the cell surface to the nucleus. These signaling pathways regulate diverse cellular processes, including cell proliferation, survival, differentiation, and metabolism, depending on the specific RTK and its associated signaling partners.</p></li>
</ol>
</section>
<section id="pi3k-akt-mtor-signalling-pathway" class="level3" data-number="7.1.5">
<h3 data-number="7.1.5" class="anchored" data-anchor-id="pi3k-akt-mtor-signalling-pathway"><span class="header-section-number">7.1.5</span> PI3K-AKT-mTOR Signalling Pathway</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/pi3k_stuff.png" class="img-fluid figure-img" width="467"></p>
<p></p><figcaption class="figure-caption">PI3K-AKT-mTOR Signalling Pathway</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>PI3K-AKT-mTOR signaling pathway</strong> is a key intracellular signaling pathway involved in the regulation of various cellular processes, including cell growth, proliferation, survival, metabolism, and protein synthesis. Here’s an overview of how this pathway operates:</p>
<ol type="1">
<li><p><strong>Activation of PI3K:</strong> The pathway is initiated by the activation of phosphoinositide 3-kinase (PI3K) in response to extracellular stimuli such as growth factors (e.g., insulin, insulin-like growth factor 1, epidermal growth factor). Upon ligand binding to their respective receptor tyrosine kinases (RTKs), PI3K is recruited to the plasma membrane, where it phosphorylates phosphatidylinositol 4,5-bisphosphate (PIP2) to generate phosphatidylinositol 3,4,5-trisphosphate (PIP3).</p></li>
<li><p><strong>Activation of AKT:</strong> PIP3 serves as a second messenger that recruits AKT (also known as protein kinase B) to the plasma membrane via its pleckstrin homology (PH) domain. Once localized to the membrane, AKT is phosphorylated and activated by phosphoinositide-dependent kinase 1 (PDK1) at threonine 308. Full activation of AKT also requires phosphorylation at serine 473, which can be mediated by mTOR complex 2 (mTORC2) or other kinases.</p></li>
<li><p><strong>AKT-Mediated Signaling:</strong> Activated AKT phosphorylates and regulates numerous downstream targets involved in various cellular processes. Some of the key targets of AKT include:</p>
<ul>
<li><strong>mTORC1:</strong> AKT activates mTORC1 by phosphorylating and inhibiting tuberous sclerosis complex 2 (TSC2), leading to the activation of mTORC1 and subsequent stimulation of protein synthesis and cell growth.</li>
<li><strong>Glycogen synthase kinase 3 (GSK3):</strong> AKT phosphorylates and inhibits GSK3, which regulates glycogen synthesis, protein translation, and cell survival.</li>
<li><strong>Bad:</strong> AKT phosphorylates and inactivates the pro-apoptotic protein Bad, preventing its association with anti-apoptotic Bcl-2 family members and promoting cell survival.</li>
<li><strong>FOXO transcription factors:</strong> AKT phosphorylates FOXO transcription factors, leading to their cytoplasmic sequestration and inhibition of their transcriptional activity, which regulates genes involved in apoptosis, cell cycle arrest, and oxidative stress response.</li>
</ul></li>
<li><p><strong>mTOR Signaling:</strong> The activation of mTORC1 by AKT leads to the phosphorylation and activation of downstream effectors such as ribosomal protein S6 kinase 1 (S6K1) and eukaryotic initiation factor 4E-binding protein 1 (4E-BP1). Activation of these effectors promotes protein synthesis, ribosome biogenesis, and cell growth.</p></li>
<li><p><strong>Cellular Responses:</strong> The PI3K-AKT-mTOR signaling pathway regulates various cellular processes in response to extracellular cues, including cell growth, proliferation, survival, metabolism, and protein synthesis. Dysregulation of this pathway is implicated in numerous diseases, including cancer, diabetes, and neurological disorders.</p></li>
</ol>
</section>
<section id="insulin-and-glucose-uptake" class="level3" data-number="7.1.6">
<h3 data-number="7.1.6" class="anchored" data-anchor-id="insulin-and-glucose-uptake"><span class="header-section-number">7.1.6</span> Insulin and Glucose Uptake</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/insulin_uptake.png" class="img-fluid figure-img" width="446"></p>
<p></p><figcaption class="figure-caption">Cells With and Without Glucose</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Insulin</strong> plays a central role in the regulation of glucose uptake by cells, particularly in skeletal muscle, adipose tissue, and liver. Here’s how insulin facilitates glucose uptake:</p>
<ol type="1">
<li><p><strong>Insulin Secretion:</strong> After a meal, blood glucose levels rise, triggering the secretion of insulin from pancreatic beta cells. Insulin acts as a hormone that signals cells to take up glucose from the bloodstream.</p></li>
<li><p><strong>Insulin Receptor Activation:</strong> Insulin binds to its receptor, the insulin receptor (IR), which is a receptor tyrosine kinase located on the cell surface of target cells. Binding of insulin to the IR leads to autophosphorylation and activation of the receptor’s tyrosine kinase activity.</p></li>
<li><p><strong>Insulin Signaling Cascade:</strong> Activated insulin receptors phosphorylate intracellular substrates, including insulin receptor substrate proteins (IRS), which serve as docking sites for downstream signaling molecules. Phosphorylated IRS proteins recruit and activate phosphoinositide 3-kinase (PI3K), initiating a signaling cascade.</p></li>
<li><p><strong>Activation of GLUT4 Translocation:</strong> PI3K activation leads to the phosphorylation and activation of protein kinase B (AKT), which in turn phosphorylates and activates AS160 (Akt substrate of 160 kDa). Phosphorylated AS160 facilitates the translocation of glucose transporter 4 (GLUT4) vesicles from intracellular storage compartments to the cell membrane.</p></li>
<li><p><strong>Glucose Uptake:</strong> Upon translocation to the cell membrane, GLUT4 facilitates the uptake of glucose from the extracellular space into the cytoplasm of target cells. In skeletal muscle and adipose tissue, this process significantly increases glucose uptake, allowing cells to utilize glucose for energy production or storage as glycogen or fat.</p></li>
<li><p><strong>Glucose Metabolism:</strong> Once inside the cell, glucose undergoes metabolism through glycolysis, the tricarboxylic acid (TCA) cycle, and oxidative phosphorylation to produce ATP, the cell’s primary energy source. Additionally, glucose can be stored as glycogen in the liver and muscle or converted to fatty acids for storage in adipose tissue.</p></li>
</ol>
</section>
</section>
<section id="telomeres-and-aging" class="level2" data-number="7.2">
<h2 data-number="7.2" class="anchored" data-anchor-id="telomeres-and-aging"><span class="header-section-number">7.2</span> Telomeres and Aging</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/end_stage.png" class="img-fluid figure-img" width="474"></p>
<p></p><figcaption class="figure-caption">End-Replication Problem</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Cellular senescence</strong> is a phenomenon where cells undergo irreversible growth arrest in response to various stressors, such as DNA damage, telomere shortening, oxidative stress, or oncogenic signaling. One of the key mechanisms underlying cellular senescence is the “end-replication problem.”</p>
<p>The <strong>end-replication problem</strong> refers to the inability of DNA polymerases to fully replicate the ends of linear chromosomes during DNA replication. In eukaryotic cells, chromosomes are capped by structures called <strong>telomeres</strong>, which consist of repetitive DNA sequences and associated proteins. Telomeres play a crucial role in protecting the integrity of chromosomes and preventing the loss of genetic information during DNA replication.</p>
<p>During each round of DNA replication, the DNA polymerase complex synthesizes new DNA strands in the 5’ to 3’ direction. However, due to the nature of DNA polymerases and the absence of a primer at the end of the template strand, the replication machinery is unable to replicate the extreme ends of linear chromosomes completely. As a result, with each round of cell division, telomeres progressively shorten.</p>
<p>Telomere shortening serves as a “mitotic clock” that limits the replicative lifespan of cells. When telomeres become critically short, they lose their protective function, leading to the activation of DNA damage response pathways and triggering cellular senescence. Additionally, dysfunctional telomeres can undergo inappropriate DNA repair mechanisms, resulting in genomic instability and promoting oncogenic transformation.</p>
<p>Cellular senescence mediated by the end-replication problem serves as a tumor suppressive mechanism by preventing the proliferation of cells with damaged or unstable genomes. However, excessive accumulation of senescent cells over time contributes to aging and age-related diseases, as senescent cells secrete pro-inflammatory cytokines, growth factors, and proteases collectively known as the <strong>senescence-associated secretory phenotype (SASP)</strong>, which can disrupt tissue homeostasis and promote chronic inflammation.</p>
<section id="end-replication-by-telomerase" class="level3" data-number="7.2.1">
<h3 data-number="7.2.1" class="anchored" data-anchor-id="end-replication-by-telomerase"><span class="header-section-number">7.2.1</span> End Replication by Telomerase</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/telomerase.png" class="img-fluid figure-img" width="365"></p>
<p></p><figcaption class="figure-caption">Telomerase Replication</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Telomerase</strong> is a specialized enzyme that counteracts the end-replication problem by adding repetitive DNA sequences to the ends of linear chromosomes, known as telomeres. Telomerase activity is particularly important in cells with high proliferative capacity, such as stem cells and cancer cells, as it helps maintain telomere length and ensures genomic stability during repeated rounds of cell division.</p>
<p>Here’s how telomerase mediates the end-replication process:</p>
<ol type="1">
<li><p><strong>Telomere Structure:</strong> Telomeres consist of repetitive DNA sequences (TTAGGG in humans) and associated proteins that cap the ends of linear chromosomes. Telomeres protect chromosome ends from degradation and prevent them from being recognized as DNA breaks by the cellular DNA repair machinery.</p></li>
<li><p><strong>Telomerase Enzyme:</strong> Telomerase is a ribonucleoprotein complex composed of two main components: a catalytic protein subunit called telomerase reverse transcriptase (TERT) and an RNA component called telomerase RNA (TERC). TERC serves as a template for the synthesis of telomeric DNA by TERT.</p></li>
<li><p><strong>Extension of Telomeres:</strong> During DNA replication, the lagging strand of the chromosome cannot be fully replicated due to the end-replication problem. Telomerase acts to extend the telomeres by adding repetitive DNA sequences to the 3’ end of the lagging strand.</p></li>
<li><p><strong>Telomerase Action:</strong> Telomerase extends telomeres in a processive manner by using the RNA template within TERC to synthesize repetitive telomeric DNA sequences onto the chromosome ends. This elongation process compensates for the telomere shortening that occurs during each round of DNA replication.</p></li>
<li><p><strong>Telomere Length Maintenance:</strong> By counteracting telomere shortening, telomerase helps maintain telomere length and prevents telomeres from becoming critically short. This ensures the stability of linear chromosomes and promotes cell proliferation and genomic integrity.</p></li>
<li><p><strong>Role in Stem Cells and Cancer:</strong> Telomerase activity is particularly high in stem cells and cancer cells, where it helps maintain telomere length and sustains the replicative capacity of these cells. In cancer cells, upregulation of telomerase activity allows for unlimited cell proliferation, contributing to tumor growth and progression.</p></li>
</ol>
</section>
<section id="dna-damage-response" class="level3" data-number="7.2.2">
<h3 data-number="7.2.2" class="anchored" data-anchor-id="dna-damage-response"><span class="header-section-number">7.2.2</span> DNA Damage Response</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/dna_damage.png" class="img-fluid figure-img" width="342"></p>
<p></p><figcaption class="figure-caption">Illustration of the DNA Damage Response</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>DNA damage response (DDR)</strong> is a complex network of signaling pathways that detects and repairs damaged DNA, maintaining genomic stability and preventing the accumulation of mutations that can lead to diseases such as cancer. The DDR involves a series of coordinated molecular events that are activated in response to various types of DNA damage, including DNA double-strand breaks (DSBs), single-strand breaks (SSBs), cross-links, and base modifications. Here’s an overview of the key components and processes involved in the DDR:</p>
<ol type="1">
<li><p><strong>Detection of DNA Damage:</strong> DNA damage is detected by sensor proteins that recognize specific types of lesions or abnormalities in the DNA structure. These sensor proteins include the MRN complex (MRE11-RAD50-NBS1), Ku70/Ku80 heterodimer, PARP1 (poly(ADP-ribose) polymerase 1), and various DNA damage checkpoint proteins.</p></li>
<li><p><strong>Activation of Checkpoint Signaling:</strong> Upon detection of DNA damage, sensor proteins activate checkpoint signaling pathways to halt the cell cycle and prevent the propagation of damaged DNA to daughter cells. Key checkpoint kinases involved in this process include ATM (ataxia telangiectasia mutated), ATR (ataxia telangiectasia and Rad3-related), and DNA-PK (DNA-dependent protein kinase).</p></li>
<li><p><strong>Transduction of DNA Damage Signals:</strong> Activated checkpoint kinases phosphorylate downstream effector proteins, including CHK1 and CHK2, which in turn phosphorylate a variety of target proteins involved in cell cycle regulation, DNA repair, and apoptosis. Phosphorylation of these effector proteins leads to the activation of DNA damage response pathways and the initiation of appropriate cellular responses.</p></li>
<li><p><strong>DNA Repair Pathways:</strong> The DDR coordinates several DNA repair pathways to remove or repair damaged DNA lesions, including:</p>
<ul>
<li><strong>Non-homologous end joining (NHEJ):</strong> Repairs DNA DSBs by directly ligating broken DNA ends together. NHEJ is error-prone and can lead to the introduction of small insertions or deletions.</li>
<li><strong>Homologous recombination (HR):</strong> Repairs DNA DSBs using an undamaged sister chromatid or homologous DNA sequence as a template. HR is a high-fidelity repair pathway that restores the original DNA sequence.</li>
<li><strong>Base excision repair (BER):</strong> Repairs damaged DNA bases and single-strand breaks by excising the damaged nucleotide and replacing it with a new one.</li>
<li><strong>Nucleotide excision repair (NER):</strong> Repairs bulky DNA lesions, such as UV-induced thymine dimers or chemical adducts, by removing a short stretch of nucleotides surrounding the lesion and replacing it with newly synthesized DNA.</li>
</ul></li>
<li><p><strong>Cellular Responses:</strong> Depending on the extent and type of DNA damage, cells may undergo various responses, including cell cycle arrest, DNA repair, senescence, or apoptosis. These responses are critical for maintaining genomic integrity and preventing the propagation of damaged DNA to daughter cells.</p></li>
</ol>
</section>
<section id="shelterin" class="level3" data-number="7.2.3">
<h3 data-number="7.2.3" class="anchored" data-anchor-id="shelterin"><span class="header-section-number">7.2.3</span> Shelterin</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/shelterin.png" class="img-fluid figure-img" width="506"></p>
<p></p><figcaption class="figure-caption">Shelterin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Shelterin</strong> is a protein complex that plays a critical role in protecting and regulating the structure of telomeres, the repetitive DNA sequences located at the ends of linear chromosomes. Telomeres function to maintain chromosome stability by preventing the ends of chromosomes from being recognized as DNA breaks and undergoing inappropriate DNA repair or degradation processes. Shelterin proteins bind to telomeric DNA and help to form a specialized nucleoprotein structure that shields telomeres from being recognized as damaged DNA.</p>
<p>The Shelterin complex consists of several key protein components, each with specific functions:</p>
<ol type="1">
<li><p><strong>TRF1 (Telomeric Repeat-binding Factor 1) and TRF2 (Telomeric Repeat-binding Factor 2):</strong> TRF1 and TRF2 are DNA-binding proteins that specifically recognize and bind to double-stranded telomeric DNA. They play essential roles in forming and maintaining the structure of telomeres by facilitating the formation of higher-order telomeric loops, known as T-loops. TRF2 is particularly important for protecting telomeres from being recognized as DNA double-strand breaks and promoting telomere end-capping.</p></li>
<li><p><strong>TIN2 (TRF1-interacting Nuclear factor 2):</strong> TIN2 serves as a central scaffold protein within the Shelterin complex, interacting with TRF1, TRF2, and other Shelterin components. TIN2 helps to stabilize the Shelterin complex and regulate its interactions with other proteins involved in telomere maintenance and DNA repair.</p></li>
<li><p><strong>RAP1 (Repressor/activator Protein 1):</strong> RAP1 binds to the telomeric DNA sequence in a sequence-specific manner and interacts with TRF2. RAP1 plays a role in regulating telomere length and function, as well as in coordinating telomere maintenance with other cellular processes such as transcriptional regulation and DNA repair.</p></li>
<li><p><strong>POT1 (Protection of Telomeres 1):</strong> POT1 binds specifically to the single-stranded DNA overhang present at the 3’ end of telomeres. POT1 protects telomeres from inappropriate DNA damage responses and regulates telomere length by interacting with telomerase and regulating its access to telomeric DNA.</p></li>
<li><p><strong>TPP1 (Adrenocortical Dysplasia Homolog (ACD) Protein 1):</strong> TPP1 interacts with POT1 and TIN2 within the Shelterin complex and plays a crucial role in stabilizing POT1 binding to telomeric DNA. TPP1 also facilitates the recruitment and activation of telomerase at telomeres, promoting telomere elongation and maintenance.</p></li>
</ol>
<section id="telomerase-and-shelterin" class="level4" data-number="7.2.3.1">
<h4 data-number="7.2.3.1" class="anchored" data-anchor-id="telomerase-and-shelterin"><span class="header-section-number">7.2.3.1</span> Telomerase and Shelterin</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/tloop.png" class="img-fluid figure-img" width="432"></p>
<p></p><figcaption class="figure-caption">Telomere Loop</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The telomere loop, or T-loop, is a specialized nucleoprotein structure that forms at the ends of linear chromosomes, where telomeric DNA sequences are located. The T-loop plays a crucial role in protecting chromosome ends from being recognized as DNA double-strand breaks and from undergoing inappropriate DNA repair processes. It consists of the single-stranded 3’ overhang of telomeric DNA invading and pairing with double-stranded telomeric DNA upstream, creating a looped structure.</p>
<p>Telomerase and Shelterin proteins work together to solve the end-replication problem and maintain telomere integrity:</p>
<ol type="1">
<li><p><strong>Telomerase:</strong> Telomerase is a specialized enzyme that adds repetitive DNA sequences to the ends of chromosomes, compensating for the loss of telomeric DNA during each round of DNA replication. Telomerase contains an RNA component (TERC) that serves as a template for synthesizing telomeric DNA and a catalytic protein component (TERT) that carries out the DNA synthesis reaction. By adding telomeric repeats to chromosome ends, telomerase helps counteract the gradual shortening of telomeres and ensures their stability.</p></li>
<li><p><strong>Shelterin Complex:</strong> The Shelterin complex is a group of proteins that bind to telomeric DNA and help maintain the structural integrity of telomeres. Key components of the Shelterin complex include TRF1, TRF2, TIN2, RAP1, POT1, and TPP1. Shelterin proteins play essential roles in protecting telomeres from being recognized as DNA damage sites, regulating telomere length, and coordinating telomere maintenance with other cellular processes.</p></li>
</ol>
<p>Together, telomerase and Shelterin proteins collaborate to solve the end-replication problem and maintain telomere stability. Telomerase adds telomeric repeats to chromosome ends, counteracting telomere shortening, while Shelterin proteins help form and stabilize the T-loop structure, preventing chromosome ends from being recognized as DNA breaks and protecting them from inappropriate DNA repair mechanisms.</p>
</section>
</section>
<section id="cellular-senescence" class="level3" data-number="7.2.4">
<h3 data-number="7.2.4" class="anchored" data-anchor-id="cellular-senescence"><span class="header-section-number">7.2.4</span> Cellular Senescence</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<p><img src="../imgs/week7/cellular_senesence_in.png" class="img-fluid" alt="Cellular Senescence">aging.png){fig-align=‘center’ width=506}</p>
</div>
</div>
<p>Cellular senescence is a process in which cells undergo irreversible growth arrest in response to various stressors, such as DNA damage, telomere shortening, oncogene activation, or oxidative stress. Senescent cells display distinct phenotypic changes, including altered gene expression patterns, increased secretion of inflammatory cytokines and growth factors (known as the senescence-associated secretory phenotype or SASP), and changes in cell morphology.</p>
<p>One of the key triggers for cellular senescence is telomere shortening. Telomeres are repetitive DNA sequences located at the ends of linear chromosomes, and they protect chromosome ends from degradation and fusion with neighboring chromosomes. During each round of cell division, telomeres undergo gradual shortening due to the end-replication problem, where DNA polymerases are unable to fully replicate the ends of linear chromosomes.</p>
<p>In normal somatic cells, telomerase activity is typically repressed or downregulated, leading to progressive telomere shortening with each cell division. Eventually, telomeres become critically short, triggering a DNA damage response and cellular senescence. This downregulation of telomerase expression in somatic cells serves as a tumor suppressive mechanism, as it limits the proliferative potential of cells and prevents the accumulation of genetic mutations that could lead to cancer.</p>
<p>The regulation of telomerase expression in somatic cells is complex and involves multiple factors, including epigenetic modifications, transcriptional regulation, and telomere-associated proteins. Various signaling pathways, such as the p53/p21 and p16INK4a/Rb pathways, play important roles in controlling telomerase expression and cellular senescence. For example, activation of the p53 tumor suppressor pathway in response to DNA damage or other stress signals can lead to transcriptional repression of the TERT gene, which encodes the catalytic subunit of telomerase.</p>
</section>
<section id="shelterin-and-non-homologous-end-joining" class="level3" data-number="7.2.5">
<h3 data-number="7.2.5" class="anchored" data-anchor-id="shelterin-and-non-homologous-end-joining"><span class="header-section-number">7.2.5</span> Shelterin and Non-Homologous End Joining</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week7/nh3j.png" class="img-fluid figure-img" width="510"></p>
<p></p><figcaption class="figure-caption">NHEJ and Shelterin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Telomeres play a critical role in maintaining genomic stability by protecting chromosome ends from being recognized as DNA double-strand breaks (DSBs) and undergoing inappropriate DNA repair mechanisms. Dysfunctional telomeres, resulting from excessive shortening or loss of telomeric proteins, can lead to genome instability and contribute to the development of cancer.</p>
<p>The <strong>Shelterin complex</strong>, comprised of several key proteins including TRF1, TRF2, TIN2, RAP1, POT1, and TPP1, is essential for maintaining telomere integrity and function. Shelterin proteins bind to telomeric DNA and help form a specialized nucleoprotein structure that protects chromosome ends. One of the critical functions of Shelterin is to prevent non-homologous end joining (NHEJ) at telomeres.</p>
<p>NHEJ is an error-prone DNA repair pathway that directly ligates broken DNA ends together, often resulting in the loss or addition of nucleotides and potentially leading to chromosomal rearrangements or deletions. At dysfunctional telomeres, NHEJ-mediated repair can result in telomere fusion, chromosome end-to-end fusions, or other chromosomal abnormalities, all of which contribute to genome instability and promote tumorigenesis.</p>
<p>Shelterin proteins help prevent NHEJ at telomeres through several mechanisms:</p>
<ol type="1">
<li><p><strong>TRF2-Mediated Protection:</strong> TRF2 plays a key role in protecting telomeres from being recognized as DSBs and inhibiting the activation of the DNA damage response (DDR) at chromosome ends. By forming a protective cap at telomeres, TRF2 prevents the binding of NHEJ factors and blocks the initiation of inappropriate DNA repair processes.</p></li>
<li><p><strong>Inhibition of Ku70/Ku80:</strong> The Ku70/Ku80 heterodimer is a critical component of the NHEJ pathway and is involved in DNA end recognition and tethering during repair. Shelterin proteins, particularly TRF2, have been shown to inhibit the recruitment and function of Ku70/Ku80 at telomeres, thereby preventing NHEJ-mediated repair.</p></li>
<li><p><strong>Shelterin-Mediated Sequestration:</strong> Shelterin proteins help maintain the structural integrity of telomeres and promote the formation of higher-order nucleoprotein structures, such as the T-loop. These structures sequester the chromosome end and prevent it from being recognized as a DNA break, thereby minimizing the likelihood of NHEJ-mediated repair.</p></li>
</ol>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week6.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week8.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>