<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 5&nbsp; Mitochondria in Aging and Neurodegeneration</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week6.html" rel="next">
<link href="../chapters/week4.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Plant Genome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Bioenergy and Biomaterials</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Secondary Metabolism in Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Plants as Medicine</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#mitochondria-structure-and-function" id="toc-mitochondria-structure-and-function" class="nav-link active" data-scroll-target="#mitochondria-structure-and-function"><span class="toc-section-number">5.1</span>  Mitochondria Structure and Function</a>
  <ul class="collapse">
  <li><a href="#energy-converting-metabolism-in-mitochondria" id="toc-energy-converting-metabolism-in-mitochondria" class="nav-link" data-scroll-target="#energy-converting-metabolism-in-mitochondria"><span class="toc-section-number">5.1.1</span>  Energy-Converting Metabolism in Mitochondria</a></li>
  <li><a href="#glucose-to-nadh" id="toc-glucose-to-nadh" class="nav-link" data-scroll-target="#glucose-to-nadh"><span class="toc-section-number">5.1.2</span>  Glucose to NADH</a></li>
  <li><a href="#electron-transport-chain" id="toc-electron-transport-chain" class="nav-link" data-scroll-target="#electron-transport-chain"><span class="toc-section-number">5.1.3</span>  Electron Transport Chain</a></li>
  <li><a href="#baffle-versus-crista-junction-model" id="toc-baffle-versus-crista-junction-model" class="nav-link" data-scroll-target="#baffle-versus-crista-junction-model"><span class="toc-section-number">5.1.4</span>  Baffle versus Crista Junction Model</a></li>
  <li><a href="#electron-transport-in-the-cristae" id="toc-electron-transport-in-the-cristae" class="nav-link" data-scroll-target="#electron-transport-in-the-cristae"><span class="toc-section-number">5.1.5</span>  Electron Transport in the Cristae</a></li>
  <li><a href="#origins-of-the-mitochondria" id="toc-origins-of-the-mitochondria" class="nav-link" data-scroll-target="#origins-of-the-mitochondria"><span class="toc-section-number">5.1.6</span>  Origins of the Mitochondria</a></li>
  <li><a href="#biogenesis-of-respiratory-chain-proteins" id="toc-biogenesis-of-respiratory-chain-proteins" class="nav-link" data-scroll-target="#biogenesis-of-respiratory-chain-proteins"><span class="toc-section-number">5.1.7</span>  Biogenesis of Respiratory Chain Proteins</a></li>
  </ul></li>
  <li><a href="#reactive-oxygen-species" id="toc-reactive-oxygen-species" class="nav-link" data-scroll-target="#reactive-oxygen-species"><span class="toc-section-number">5.2</span>  Reactive Oxygen Species</a>
  <ul class="collapse">
  <li><a href="#radical-theory" id="toc-radical-theory" class="nav-link" data-scroll-target="#radical-theory"><span class="toc-section-number">5.2.1</span>  Radical Theory</a></li>
  <li><a href="#superoxide-dismutase" id="toc-superoxide-dismutase" class="nav-link" data-scroll-target="#superoxide-dismutase"><span class="toc-section-number">5.2.2</span>  Superoxide Dismutase</a></li>
  </ul></li>
  <li><a href="#mitochondria-and-apoptosis" id="toc-mitochondria-and-apoptosis" class="nav-link" data-scroll-target="#mitochondria-and-apoptosis"><span class="toc-section-number">5.3</span>  Mitochondria and Apoptosis</a>
  <ul class="collapse">
  <li><a href="#caspase-activation" id="toc-caspase-activation" class="nav-link" data-scroll-target="#caspase-activation"><span class="toc-section-number">5.3.1</span>  Caspase Activation</a></li>
  <li><a href="#phagocytosis-of-apoptotic-cells" id="toc-phagocytosis-of-apoptotic-cells" class="nav-link" data-scroll-target="#phagocytosis-of-apoptotic-cells"><span class="toc-section-number">5.3.2</span>  Phagocytosis of Apoptotic Cells</a></li>
  <li><a href="#pathways-of-apoptosis" id="toc-pathways-of-apoptosis" class="nav-link" data-scroll-target="#pathways-of-apoptosis"><span class="toc-section-number">5.3.3</span>  Pathways of Apoptosis</a></li>
  <li><a href="#bak-bax-and-cytochrome-c-release" id="toc-bak-bax-and-cytochrome-c-release" class="nav-link" data-scroll-target="#bak-bax-and-cytochrome-c-release"><span class="toc-section-number">5.3.4</span>  Bak, Bax, and Cytochrome C Release</a></li>
  <li><a href="#inactivating-bak-and-bax-with-bcl-2" id="toc-inactivating-bak-and-bax-with-bcl-2" class="nav-link" data-scroll-target="#inactivating-bak-and-bax-with-bcl-2"><span class="toc-section-number">5.3.5</span>  Inactivating Bak and Bax with Bcl-2</a></li>
  <li><a href="#bh3-only-proteins" id="toc-bh3-only-proteins" class="nav-link" data-scroll-target="#bh3-only-proteins"><span class="toc-section-number">5.3.6</span>  BH3-Only Proteins</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week5.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<section id="mitochondria-structure-and-function" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="mitochondria-structure-and-function"><span class="header-section-number">5.1</span> Mitochondria Structure and Function</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/mitochondria.png" class="img-fluid figure-img" width="243"></p>
<p></p><figcaption class="figure-caption">Photo of a Mitochondrion</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>mitochondrion</strong>, often referred to as the “powerhouse of the cell,” is a complex organelle with a distinct structure that allows it to carry out essential cellular functions, particularly energy production through oxidative phosphorylation. Here’s an overview of its structure:</p>
<ol type="1">
<li><p><strong>Matrix:</strong></p>
<p>The matrix is the innermost compartment of the mitochondrion and contains a gel-like substance rich in enzymes, including those involved in the oxidation of pyruvate and fatty acids, as well as enzymes of the Krebs cycle (also known as the citric acid cycle or tricarboxylic acid cycle). Additionally, the matrix houses the mitochondrial DNA (mtDNA) and the machinery for gene expression, including RNA polymerase and ribosomes.</p></li>
<li><p><strong>Inner Membrane:</strong></p>
<p>The inner membrane surrounds the matrix and is highly folded into structures called cristae, which increase the surface area available for biochemical reactions. The inner membrane contains the electron transport chain (ETC), a series of protein complexes that transfer electrons derived from nutrients to generate ATP through oxidative phosphorylation. It also houses ATP synthase, an enzyme complex responsible for ATP production, as well as various transporters that regulate the passage of ions and metabolites across the membrane.</p></li>
<li><p><strong>Outer Membrane:</strong></p>
<p>The outer membrane surrounds the inner membrane and separates the mitochondrial interior from the cytoplasm. It is relatively permeable to small molecules with a molecular weight of less than 5 kilodaltons (kDa), allowing for the passage of ions, metabolites, and other small molecules.</p></li>
<li><p><strong>Intermembrane Space:</strong></p>
<p>The intermembrane space is the region between the inner and outer mitochondrial membranes. It contains a lower concentration of proteins compared to the matrix but may contain proteins involved in processes such as apoptosis regulation.</p></li>
</ol>
<section id="energy-converting-metabolism-in-mitochondria" class="level3" data-number="5.1.1">
<h3 data-number="5.1.1" class="anchored" data-anchor-id="energy-converting-metabolism-in-mitochondria"><span class="header-section-number">5.1.1</span> Energy-Converting Metabolism in Mitochondria</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/energy_converting.png" class="img-fluid figure-img" width="332"></p>
<p></p><figcaption class="figure-caption">Energy Converting Metabolism Diagram</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Energy-converting metabolism in mitochondria is a complex process that involves several interconnected pathways aimed at generating adenosine triphosphate (ATP), the cell’s primary energy currency. Mitochondria play a central role in cellular respiration, a process that involves the oxidation of nutrients to produce ATP through oxidative phosphorylation. Here’s an overview of the key metabolic pathways involved:</p>
<ol type="1">
<li><p><strong>Glycolysis:</strong> Glycolysis is the initial stage of glucose metabolism that occurs in the cytoplasm. Glucose is broken down into pyruvate, producing a small amount of ATP and reducing equivalents in the form of NADH. Pyruvate is then transported into the mitochondria for further metabolism.</p></li>
<li><p><strong>Pyruvate Oxidation:</strong> Pyruvate produced from glycolysis enters the mitochondria, where it undergoes oxidation to produce acetyl-CoA. This process generates reducing equivalents in the form of NADH and releases carbon dioxide. Acetyl-CoA then enters the citric acid cycle (also known as the Krebs cycle) for further metabolism.</p></li>
<li><p><strong>Citric Acid Cycle (Krebs Cycle):</strong> The citric acid cycle takes place in the mitochondrial matrix. Acetyl-CoA combines with oxaloacetate to form citrate, which undergoes a series of enzymatic reactions, ultimately regenerating oxaloacetate and producing reducing equivalents (NADH and FADH2) and ATP precursors (GTP). The cycle generates NADH and FADH2, which donate electrons to the electron transport chain (ETC).</p></li>
<li><p><strong>Electron Transport Chain (ETC):</strong> The electron transport chain is located in the inner mitochondrial membrane. It consists of a series of protein complexes (I-IV) and coenzyme Q and cytochrome c as electron carriers. As electrons move through the ETC, they release energy, which is used to pump protons (H+) across the inner mitochondrial membrane, establishing an electrochemical gradient.</p></li>
<li><p><strong>ATP Synthesis:</strong> The proton gradient generated by the ETC drives ATP synthesis through ATP synthase, an enzyme complex located in the inner mitochondrial membrane. Protons flow back into the mitochondrial matrix through ATP synthase, driving the synthesis of ATP from adenosine diphosphate (ADP) and inorganic phosphate (Pi). This process is known as oxidative phosphorylation.</p></li>
</ol>
</section>
<section id="glucose-to-nadh" class="level3" data-number="5.1.2">
<h3 data-number="5.1.2" class="anchored" data-anchor-id="glucose-to-nadh"><span class="header-section-number">5.1.2</span> Glucose to NADH</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/nadh_generation.png" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">NADH Generation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Glucose undergoes several metabolic reactions to produce NADH during cellular respiration. The process begins with glycolysis, where glucose is converted into pyruvate. Here’s a simplified breakdown of the steps involved:</p>
<ol type="1">
<li><p><strong>Glycolysis:</strong> Glucose (a 6-carbon molecule) is phosphorylated and split into two molecules of glyceraldehyde-3-phosphate (G3P), each containing three carbons. These G3P molecules are further oxidized and phosphorylated, ultimately yielding two molecules of pyruvate (a 3-carbon molecule). Along the way, glycolysis produces a net gain of 2 ATP molecules and 2 NADH molecules per glucose molecule.</p></li>
<li><p><strong>Pyruvate Oxidation:</strong> Each pyruvate molecule produced during glycolysis enters the mitochondria, where it undergoes oxidation to produce acetyl-CoA. This process generates reducing equivalents in the form of NADH. While pyruvate oxidation does not directly produce NADH molecules, it generates NADH during subsequent steps of the citric acid cycle (Krebs cycle).</p></li>
<li><p><strong>Krebs Cycle (Citric Acid Cycle):</strong> Acetyl-CoA combines with oxaloacetate to form citrate, initiating the citric acid cycle. Through a series of enzymatic reactions, citrate is metabolized, ultimately regenerating oxaloacetate and producing reducing equivalents (NADH and FADH2). While the precise contribution of each pyruvate to NADH production in the Krebs cycle varies, each glucose molecule ultimately yields multiple NADH molecules through this pathway.</p></li>
</ol>
<section id="nadh-usage" class="level4" data-number="5.1.2.1">
<h4 data-number="5.1.2.1" class="anchored" data-anchor-id="nadh-usage"><span class="header-section-number">5.1.2.1</span> NADH Usage</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/nadh_usage.png" class="img-fluid figure-img" width="437"></p>
<p></p><figcaption class="figure-caption">NADH Usage</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>NADH (nicotinamide adenine dinucleotide, reduced form)</strong> is a critical molecule in cellular metabolism, serving as a carrier of electrons during various biochemical reactions. Its primary role is to transfer high-energy electrons to the electron transport chain (ETC) in the mitochondria, where they are ultimately used to generate ATP through oxidative phosphorylation.</p>
<p>Here’s how NADH is used in cellular metabolism:</p>
<ol type="1">
<li><p><strong>Electron Transport Chain (ETC):</strong> NADH donates its electrons to complex I (NADH dehydrogenase) of the electron transport chain in the inner mitochondrial membrane. These electrons then pass through a series of protein complexes (I-IV), including flavin mononucleotide (FMN), iron-sulfur clusters, and cytochromes, leading to the pumping of protons (H+) from the mitochondrial matrix to the intermembrane space. This creates an electrochemical gradient across the inner mitochondrial membrane.</p></li>
<li><p><strong>ATP Synthesis:</strong> The flow of protons back into the mitochondrial matrix through complex V (ATP synthase) harnesses the energy released to drive the synthesis of ATP from adenosine diphosphate (ADP) and inorganic phosphate (Pi). This process, known as oxidative phosphorylation, is responsible for the majority of ATP production in aerobic cells.</p></li>
<li><p><strong>Regeneration of NAD+:</strong> After donating its electrons to the ETC, NADH is oxidized back to its original form, NAD+, allowing it to participate in subsequent rounds of metabolism. This regeneration of NAD+ is crucial for sustaining glycolysis, the citric acid cycle, and other metabolic pathways that rely on NAD+ as a cofactor.</p></li>
</ol>
</section>
</section>
<section id="electron-transport-chain" class="level3" data-number="5.1.3">
<h3 data-number="5.1.3" class="anchored" data-anchor-id="electron-transport-chain"><span class="header-section-number">5.1.3</span> Electron Transport Chain</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/electron_transport_chian.png" class="img-fluid figure-img" width="428"></p>
<p></p><figcaption class="figure-caption">Illustration of the Electron Transport Chain</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>Electron Transport Chain (ETC)</strong> is a critical component of cellular respiration, occurring in the inner mitochondrial membrane of eukaryotic cells and the plasma membrane of prokaryotic cells. It is responsible for the final stage of oxidative phosphorylation, the process by which cells generate adenosine triphosphate (ATP), the primary energy currency.</p>
<p>Here’s how the Electron Transport Chain works:</p>
<ol type="1">
<li><p><strong>Electron Carriers:</strong> The ETC consists of a series of protein complexes and small molecules that serve as electron carriers. These include flavoproteins, iron-sulfur proteins, cytochromes, and coenzyme Q (ubiquinone).</p></li>
<li><p><strong>Electron Flow:</strong> Electrons donated by NADH and FADH2 (produced during glycolysis, pyruvate oxidation, and the citric acid cycle) enter the ETC at complex I (NADH dehydrogenase) and complex II (succinate dehydrogenase), respectively. These electrons are transferred sequentially from one carrier to another along the ETC, losing energy with each transfer.</p></li>
<li><p><strong>Proton Pumping:</strong> As electrons flow through the ETC, energy released from their transfer is used to pump protons (H+) from the mitochondrial matrix to the intermembrane space, creating an electrochemical gradient across the inner mitochondrial membrane. Complexes I, III, and IV are involved in this process.</p></li>
<li><p><strong>Chemiosmosis:</strong> The electrochemical gradient generated by proton pumping creates a proton motive force (PMF), which drives protons back into the mitochondrial matrix through complex V, also known as ATP synthase. As protons flow through ATP synthase, the energy released is used to convert adenosine diphosphate (ADP) and inorganic phosphate (Pi) into ATP, a process known as oxidative phosphorylation.</p></li>
<li><p><strong>Oxygen as the Final Electron Acceptor:</strong> At the end of the ETC, electrons are transferred to molecular oxygen (O2), the final electron acceptor. Oxygen combines with protons to form water (H2O), completing the electron transport chain.</p></li>
</ol>
<section id="alternative-pathways" class="level4" data-number="5.1.3.1">
<h4 data-number="5.1.3.1" class="anchored" data-anchor-id="alternative-pathways"><span class="header-section-number">5.1.3.1</span> Alternative Pathways</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/chain.png" class="img-fluid figure-img" width="516"></p>
<p></p><figcaption class="figure-caption">Different Paths in the Electron Transport Chain</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In the Electron Transport Chain (ETC), there are two main pathways for the flow of electrons:</p>
<ol type="1">
<li><p><strong>Complex I → Complex III → Complex IV:</strong> This pathway begins with complex I (NADH dehydrogenase) receiving electrons from NADH. Complex I transfers these electrons to coenzyme Q (ubiquinone), which then delivers them to complex III (cytochrome bc1 complex). Complex III passes the electrons to cytochrome c, and finally, they are transferred to complex IV (cytochrome c oxidase). Complex IV uses these electrons to reduce molecular oxygen (O2) to water (H2O).</p></li>
<li><p><strong>Complex II → Complex III → Complex IV:</strong> This pathway involves complex II (succinate dehydrogenase) receiving electrons directly from succinate in the citric acid cycle. Complex II transfers these electrons to coenzyme Q, which then follows the same path as in the previous pathway, delivering the electrons to complex III and then complex IV for the reduction of oxygen to water.</p></li>
</ol>
</section>
</section>
<section id="baffle-versus-crista-junction-model" class="level3" data-number="5.1.4">
<h3 data-number="5.1.4" class="anchored" data-anchor-id="baffle-versus-crista-junction-model"><span class="header-section-number">5.1.4</span> Baffle versus Crista Junction Model</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/models.png" class="img-fluid figure-img" width="466"></p>
<p></p><figcaption class="figure-caption">Different Models of the ER</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The Baffle Model and the Crista Junction Model are two proposed mechanisms that describe the organization of cristae, the internal membrane structures within mitochondria, and their role in mitochondrial function. Let’s delve into each model:</p>
<ol type="1">
<li><p><strong>Baffle Model:</strong></p>
<p>The Baffle Model suggests that cristae are formed by invaginations of the inner mitochondrial membrane (IMM) that protrude into the mitochondrial matrix. These invaginations create a labyrinth-like structure, increasing the surface area of the inner membrane and providing more space for oxidative phosphorylation (ATP production). The term “baffles” refers to the folds or ridges formed by these invaginations.</p>
<p>In this model, the cristae junctions, which are the points where the inner membrane folds connect to the outer membrane, are considered to be relatively sparse and randomly distributed. The spaces between cristae (intercristae spaces) are thought to allow for efficient diffusion of metabolites and ions, facilitating metabolic processes within the mitochondria.</p></li>
<li><p><strong>Crista Junction Model:</strong></p>
<p>The Crista Junction Model proposes a different organization of cristae within mitochondria. According to this model, cristae are interconnected by narrow tubular structures known as cristae junctions. These junctions serve as gateways that regulate the exchange of molecules between the intermembrane space and the mitochondrial matrix.</p>
<p>In the Crista Junction Model, the cristae are believed to be more tightly packed and organized compared to the Baffle Model. The presence of cristae junctions allows for compartmentalization of the inner mitochondrial membrane, segregating different metabolic processes and creating microenvironments within the mitochondria.</p></li>
</ol>
</section>
<section id="electron-transport-in-the-cristae" class="level3" data-number="5.1.5">
<h3 data-number="5.1.5" class="anchored" data-anchor-id="electron-transport-in-the-cristae"><span class="header-section-number">5.1.5</span> Electron Transport in the Cristae</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/electron_transport_cristae.png" class="img-fluid figure-img" width="357"></p>
<p></p><figcaption class="figure-caption">Electron Transport in the Cristae</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The electron transport chain (ETC) in mitochondria is a crucial process in cellular respiration, where electrons derived from the oxidation of nutrients are passed along a series of protein complexes embedded in the inner mitochondrial membrane (IMM). This process generates a proton gradient across the membrane, which is used to produce adenosine triphosphate (ATP) through oxidative phosphorylation. Cristae, the inward-folded structures of the IMM, play a critical role in facilitating the ETC and ATP synthesis. Here’s how the ETC functions within the cristae:</p>
<ol type="1">
<li><p><strong>Organization within Cristae:</strong> Cristae provide an extensive surface area for the assembly and functioning of the ETC complexes. The proteins comprising the ETC, including complex I (NADH dehydrogenase), complex II (succinate dehydrogenase), complex III (cytochrome bc1 complex), complex IV (cytochrome c oxidase), and ATP synthase (complex V), are embedded within the cristae membranes.</p></li>
<li><p><strong>Electron Transfer:</strong> The ETC begins with the transfer of electrons from NADH or FADH2 to complex I or complex II, respectively. These electrons are then passed sequentially through the ETC complexes, with each complex accepting and donating electrons to the next in the chain. As electrons move along the chain, they release energy that is used to pump protons (H+) across the cristae membrane from the mitochondrial matrix to the intermembrane space.</p></li>
<li><p><strong>Proton Gradient Formation:</strong> The pumping of protons from the mitochondrial matrix into the intermembrane space creates a proton gradient across the cristae membrane. This gradient represents stored energy that can be harnessed to drive ATP synthesis.</p></li>
<li><p><strong>ATP Synthesis:</strong> The proton gradient established by the ETC drives the rotation of the rotor subunit within ATP synthase (complex V). This rotational motion catalyzes the synthesis of ATP from adenosine diphosphate (ADP) and inorganic phosphate (Pi) in a process known as chemiosmosis. ATP synthase is located within the cristae membrane and utilizes the energy from proton movement to generate ATP.</p></li>
</ol>
</section>
<section id="origins-of-the-mitochondria" class="level3" data-number="5.1.6">
<h3 data-number="5.1.6" class="anchored" data-anchor-id="origins-of-the-mitochondria"><span class="header-section-number">5.1.6</span> Origins of the Mitochondria</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/origins.png" class="img-fluid figure-img" width="496"></p>
<p></p><figcaption class="figure-caption">Origins of the Mitochondria</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Around 1.5 billion years ago, a remarkable event occurred in the history of life on Earth: the engulfment of a free-living bacterium by a primitive eukaryotic cell. This bacterium was likely an alpha-proteobacterium, a group of bacteria closely related to modern-day mitochondria. Instead of being digested by the host cell, this bacterium formed a symbiotic relationship with it, possibly providing additional energy in the form of ATP through aerobic respiration.</p>
<p>Over time, this symbiotic relationship evolved, with the engulfed bacterium becoming increasingly integrated into the host cell. This integration involved a gradual transfer of genetic material from the bacterium to the host cell nucleus, resulting in the loss of many bacterial genes and the retention of only a small genome within the mitochondria. This genome is now known as mitochondrial DNA (mtDNA).</p>
<p>As this symbiosis continued, the host cell and the engulfed bacterium became interdependent, with the bacterium specializing in energy production through oxidative phosphorylation (aerobic respiration) while the host cell provided protection and resources. This symbiotic relationship proved to be highly advantageous, allowing eukaryotic cells to generate more energy and thus thrive in diverse environments.</p>
<p>Through natural selection and genetic drift, the symbiotic bacterium gradually evolved into the mitochondria we see in modern eukaryotic cells. Mitochondria retained their own distinct characteristics, such as a double membrane structure and their own genome (mtDNA), while also becoming integrated into the cellular machinery of the host cell.</p>
</section>
<section id="biogenesis-of-respiratory-chain-proteins" class="level3" data-number="5.1.7">
<h3 data-number="5.1.7" class="anchored" data-anchor-id="biogenesis-of-respiratory-chain-proteins"><span class="header-section-number">5.1.7</span> Biogenesis of Respiratory Chain Proteins</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/resp_chain_proteins.png" class="img-fluid figure-img" width="440"></p>
<p></p><figcaption class="figure-caption">Biogenesis of Respiratory Chain Proteins</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The biogenesis of respiratory chain proteins, which are essential components of the electron transport chain (ETC) in mitochondria, involves a complex process of synthesis, assembly, and insertion into the inner mitochondrial membrane (IMM). Here’s an overview of the steps involved:</p>
<ol type="1">
<li><p><strong>Synthesis of Mitochondrial Proteins:</strong> The majority of mitochondrial proteins, including those involved in the respiratory chain, are encoded by nuclear genes and synthesized in the cytoplasm as precursor proteins. These precursor proteins typically contain N-terminal targeting sequences, called mitochondrial targeting sequences (MTS) or presequences, that direct them to the mitochondria.</p></li>
<li><p><strong>Import into Mitochondria:</strong> Upon synthesis in the cytoplasm, mitochondrial precursor proteins are recognized by cytoplasmic chaperones and escorted to the translocase of the outer membrane (TOM) complex, which is responsible for the initial recognition and translocation of precursor proteins across the outer mitochondrial membrane (OMM). The MTS directs the precursor protein through the TOM complex into the intermembrane space (IMS).</p></li>
<li><p><strong>Sorting and Processing:</strong> Once in the IMS, the MTS is cleaved by the mitochondrial processing peptidase (MPP), resulting in the removal of the targeting sequence and yielding the mature protein. Some proteins require further processing by additional proteases within the IMS.</p></li>
<li><p><strong>Assembly of Respiratory Complexes:</strong> Mature proteins synthesized in the cytoplasm or imported into the mitochondria are assembled into multi-subunit complexes that constitute the respiratory chain. These complexes include complex I (NADH dehydrogenase), complex II (succinate dehydrogenase), complex III (cytochrome bc1 complex), complex IV (cytochrome c oxidase), and ATP synthase (complex V).</p></li>
<li><p><strong>Insertion into the Inner Mitochondrial Membrane (IMM):</strong> The assembled respiratory chain complexes are inserted into the IMM with the assistance of various assembly factors and chaperones. The precise mechanisms of insertion and assembly vary for each complex but often involve interactions with specific lipid components of the IMM and assembly factors that facilitate proper folding and integration into the membrane.</p></li>
<li><p><strong>Formation of Supramolecular Structures:</strong> Once inserted into the IMM, individual respiratory chain complexes can interact with each other to form supramolecular structures known as supercomplexes or respirasomes. These structures optimize electron transfer efficiency and minimize reactive oxygen species (ROS) production by facilitating the rapid and efficient transfer of electrons through the ETC.</p></li>
</ol>
<section id="mitochondrial-targeting-sequence" class="level4" data-number="5.1.7.1">
<h4 data-number="5.1.7.1" class="anchored" data-anchor-id="mitochondrial-targeting-sequence"><span class="header-section-number">5.1.7.1</span> Mitochondrial Targeting Sequence</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/targets.png" class="img-fluid figure-img" width="410"></p>
<p></p><figcaption class="figure-caption">Targeting Sequences</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>mitochondrial targeting sequence (MTS)</strong>, also known as a mitochondrial transit peptide or presequence, is a short, hydrophobic peptide sequence found at the N-terminus of proteins destined for import into mitochondria. The MTS serves as a molecular tag that directs precursor proteins from the cytoplasm to the mitochondria, where they undergo further processing and are ultimately integrated into the mitochondrial compartments.</p>
<p>Here’s how the mitochondrial targeting sequence works:</p>
<ol type="1">
<li><p><strong>Recognition in the Cytoplasm:</strong> Newly synthesized precursor proteins containing an MTS are recognized by cytoplasmic chaperones and targeting factors. These factors prevent the premature folding of the precursor protein and help maintain its unfolded state, facilitating its recognition by the mitochondrial import machinery.</p></li>
<li><p><strong>Translocation across the Outer Membrane:</strong> The precursor protein, bound to cytoplasmic chaperones, is guided to the translocase of the outer membrane (TOM) complex, which is located in the outer mitochondrial membrane (OMM). The TOM complex recognizes the MTS and facilitates the translocation of the precursor protein across the OMM into the intermembrane space (IMS) of the mitochondria.</p></li>
<li><p><strong>Processing and Sorting:</strong> Once in the IMS, the MTS is cleaved by the mitochondrial processing peptidase (MPP), resulting in the removal of the targeting sequence. This cleavage event releases the mature protein, which may then be further sorted and directed to its specific destination within the mitochondria, such as the matrix, inner mitochondrial membrane (IMM), or intermembrane space.</p></li>
<li><p><strong>Integration into Mitochondrial Compartments:</strong> The mature protein is then integrated into the appropriate mitochondrial compartment, where it performs its specific function. This integration process may involve additional folding, assembly into multi-subunit complexes, or insertion into the IMM, depending on the protein’s destination and function within the mitochondria.</p></li>
</ol>
</section>
<section id="mitochondrial-protein-import" class="level4" data-number="5.1.7.2">
<h4 data-number="5.1.7.2" class="anchored" data-anchor-id="mitochondrial-protein-import"><span class="header-section-number">5.1.7.2</span> Mitochondrial Protein Import</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/protein_import.png" class="img-fluid figure-img" width="473"></p>
<p></p><figcaption class="figure-caption">Importing Proteins into the Mitochondria</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Mitochondrial protein import is a fundamental process by which proteins synthesized in the cytoplasm are transported into mitochondria, where they perform essential functions in various mitochondrial compartments. This import process is highly regulated and involves several steps, ensuring that proteins are correctly targeted to their specific destinations within mitochondria. Here’s an overview of mitochondrial protein import:</p>
<ol type="1">
<li><p><strong>Recognition of Mitochondrial Targeting Sequence (MTS):</strong> Proteins destined for import into mitochondria typically contain an N-terminal mitochondrial targeting sequence (MTS), also known as a mitochondrial transit peptide or presequence. The MTS is recognized by cytoplasmic chaperones and targeting factors, which prevent premature folding of the precursor protein and facilitate its recognition by the mitochondrial import machinery.</p></li>
<li><p><strong>Translocation Across the Outer Membrane:</strong> The precursor protein, bound to cytoplasmic chaperones, is directed to the translocase of the outer membrane (TOM) complex, located in the outer mitochondrial membrane (OMM). The TOM complex recognizes the MTS and facilitates the translocation of the precursor protein across the OMM into the intermembrane space (IMS) of the mitochondria.</p></li>
<li><p><strong>Sorting and Processing in the Intermembrane Space (IMS):</strong> Once in the IMS, the MTS is cleaved by the mitochondrial processing peptidase (MPP), resulting in the removal of the targeting sequence. This cleavage event releases the mature protein, which may then be further sorted and directed to its specific destination within the mitochondria, such as the matrix, inner mitochondrial membrane (IMM), or intermembrane space.</p></li>
<li><p><strong>Translocation Across the Inner Membrane:</strong> Proteins targeted to the mitochondrial matrix or inner mitochondrial membrane (IMM) undergo additional translocation steps across the inner membrane. This process is facilitated by protein translocases, including the presequence translocase-associated motor (PAM) complex and the TIM22 and TIM23 complexes. These translocases recognize and transport proteins with specific targeting signals across the IMM into the matrix or inner membrane.</p></li>
<li><p><strong>Integration into Mitochondrial Compartments:</strong> Once inside the mitochondrial matrix or IMM, proteins may undergo further folding, assembly into multi-subunit complexes, or insertion into the membrane, depending on their specific functions within mitochondria. Some proteins may also remain in the IMS, where they perform specialized roles in processes such as protein folding, oxidative protein folding, or apoptosis regulation.</p></li>
</ol>
</section>
<section id="energy-and-protein-import" class="level4" data-number="5.1.7.3">
<h4 data-number="5.1.7.3" class="anchored" data-anchor-id="energy-and-protein-import"><span class="header-section-number">5.1.7.3</span> Energy and Protein Import</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/energy_import.png" class="img-fluid figure-img" width="506"></p>
<p></p><figcaption class="figure-caption">Energy Requirements During Import</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Energy plays a crucial role in mitochondrial protein import, facilitating the translocation of precursor proteins across the mitochondrial membranes and their subsequent sorting and processing within mitochondria. Here’s how energy is involved in various stages of mitochondrial protein import:</p>
<ol type="1">
<li><p><strong>Translocation Across the Outer Membrane:</strong> The translocation of precursor proteins across the outer mitochondrial membrane (OMM) is facilitated by the translocase of the outer membrane (TOM) complex. While some precursor proteins are passively translocated through the TOM complex, others require energy in the form of ATP to overcome the electrochemical gradient across the membrane and enter the intermembrane space (IMS) of the mitochondria. ATP hydrolysis provides the energy needed for the initial translocation step.</p></li>
<li><p><strong>Translocation Across the Inner Membrane:</strong> Precursor proteins targeted to the mitochondrial matrix or inner mitochondrial membrane (IMM) undergo additional translocation steps across the inner membrane. This process is facilitated by protein translocases, including the presequence translocase-associated motor (PAM) complex and the TIM22 and TIM23 complexes. ATP hydrolysis by the PAM complex provides the energy required for the translocation of proteins across the inner membrane into the matrix or inner membrane.</p></li>
<li><p><strong>Folding, Sorting, and Processing:</strong> Once inside the mitochondrial matrix or IMM, precursor proteins may undergo folding, assembly into multi-subunit complexes, or insertion into the membrane. These processes often require energy in the form of ATP to facilitate protein folding, conformational changes, and membrane insertion. ATP-dependent chaperones and protein folding enzymes within mitochondria utilize ATP hydrolysis to assist in protein folding and quality control.</p></li>
<li><p><strong>Maintenance of Membrane Potential:</strong> The maintenance of an electrochemical gradient across the mitochondrial membranes, driven by proton pumping by the electron transport chain (ETC), is essential for mitochondrial protein import. This proton gradient provides the driving force for the import of precursor proteins into mitochondria and the translocation of proteins across the inner membrane. ATP hydrolysis by the ATP synthase complex, located in the inner membrane, contributes to the maintenance of the proton gradient by consuming ATP to drive proton pumping.</p></li>
</ol>
</section>
</section>
</section>
<section id="reactive-oxygen-species" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="reactive-oxygen-species"><span class="header-section-number">5.2</span> Reactive Oxygen Species</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/superoxide.png" class="img-fluid figure-img" width="298"></p>
<p></p><figcaption class="figure-caption">Superoxide Generation from the Mitochondria</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Superoxide generation by mitochondria is a natural byproduct of the electron transport chain (ETC) during oxidative phosphorylation, the process by which cells produce adenosine triphosphate (ATP). While mitochondria play a crucial role in cellular energy production, they can also generate reactive oxygen species (ROS), including superoxide radicals (O2•−), as a result of electron leakage from the ETC. Here’s how superoxide is generated by mitochondria:</p>
<ol type="1">
<li><p><strong>Electron Leakage:</strong> During oxidative phosphorylation, electrons are transferred along the ETC through a series of protein complexes embedded in the inner mitochondrial membrane (IMM). While the majority of electrons flow smoothly through the ETC, some leakage can occur at various points along the chain, particularly at complexes I and III. This electron leakage leads to the partial reduction of molecular oxygen (O2) to form superoxide radicals (O2•−).</p></li>
<li><p><strong>Site of Generation:</strong> The primary sites of superoxide generation within mitochondria are believed to be complexes I and III of the ETC. Complex I (NADH dehydrogenase) and complex III (cytochrome bc1 complex) are known to produce superoxide radicals as a result of single-electron reduction of molecular oxygen by ubisemiquinone radicals (ubisemiquinone-O2 reaction) and cytochrome b, respectively.</p></li>
<li><p><strong>Regulation and Mitochondrial ROS Production:</strong> Mitochondrial superoxide generation is regulated by various factors, including the redox state of electron carriers within the ETC, the proton motive force across the inner mitochondrial membrane (IMM), and the availability of oxygen as a terminal electron acceptor. Changes in mitochondrial membrane potential, substrate availability, or ETC inhibitors can alter ROS production by mitochondria.</p></li>
<li><p><strong>Physiological and Pathological Roles:</strong> While superoxide generation by mitochondria is a natural process essential for cellular signaling and redox homeostasis, excessive ROS production can have detrimental effects on cellular components, leading to oxidative damage, mitochondrial dysfunction, and cell death. Mitochondrial ROS have been implicated in various physiological processes, including cell signaling, apoptosis, and aging, as well as in the pathogenesis of numerous diseases, including neurodegenerative disorders, cardiovascular diseases, and cancer.</p></li>
</ol>
<section id="radical-theory" class="level3" data-number="5.2.1">
<h3 data-number="5.2.1" class="anchored" data-anchor-id="radical-theory"><span class="header-section-number">5.2.1</span> Radical Theory</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/radical_theory.png" class="img-fluid figure-img" width="267"></p>
<p></p><figcaption class="figure-caption">Radical Theory of Aging</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>mitochondrial free radical theory of aging</strong> proposes that aging and age-related diseases are, at least in part, driven by the cumulative damage caused by reactive oxygen species (ROS) generated by mitochondria during cellular respiration. Here’s an overview of the theory:</p>
<ol type="1">
<li><p><strong>ROS Production by Mitochondria:</strong> Mitochondria are the primary producers of ROS within cells. During oxidative phosphorylation, electrons leaking from the electron transport chain (ETC) can react with molecular oxygen (O2), resulting in the formation of superoxide radicals (O2•−). Superoxide can then be converted into other ROS, including hydrogen peroxide (H2O2) and hydroxyl radicals (•OH), through various enzymatic and non-enzymatic reactions.</p></li>
<li><p><strong>Damage to Cellular Components:</strong> ROS generated by mitochondria can react with and damage cellular components, including lipids, proteins, and DNA. This oxidative damage can lead to disruptions in cellular homeostasis, impaired cellular function, and ultimately contribute to the aging process. For example, ROS-induced lipid peroxidation can lead to membrane damage, protein oxidation can impair enzymatic function, and DNA damage can result in mutations and cellular dysfunction.</p></li>
<li><p><strong>Mitochondrial Dysfunction:</strong> Over time, the accumulation of oxidative damage to mitochondrial components can lead to mitochondrial dysfunction, including impaired ATP production, altered mitochondrial membrane potential, and increased production of ROS. This creates a vicious cycle whereby dysfunctional mitochondria produce even more ROS, exacerbating oxidative damage and further compromising cellular function.</p></li>
<li><p><strong>Age-Related Diseases:</strong> Mitochondrial ROS and oxidative stress have been implicated in the pathogenesis of various age-related diseases, including neurodegenerative disorders (e.g., Alzheimer’s disease, Parkinson’s disease), cardiovascular diseases, metabolic disorders, and cancer. Oxidative damage to cellular components contributes to the development and progression of these diseases, highlighting the role of mitochondrial dysfunction in aging-related pathologies.</p></li>
<li><p><strong>Antioxidant Defenses and Longevity:</strong> While ROS are inherently reactive and damaging, cells possess antioxidant defense mechanisms to neutralize and mitigate their effects. These include enzymatic antioxidants (e.g., superoxide dismutase, catalase, glutathione peroxidase) and non-enzymatic antioxidants (e.g., vitamin C, vitamin E, glutathione). Enhanced antioxidant defenses have been associated with increased longevity and reduced age-related pathologies in various model organisms.</p></li>
</ol>
</section>
<section id="superoxide-dismutase" class="level3" data-number="5.2.2">
<h3 data-number="5.2.2" class="anchored" data-anchor-id="superoxide-dismutase"><span class="header-section-number">5.2.2</span> Superoxide Dismutase</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/dismutase.png" class="img-fluid figure-img" width="504"></p>
<p></p><figcaption class="figure-caption">Illustration of Superoxide Dismutase’s Function</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Superoxide dismutase (SOD) is an enzyme that catalyzes the conversion of superoxide radicals (O2•−) into hydrogen peroxide (H2O2) and molecular oxygen (O2). However, hydrogen peroxide itself is also a reactive oxygen species (ROS) that can cause cellular damage if not properly neutralized. To further detoxify hydrogen peroxide, cells utilize other enzymes such as catalase and glutathione peroxidase. Here’s how the conversion of hydrogen peroxide to water (H2O) occurs:</p>
<ol type="1">
<li><p><strong>Catalase:</strong> Catalase is an enzyme found in peroxisomes and certain other cellular compartments. It catalyzes the decomposition of hydrogen peroxide into water and molecular oxygen, as follows:</p>
<p><span class="math inline">\(2H_2O_2 \rightarrow 2H_2O + O_2\)</span></p>
<p>Catalase rapidly converts hydrogen peroxide into water and oxygen, thereby neutralizing its harmful effects on cells.</p></li>
<li><p><strong>Glutathione Peroxidase:</strong> Glutathione peroxidase is another enzyme involved in the detoxification of hydrogen peroxide. It utilizes reduced glutathione (GSH) as a cofactor to reduce hydrogen peroxide to water while oxidizing glutathione to its disulfide form. The reaction is as follows:</p>
<p><span class="math inline">\(2GSH + H_2O_2 \rightarrow GSSG + 2H_2O\)</span></p>
<p>In this reaction, glutathione peroxidase reduces two molecules of hydrogen peroxide to two molecules of water while simultaneously oxidizing two molecules of glutathione.</p></li>
</ol>
</section>
</section>
<section id="mitochondria-and-apoptosis" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="mitochondria-and-apoptosis"><span class="header-section-number">5.3</span> Mitochondria and Apoptosis</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/apoptosis_necrosis.png" class="img-fluid figure-img" width="402"></p>
<p></p><figcaption class="figure-caption">Differences Between Apoptosis and Necrosis</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Apoptosis</strong>, often referred to as programmed cell death, is a highly regulated process that occurs in multicellular organisms. It is characterized by a series of biochemical events that lead to the controlled and orderly dismantling of cells. Apoptosis is initiated by either extrinsic (death receptor-mediated) or intrinsic (mitochondrial-mediated) pathways. In the intrinsic pathway, various stress signals trigger the release of cytochrome c from mitochondria, which activates caspase enzymes, leading to cell death. In the extrinsic pathway, binding of death ligands to death receptors on the cell surface activates caspase enzymes directly.</p>
<p>Apoptosis exhibits specific characteristics, including cell shrinkage and condensation of chromatin, as well as the formation of apoptotic bodies, which are small membrane-bound vesicles containing cellular contents. These apoptotic bodies are subsequently phagocytosed by neighboring cells or professional phagocytes, such as macrophages. In terms of its physiological role, apoptosis plays essential roles in various processes, including tissue development, immune system regulation, and the elimination of damaged or infected cells. It helps maintain tissue homeostasis and contributes to overall organismal health.</p>
<p><strong>Necrosis</strong>, in contrast to apoptosis, is a form of cell death characterized by rapid and uncontrolled cellular degradation, often resulting from acute injury, ischemia, or toxic insults. Unlike apoptosis, necrosis does not follow a programmed pathway and is associated with inflammation and tissue damage. Necrosis typically involves cellular swelling, organelle dysfunction, and rupture of the plasma membrane, leading to the release of cellular contents into the extracellular space.</p>
<p>Necrosis is not a regulated process and can occur as a result of various insults, such as physical trauma, infection, or exposure to toxins. The inflammatory response triggered by necrotic cell death serves to remove the damaged tissue and initiate repair processes. However, excessive or prolonged necrosis can contribute to tissue damage and exacerbate pathological conditions. Unlike apoptosis, necrosis is generally considered detrimental to organismal health and is often associated with pathological states, including ischemic injury, neurodegenerative diseases, and cancer.</p>
<section id="caspase-activation" class="level3" data-number="5.3.1">
<h3 data-number="5.3.1" class="anchored" data-anchor-id="caspase-activation"><span class="header-section-number">5.3.1</span> Caspase Activation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/caspase_activation.png" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Caspase Activation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>During apoptosis, <strong>caspases</strong>, which are a family of cysteine–aspartic protease enzymes, play a central role in orchestrating the dismantling of the cell. The process begins with the initiation of apoptosis by various signals, such as DNA damage, cellular stress, or signaling molecules from neighboring cells. These signals trigger the activation of initiator caspases, such as caspase 8 and caspase 9.</p>
<p>Initiator caspases exist in an inactive form within the cell until they are activated by apoptotic signals. Once activated, caspase 8 is involved in the extrinsic pathway of apoptosis, while caspase 9 is involved in the intrinsic pathway. Both pathways ultimately converge to activate downstream effector caspases, including caspase 3, caspase 6, and caspase 7.</p>
<p>Activated initiator caspases cleave and activate effector caspases, leading to the amplification of the apoptotic signal and the execution of cell death. Effector caspases, in turn, cleave multiple cellular substrates, including structural proteins, DNA repair enzymes, and signaling molecules, ultimately leading to the characteristic morphological and biochemical changes associated with apoptosis.</p>
</section>
<section id="phagocytosis-of-apoptotic-cells" class="level3" data-number="5.3.2">
<h3 data-number="5.3.2" class="anchored" data-anchor-id="phagocytosis-of-apoptotic-cells"><span class="header-section-number">5.3.2</span> Phagocytosis of Apoptotic Cells</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/phagocytosis.png" class="img-fluid figure-img" width="346"></p>
<p></p><figcaption class="figure-caption">Phagocytosis of Apoptotic Cells</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Phagocytosis of apoptotic cells is a vital process in the clearance of dying cells and the maintenance of tissue homeostasis. When cells undergo apoptosis, they display specific signals on their surface that mark them for recognition and engulfment by phagocytes, such as macrophages and dendritic cells. Here’s how phagocytosis of apoptotic cells occurs:</p>
<ol type="1">
<li><p><strong>Recognition of Apoptotic Cells:</strong> During apoptosis, dying cells undergo changes in their membrane composition and structure. This includes the exposure of “eat-me” signals, such as phosphatidylserine (PS), on the outer leaflet of the plasma membrane. These signals serve as ligands for receptors on phagocytic cells, facilitating the recognition and binding of apoptotic cells by phagocytes.</p></li>
<li><p><strong>Engulfment by Phagocytes:</strong> Once recognized, apoptotic cells are engulfed by phagocytes through a process known as phagocytosis. This involves the extension of pseudopodia around the apoptotic cell, leading to its internalization within a phagocytic vesicle called a phagosome. The phagosome then undergoes maturation, fusing with lysosomes to form a phagolysosome, where the engulfed apoptotic cell is degraded.</p></li>
<li><p><strong>Anti-inflammatory Response:</strong> Phagocytosis of apoptotic cells typically triggers an anti-inflammatory response in the phagocytic cell. This is mediated by the release of anti-inflammatory cytokines, such as transforming growth factor-beta (TGF-β), and the inhibition of pro-inflammatory cytokines, such as tumor necrosis factor-alpha (TNF-α) and interleukin-1 beta (IL-1β). This anti-inflammatory response helps to resolve inflammation and prevent tissue damage.</p></li>
<li><p><strong>Processing and Clearance:</strong> Within the phagolysosome, the engulfed apoptotic cell is broken down by lysosomal enzymes, leading to the degradation of cellular components into their constituent molecules. These molecules, including amino acids, nucleotides, and lipids, can be recycled by the phagocyte or released back into the extracellular environment. Clearance of apoptotic cells by phagocytosis helps to maintain tissue homeostasis and prevent the release of potentially harmful cellular contents.</p></li>
</ol>
</section>
<section id="pathways-of-apoptosis" class="level3" data-number="5.3.3">
<h3 data-number="5.3.3" class="anchored" data-anchor-id="pathways-of-apoptosis"><span class="header-section-number">5.3.3</span> Pathways of Apoptosis</h3>
<section id="extrinsic" class="level4" data-number="5.3.3.1">
<h4 data-number="5.3.3.1" class="anchored" data-anchor-id="extrinsic"><span class="header-section-number">5.3.3.1</span> Extrinsic</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/extrinsic_pathway.png" class="img-fluid figure-img" width="292"></p>
<p></p><figcaption class="figure-caption">Extrinsic Pathway of Apoptosis</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>extrinsic pathway of apoptosis</strong>, also known as the death receptor pathway, is one of the two main pathways by which apoptosis is initiated in multicellular organisms. This pathway is triggered by extracellular signals that bind to death receptors on the cell surface, leading to the activation of caspase enzymes and ultimately resulting in cell death. Here’s an overview of the extrinsic pathway:</p>
<ol type="1">
<li><p><strong>Initiation:</strong> The extrinsic pathway is initiated by the binding of extracellular death ligands, such as Fas ligand (FasL) or tumor necrosis factor-alpha (TNF-α), to specific death receptors on the cell surface. These death receptors belong to the tumor necrosis factor receptor (TNFR) superfamily and include Fas (CD95/APO-1), TNF receptor 1 (TNFR1), and TNF-related apoptosis-inducing ligand receptor (TRAIL-R).</p></li>
<li><p><strong>Formation of Death-Inducing Signaling Complex (DISC):</strong> Binding of the death ligand to its receptor induces conformational changes in the receptor, leading to the recruitment and clustering of adaptor proteins, such as Fas-associated death domain (FADD) and TNF receptor-associated death domain (TRADD), to the cytoplasmic domain of the receptor. These adaptor proteins then recruit procaspase-8 or procaspase-10 to form a multiprotein complex called the Death-Inducing Signaling Complex (DISC) at the cytoplasmic face of the plasma membrane.</p></li>
<li><p><strong>Activation of Caspases:</strong> Within the DISC, procaspase-8 or procaspase-10 molecules are brought into close proximity, facilitating their autoactivation by cleavage. Activated caspase-8 or caspase-10 then cleaves and activates downstream effector caspases, such as caspase-3, caspase-6, and caspase-7, initiating the execution phase of apoptosis.</p></li>
<li><p><strong>Execution of Apoptosis:</strong> Once activated, effector caspases cleave various cellular substrates, leading to characteristic morphological and biochemical changes associated with apoptosis, including DNA fragmentation, cytoskeletal reorganization, membrane blebbing, and formation of apoptotic bodies. These changes ultimately result in the dismantling of the cell and its phagocytic removal by neighboring cells or professional phagocytes.</p></li>
</ol>
</section>
<section id="intrinsic" class="level4" data-number="5.3.3.2">
<h4 data-number="5.3.3.2" class="anchored" data-anchor-id="intrinsic"><span class="header-section-number">5.3.3.2</span> Intrinsic</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/intrinsic.png" class="img-fluid figure-img" width="516"></p>
<p></p><figcaption class="figure-caption">Intrinsic Pathway of Apoptosis</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>intrinsic pathway of apoptosis</strong>, also known as the mitochondrial pathway, is a crucial mechanism by which cells initiate programmed cell death in response to internal stress signals, such as DNA damage, cellular injury, or nutrient deprivation. Unlike the extrinsic pathway, which is initiated by external death ligands binding to cell surface death receptors, the intrinsic pathway involves mitochondrial signaling events that lead to the release of pro-apoptotic proteins and the activation of caspase enzymes. Here’s an overview of the intrinsic pathway:</p>
<ol type="1">
<li><p><strong>Mitochondrial Outer Membrane Permeabilization (MOMP):</strong> The intrinsic pathway is initiated by various intracellular stress signals, such as DNA damage or oxidative stress, which lead to the activation of pro-apoptotic proteins, including Bcl-2-associated X protein (BAX) and Bcl-2-antagonist/killer (BAK). These proteins undergo conformational changes and oligomerize on the outer mitochondrial membrane, leading to mitochondrial outer membrane permeabilization (MOMP).</p></li>
<li><p><strong>Release of Pro-Apoptotic Proteins:</strong> MOMP results in the release of pro-apoptotic proteins from the intermembrane space of the mitochondria into the cytoplasm. These proteins include cytochrome c, Smac/DIABLO (Second mitochondria-derived activator of caspases/direct IAP-binding protein with low pI), and apoptosis-inducing factor (AIF). Cytochrome c binds to apoptotic protease-activating factor 1 (Apaf-1), forming the apoptosome complex, which activates caspase-9, an initiator caspase.</p></li>
<li><p><strong>Activation of Caspases:</strong> Activation of caspase-9 within the apoptosome complex leads to the activation of downstream effector caspases, such as caspase-3, caspase-6, and caspase-7. These effector caspases orchestrate the execution phase of apoptosis by cleaving various cellular substrates, leading to the characteristic morphological and biochemical changes associated with cell death.</p></li>
<li><p><strong>Regulation by Bcl-2 Family Proteins:</strong> The intrinsic pathway of apoptosis is tightly regulated by members of the Bcl-2 family of proteins, which include both pro-apoptotic (e.g., BAX, BAK, Bid) and anti-apoptotic (e.g., Bcl-2, Bcl-xL) members. The balance between pro-apoptotic and anti-apoptotic Bcl-2 family members determines the susceptibility of cells to undergo apoptosis in response to intracellular stress signals.</p></li>
<li><p><strong>Physiological and Pathological Roles:</strong> The intrinsic pathway of apoptosis plays essential roles in various physiological processes, including tissue development, immune system regulation, and the elimination of damaged or infected cells. Dysregulation of this pathway can contribute to various diseases, including cancer, neurodegenerative disorders, and autoimmune diseases.</p></li>
</ol>
</section>
</section>
<section id="bak-bax-and-cytochrome-c-release" class="level3" data-number="5.3.4">
<h3 data-number="5.3.4" class="anchored" data-anchor-id="bak-bax-and-cytochrome-c-release"><span class="header-section-number">5.3.4</span> Bak, Bax, and Cytochrome C Release</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/bak_and_bax.png" class="img-fluid figure-img" width="452"></p>
<p></p><figcaption class="figure-caption">Bak and Bax Action</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Bax</strong> and <strong>Bak</strong> are pro-apoptotic members of the Bcl-2 protein family that play a central role in initiating the intrinsic pathway of apoptosis. When cells undergo stress or receive apoptotic signals, inactive monomeric forms of Bax and Bak undergo a conformational change, leading to their activation and oligomerization. Here’s how the process unfolds:</p>
<ol type="1">
<li><p><strong>Inactive Bax and Bak:</strong> Normally, Bax and Bak exist in an inactive monomeric state within the cytoplasm or associated with intracellular membranes, such as the outer mitochondrial membrane (OMM). In this state, they are unable to induce apoptosis.</p></li>
<li><p><strong>Oligomerization of Bax and Bak:</strong> Upon receiving apoptotic signals, such as DNA damage or cellular stress, Bax and Bak undergo a conformational change that exposes their hydrophobic domains. This change allows multiple Bax and Bak molecules to interact and form oligomeric complexes on the OMM.</p></li>
<li><p><strong>Permeabilization of the Outer Mitochondrial Membrane (MOMP):</strong> The oligomerization of Bax and Bak leads to the permeabilization of the outer mitochondrial membrane (MOMP). This process allows the release of pro-apoptotic proteins, including cytochrome c, from the intermembrane space of the mitochondria into the cytoplasm.</p></li>
<li><p><strong>Cytochrome c Release:</strong> Cytochrome c is a key protein involved in the intrinsic pathway of apoptosis. Once released into the cytoplasm following MOMP, cytochrome c binds to apoptotic protease-activating factor 1 (Apaf-1), leading to the formation of the apoptosome complex.</p></li>
<li><p><strong>Activation of Caspases:</strong> The apoptosome complex activates caspase-9, an initiator caspase, which in turn activates downstream effector caspases, such as caspase-3, caspase-6, and caspase-7. These effector caspases execute the apoptotic program by cleaving various cellular substrates, ultimately leading to cell death.</p></li>
</ol>
</section>
<section id="inactivating-bak-and-bax-with-bcl-2" class="level3" data-number="5.3.5">
<h3 data-number="5.3.5" class="anchored" data-anchor-id="inactivating-bak-and-bax-with-bcl-2"><span class="header-section-number">5.3.5</span> Inactivating Bak and Bax with Bcl-2</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week5/inactivation.png" class="img-fluid figure-img" width="432"></p>
<p></p><figcaption class="figure-caption">Inactivation of Bak and Bax</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Bcl-2 plays a crucial role in preventing apoptosis by inhibiting the activation of Bax and Bak. Here’s how Bcl-2 inhibits Bax and Bak:</p>
<ol type="1">
<li><p><strong>Binding and Sequestration:</strong> Bcl-2 can directly bind to Bax and Bak, preventing their activation and oligomerization. By sequestering Bax and Bak in an inactive state, Bcl-2 inhibits their ability to permeabilize the outer mitochondrial membrane (MOMP) and release pro-apoptotic proteins such as cytochrome c.</p></li>
<li><p><strong>Maintenance of Mitochondrial Integrity:</strong> Bcl-2 also helps maintain mitochondrial integrity by stabilizing the outer mitochondrial membrane and preventing its permeabilization. This stabilization further contributes to the inhibition of Bax and Bak activation and the prevention of cytochrome c release.</p></li>
<li><p><strong>Inhibition of Apoptotic Signaling:</strong> By inhibiting Bax and Bak, Bcl-2 blocks the initiation of the intrinsic pathway of apoptosis. This prevents the downstream activation of caspases and the execution of the apoptotic program. As a result, cells are protected from undergoing apoptosis even in the presence of apoptotic stimuli.</p></li>
<li><p><strong>Regulation of Cell Fate:</strong> The balance between pro-apoptotic proteins like Bax and Bak and anti-apoptotic proteins like Bcl-2 determines the fate of the cell. In healthy cells, the presence of Bcl-2 maintains this balance, promoting cell survival. However, under conditions of cellular stress or damage, the balance may shift towards pro-apoptotic signaling, leading to the activation of Bax and Bak and the initiation of apoptosis.</p></li>
</ol>
</section>
<section id="bh3-only-proteins" class="level3" data-number="5.3.6">
<h3 data-number="5.3.6" class="anchored" data-anchor-id="bh3-only-proteins"><span class="header-section-number">5.3.6</span> BH3-Only Proteins</h3>
<p><strong>BH3-only proteins</strong> play a crucial role in regulating apoptosis by modulating the activity of Bcl-2 family proteins. These proteins are so named because they contain only the BH3 (Bcl-2 homology 3) domain, which is essential for their pro-apoptotic function. Here’s how BH3-only proteins function in the intrinsic pathway of apoptosis:</p>
<ol type="1">
<li><p><strong>Production or Activation of BH3-only Protein:</strong> BH3-only proteins can be produced in response to various cellular stress signals, such as DNA damage, nutrient deprivation, or growth factor withdrawal. Alternatively, existing BH3-only proteins may become activated through post-translational modifications or changes in cellular localization. Once produced or activated, BH3-only proteins serve as sensors of cellular stress and initiate apoptotic signaling pathways.</p></li>
<li><p><strong>Inactivation of Bcl-2:</strong> BH3-only proteins exert their pro-apoptotic effects by binding to and inhibiting the anti-apoptotic members of the Bcl-2 family, such as Bcl-2 itself. By binding to Bcl-2, BH3-only proteins disrupt its anti-apoptotic function, thereby promoting apoptosis. This frees up pro-apoptotic proteins like Bax and Bak to initiate the intrinsic pathway of apoptosis.</p></li>
<li><p><strong>Bax, Bak Activation (Oligomerization):</strong> In the absence of inhibition by anti-apoptotic Bcl-2 proteins, pro-apoptotic proteins like Bax and Bak become activated and undergo oligomerization. Oligomerization refers to the formation of multimeric complexes of Bax and Bak proteins on the outer mitochondrial membrane. These oligomeric complexes lead to mitochondrial outer membrane permeabilization (MOMP), resulting in the release of pro-apoptotic proteins such as cytochrome c into the cytoplasm.</p></li>
</ol>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week4.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week6.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>