<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 2&nbsp; Ubiquitin Proteasome System</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week3.html" rel="next">
<link href="../chapters/week1.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Alzheimer’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Parkinson’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Secondary Metabolism in Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Plants as Medicine</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#e3-ligase-and-ubiquitylation" id="toc-e3-ligase-and-ubiquitylation" class="nav-link active" data-scroll-target="#e3-ligase-and-ubiquitylation"><span class="toc-section-number">2.1</span>  E3 Ligase and Ubiquitylation</a>
  <ul class="collapse">
  <li><a href="#terminal-regions-of-ubiquitin" id="toc-terminal-regions-of-ubiquitin" class="nav-link" data-scroll-target="#terminal-regions-of-ubiquitin"><span class="toc-section-number">2.1.1</span>  Terminal Regions of Ubiquitin</a></li>
  <li><a href="#ring-and-hect-e3-ligases" id="toc-ring-and-hect-e3-ligases" class="nav-link" data-scroll-target="#ring-and-hect-e3-ligases"><span class="toc-section-number">2.1.2</span>  RING and HECT E3 Ligases</a></li>
  <li><a href="#marking-proteins-by-ubiquitin" id="toc-marking-proteins-by-ubiquitin" class="nav-link" data-scroll-target="#marking-proteins-by-ubiquitin"><span class="toc-section-number">2.1.3</span>  Marking Proteins by Ubiquitin</a></li>
  </ul></li>
  <li><a href="#n-end-rule-pathway" id="toc-n-end-rule-pathway" class="nav-link" data-scroll-target="#n-end-rule-pathway"><span class="toc-section-number">2.2</span>  N-End Rule Pathway</a>
  <ul class="collapse">
  <li><a href="#what-is-the-rule" id="toc-what-is-the-rule" class="nav-link" data-scroll-target="#what-is-the-rule"><span class="toc-section-number">2.2.1</span>  What is the Rule?</a></li>
  <li><a href="#substrates-produced" id="toc-substrates-produced" class="nav-link" data-scroll-target="#substrates-produced"><span class="toc-section-number">2.2.2</span>  Substrates Produced</a></li>
  </ul></li>
  <li><a href="#protein-quality-control-by-ups" id="toc-protein-quality-control-by-ups" class="nav-link" data-scroll-target="#protein-quality-control-by-ups"><span class="toc-section-number">2.3</span>  Protein Quality Control by UPS</a>
  <ul class="collapse">
  <li><a href="#protein-secretory-pathway" id="toc-protein-secretory-pathway" class="nav-link" data-scroll-target="#protein-secretory-pathway"><span class="toc-section-number">2.3.1</span>  Protein Secretory Pathway</a></li>
  <li><a href="#er-associated-degradation" id="toc-er-associated-degradation" class="nav-link" data-scroll-target="#er-associated-degradation"><span class="toc-section-number">2.3.2</span>  ER-Associated Degradation</a></li>
  <li><a href="#cdc48-p97-function-in-ubs" id="toc-cdc48-p97-function-in-ubs" class="nav-link" data-scroll-target="#cdc48-p97-function-in-ubs"><span class="toc-section-number">2.3.3</span>  Cdc48 / p97 Function in UBS</a></li>
  </ul></li>
  <li><a href="#degradation-by-the-proteasome" id="toc-degradation-by-the-proteasome" class="nav-link" data-scroll-target="#degradation-by-the-proteasome"><span class="toc-section-number">2.4</span>  Degradation by the Proteasome</a>
  <ul class="collapse">
  <li><a href="#multiple-myeloma-and-proteasome-inhibitor" id="toc-multiple-myeloma-and-proteasome-inhibitor" class="nav-link" data-scroll-target="#multiple-myeloma-and-proteasome-inhibitor"><span class="toc-section-number">2.4.1</span>  Multiple Myeloma and Proteasome Inhibitor</a></li>
  <li><a href="#aneuploidy-and-toxic-stress" id="toc-aneuploidy-and-toxic-stress" class="nav-link" data-scroll-target="#aneuploidy-and-toxic-stress"><span class="toc-section-number">2.4.2</span>  Aneuploidy and Toxic Stress</a></li>
  <li><a href="#problems-with-proteasomal-degradation" id="toc-problems-with-proteasomal-degradation" class="nav-link" data-scroll-target="#problems-with-proteasomal-degradation"><span class="toc-section-number">2.4.3</span>  Problems with Proteasomal Degradation</a></li>
  </ul></li>
  <li><a href="#ups-and-neurodegeneration" id="toc-ups-and-neurodegeneration" class="nav-link" data-scroll-target="#ups-and-neurodegeneration"><span class="toc-section-number">2.5</span>  UPS and Neurodegeneration</a></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week2.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/ups_roles.png" class="img-fluid figure-img" width="306"></p>
<p></p><figcaption class="figure-caption">Roles of the Ubiquitin Protein in the Cell</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Although ubiquitin is generally used to target proteins for degradation via the E3 ubiquitin ligase system, it also has roles in other cellular processes - for instance, the cell cycle, quality control, and metabolism.</p>
<section id="e3-ligase-and-ubiquitylation" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="e3-ligase-and-ubiquitylation"><span class="header-section-number">2.1</span> E3 Ligase and Ubiquitylation</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/cascade.png" class="img-fluid figure-img" width="125"></p>
<p></p><figcaption class="figure-caption">E1, E2, and E3 Ubiquitination Cascade</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>This process is threefold and has the following steps - it is usually used to tag proteins for degradation within the cell and is important for maintaining homeostasis, cellular processes, and also protein levels:</p>
<ol type="1">
<li><p><strong>E1 Activation</strong>: The process begins with an enzyme called E1 (ubiquitin-activating enzyme). E1 activates <strong>ubiquitin</strong>, a small protein found in cells that serves as a tag for proteins destined for degradation. Activation involves the binding of ubiquitin to E1 in an ATP-dependent manner, forming a thioester bond between the ubiquitin molecule and E1.</p></li>
<li><p><strong>E2 Conjugation</strong>: Once activated, the ubiquitin molecule is transferred to a second enzyme called E2 (ubiquitin-conjugating enzyme). E2 works as a carrier for the ubiquitin molecule, shuttling it to the target protein.</p></li>
<li><p><strong>E3 Ligase</strong>: The E2 enzyme, now loaded with ubiquitin, interacts with a third enzyme called E3 ligase (ubiquitin-protein ligase). E3 ligase plays a crucial role in recognizing specific target proteins and facilitating the transfer of ubiquitin from E2 to the target protein.</p></li>
<li><p><strong>Ubiquitination</strong>: The E3 ligase, along with the assistance of E2, transfers the ubiquitin molecule from the E2 enzyme to a lysine residue on the target protein, forming an isopeptide bond. This process can be repeated multiple times, resulting in the addition of multiple ubiquitin molecules to the target protein, forming a polyubiquitin chain.</p></li>
<li><p><strong>Proteasomal Degradation</strong>: Once the target protein is tagged with ubiquitin chains, it is recognized and degraded by a large multi-subunit complex called the <strong>proteasome</strong>. The proteasome acts as the cell’s “garbage disposal,” breaking down tagged proteins into smaller peptides that can be recycled by the cell.</p></li>
</ol>
<section id="terminal-regions-of-ubiquitin" class="level3" data-number="2.1.1">
<h3 data-number="2.1.1" class="anchored" data-anchor-id="terminal-regions-of-ubiquitin"><span class="header-section-number">2.1.1</span> Terminal Regions of Ubiquitin</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/terminus.png" class="img-fluid figure-img" width="500"></p>
<p></p><figcaption class="figure-caption">Cys, Lys, and C-Termini of Ubiquitin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Ubiquitin is a small protein found in cells that plays a crucial role in tagging other proteins for degradation through the ubiquitin-proteasome system. Its structure consists of 76 amino acid residues and includes several key functional elements:</p>
<ol type="1">
<li><p><strong>Cysteine (Cys)</strong>: Ubiquitin contains a reactive thiol group (-SH) on its C-terminal glycine residue. This cysteine residue is where ubiquitin forms a thioester bond with the active site of the E1 ubiquitin-activating enzyme during the first step of the ubiquitination cascade. This thioester bond formation activates ubiquitin for conjugation to target proteins.</p></li>
<li><p><strong>Lysine (Lys)</strong>: Ubiquitin contains seven lysine residues (K6, K11, K27, K29, K33, K48, and K63), any of which can serve as attachment sites for additional ubiquitin molecules during the process of polyubiquitination. The type of polyubiquitin chain formed (e.g., K48-linked, K63-linked) determines the fate of the tagged protein, such as degradation by the proteasome or involvement in other cellular processes.</p></li>
<li><p><strong>C-Terminus</strong>: The C-terminus of ubiquitin is where its Gly76 residue is located. This glycine residue is crucial for the formation of the isopeptide bond between ubiquitin and the lysine residue of the target protein during ubiquitination. The C-terminus is also where the ubiquitin molecule is initially activated by the E1 enzyme, forming a thioester bond with the cysteine residue of E1.</p></li>
</ol>
</section>
<section id="ring-and-hect-e3-ligases" class="level3" data-number="2.1.2">
<h3 data-number="2.1.2" class="anchored" data-anchor-id="ring-and-hect-e3-ligases"><span class="header-section-number">2.1.2</span> RING and HECT E3 Ligases</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/ring_and_hect.png" class="img-fluid figure-img" width="382"></p>
<p></p><figcaption class="figure-caption">Illustration of RING and HECT Ligases</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Two primary classes of E3 ligases, known as RING (Really Interesting New Gene) ligases and HECT (Homologous to the E6-AP Carboxyl Terminus) ligases, operate distinctly within this ubiquitination process.</p>
<p>Firstly, <strong>RING ligases</strong> function by directly mediating the transfer of ubiquitin from the E2 enzyme to the target protein without forming a covalent intermediate with ubiquitin. Although RING ligases lack intrinsic catalytic activity for ubiquitin transfer, they serve as scaffolds, bringing the E2 enzyme and the target protein into close proximity to facilitate efficient ubiquitin transfer. The RING domain, a characteristic feature of RING ligases, orchestrates the interaction between the E2 enzyme and the target protein, thereby enabling the ubiquitination process. Examples of RING ligases include Cullin-RING ligases (CRLs), which participate in the ubiquitination of diverse cellular proteins involved in crucial processes like cell cycle regulation and signal transduction.</p>
<p>On the other hand, <strong>HECT ligases</strong> operate differently by forming a thioester intermediate with ubiquitin prior to transferring it to the target protein. The HECT domain, distinctive to HECT ligases, contains a catalytic cysteine residue that forms a thioester bond with ubiquitin, facilitating its transfer to the target protein. Typically, HECT ligases exhibit a specific domain architecture, featuring an N-terminal domain responsible for substrate recognition and a C-terminal HECT domain accountable for ubiquitin transfer. Notable examples of HECT ligases include E6-AP (E6-associated protein) and NEDD4 (Neural Precursor Cell Expressed, Developmentally Down-Regulated 4), which respectively target specific proteins for ubiquitination and regulate the ubiquitination of membrane proteins and signaling molecules.</p>
<section id="cullin-ring-ligases" class="level4" data-number="2.1.2.1">
<h4 data-number="2.1.2.1" class="anchored" data-anchor-id="cullin-ring-ligases"><span class="header-section-number">2.1.2.1</span> Cullin-RING Ligases</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/crls.png" class="img-fluid figure-img" width="518"></p>
<p></p><figcaption class="figure-caption">Cullin-Ring Ligase Mechanisms</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>Cullin-RING ubiquitin ligase (CRL)</strong> is a type of RING E3 ubiquitin ligase that plays a crucial role in the ubiquitination and subsequent degradation of specific target proteins within the cell. CRLs are part of a large family of multisubunit E3 ligases, and they are involved in the regulation of various cellular processes, including cell cycle progression, signal transduction, and response to environmental stimuli. Some key features of CRLs include:</p>
<ol type="1">
<li><p><strong>Composition</strong>: CRLs are modular complexes composed of multiple subunits. The core components include a cullin protein, a RING finger protein, an adaptor protein, and a substrate recognition module. The cullin protein acts as a scaffold, providing a platform for the assembly of the other components.</p></li>
<li><p><strong>Cullin Family Members</strong>: Cullin proteins are a family of scaffold proteins, with Cul1, Cul2, Cul3, Cul4, Cul5, and Cul7 being the most well-known members. Each cullin family member associates with specific adaptor proteins and substrate recognition modules, conferring specificity to the CRL complex.</p></li>
<li><p><strong>RING Finger Protein</strong>: The RING finger protein, often referred to as Rbx1 or Rbx2, is responsible for binding to the E2 ubiquitin-conjugating enzyme and facilitating the transfer of ubiquitin from the E2 enzyme to the target protein.</p></li>
<li><p><strong>Adaptor Protein</strong>: The adaptor protein (e.g., Skp1, Elongin C) connects the cullin scaffold to the substrate recognition module, ensuring proper substrate recruitment.</p></li>
<li><p><strong>Substrate Recognition Module</strong>: This module is responsible for recognizing and binding to specific target proteins marked for ubiquitination. Different CRLs have distinct substrate recognition modules, contributing to the specificity of ubiquitination events.</p></li>
<li><p><strong>Ubiquitin Transfer</strong>: Once assembled, the CRL complex catalyzes the transfer of ubiquitin molecules from the E2 enzyme to the lysine residues on the target protein, leading to the formation of a polyubiquitin chain. This ubiquitin chain serves as a signal for the targeted protein to be recognized and degraded by the proteasome.</p></li>
<li><p><strong>Regulation of CRL Activity</strong>: The activity of CRLs is tightly regulated. Post-translational modifications, such as neddylation (attachment of the small ubiquitin-like modifier NEDD8 to cullin), are essential for CRL activation. Additionally, CRLs can be regulated by specific inhibitors and adaptors.</p></li>
<li><p><strong>Cellular Functions</strong>: Due to their involvement in the ubiquitin-proteasome system, CRLs play critical roles in maintaining cellular homeostasis by controlling the levels of key regulatory proteins. Dysregulation of CRLs has been implicated in various diseases, including cancer.</p></li>
</ol>
</section>
</section>
<section id="marking-proteins-by-ubiquitin" class="level3" data-number="2.1.3">
<h3 data-number="2.1.3" class="anchored" data-anchor-id="marking-proteins-by-ubiquitin"><span class="header-section-number">2.1.3</span> Marking Proteins by Ubiquitin</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/marking_proteins.png" class="img-fluid figure-img" width="515"></p>
<p></p><figcaption class="figure-caption">Protein Marking by Ubiquitin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The marking of proteins by ubiquitin is a crucial process in cells, ensuring proper protein turnover and regulation. Ubiquitin is a small protein consisting of 76 amino acids. Its structure contains seven lysine residues (K6, K11, K27, K29, K33, K48, and K63) and an N-terminal methionine residue; each possible linkage leads to different cellular reactions.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/linkage_types.png" class="img-fluid figure-img" width="497"></p>
<p></p><figcaption class="figure-caption">Different Linkage Types and Functions of Ubiquitin</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/various_signals.png" class="img-fluid figure-img" width="424"></p>
<p></p><figcaption class="figure-caption">Different Linkage Types and Functions of Ubiquitin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Ubiquitin can attach to target proteins through its lysine residues or its N-terminal methionine residue, forming different types of ubiquitin chains. The most well-characterized ubiquitin chain types are formed through the linkage of ubiquitin molecules via one of its seven lysine residues or through the N-terminal methionine residue. These different linkages lead to diverse cellular outcomes:</p>
<ol type="1">
<li><strong>Linear Chains</strong>: Ubiquitin can also form linear chains where the C-terminal glycine of one ubiquitin molecule is linked to the N-terminal methionine of the next ubiquitin molecule. Linear chains are involved in various signaling pathways, including NF-κB activation and apoptosis.</li>
</ol>
</section>
</section>
<section id="n-end-rule-pathway" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="n-end-rule-pathway"><span class="header-section-number">2.2</span> N-End Rule Pathway</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/deubiquitinase.png" class="img-fluid figure-img" width="432"></p>
<p></p><figcaption class="figure-caption">Various Roles of Deubiquitinase</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Deubiquitinases (DUBs)</strong> are enzymes that play a crucial role in the regulation of ubiquitin-mediated signaling pathways by removing ubiquitin molecules from proteins. While ubiquitination marks proteins for degradation or alters their function, deubiquitination mediated by DUBs reverses these modifications, allowing for precise control over protein stability, localization, and activity within the cell. They play some of the following roles in cells:</p>
<ol type="1">
<li><p><strong>Regulation of Protein Stability</strong>: DUBs counteract the activity of ubiquitin ligases by removing ubiquitin chains from proteins targeted for degradation. By deubiquitinating these proteins, DUBs can stabilize them and prevent their degradation by the proteasome. This process is crucial for maintaining the appropriate levels of key regulatory proteins involved in various cellular processes.</p></li>
<li><p><strong>Control of Signaling Pathways</strong>: Ubiquitination plays a central role in the regulation of signaling pathways involved in cell growth, differentiation, apoptosis, and immune responses. DUBs modulate these signaling pathways by removing ubiquitin chains from signaling molecules, thereby regulating their activity and localization. For example, DUBs can terminate signaling cascades by deubiquitinating and inactivating key signaling molecules.</p></li>
<li><p><strong>DNA Repair</strong>: DUBs are involved in the regulation of DNA repair pathways by controlling the ubiquitination status of proteins involved in DNA damage response and repair. By removing ubiquitin chains from DNA repair proteins, DUBs help maintain genomic integrity and prevent the accumulation of DNA damage.</p></li>
<li><p><strong>Endocytosis and Vesicular Trafficking</strong>: DUBs regulate the ubiquitination status of proteins involved in endocytosis, vesicular trafficking, and membrane protein turnover. By deubiquitinating membrane proteins, DUBs can control their internalization, recycling, and degradation, thereby regulating cellular processes such as receptor signaling and membrane protein turnover.</p></li>
<li><p><strong>Mitochondrial Dynamics</strong>: DUBs play a role in regulating mitochondrial dynamics and function by modulating the ubiquitination status of proteins involved in mitochondrial quality control, fusion, fission, and mitophagy (selective autophagy of damaged mitochondria).</p></li>
</ol>
<section id="what-is-the-rule" class="level3" data-number="2.2.1">
<h3 data-number="2.2.1" class="anchored" data-anchor-id="what-is-the-rule"><span class="header-section-number">2.2.1</span> What is the Rule?</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/n_end_rule.png" class="img-fluid figure-img" width="418"></p>
<p></p><figcaption class="figure-caption">Depiction of the N-End Rule Pathway</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>N-end rule pathway</strong> is a proteolytic system found in cells that targets proteins for degradation based on the identity of their N-terminal amino acid residue. This pathway was first proposed by Alexander Varshavsky in the late 1980s and has since been extensively studied in various organisms, including yeast, plants, and mammals. Key features of the N-end rule pathway include:</p>
<ol type="1">
<li><p><strong>Recognition of N-terminal Residues</strong>: The N-end rule pathway recognizes specific amino acids at the N-terminus of proteins. The identity of the N-terminal residue determines the fate of the protein. Certain amino acids, known as destabilizing residues, target proteins for degradation, while others are considered stabilizing residues.</p></li>
<li><p><strong>Degradation Signal</strong>: Proteins with destabilizing N-terminal residues are marked for degradation by the covalent attachment of ubiquitin molecules. This process involves the sequential action of enzymes known as N-terminal amino peptidases and E3 ubiquitin ligases, which recognize and ubiquitinate the target proteins, respectively.</p></li>
<li><p><strong>Recognition by the Proteasome</strong>: Ubiquitinated proteins are recognized and degraded by the proteasome, a large multi-subunit protease complex found in the cytoplasm and nucleus of eukaryotic cells. The proteasome cleaves ubiquitinated proteins into short peptides, which can then be further degraded into amino acids by cellular peptidases.</p></li>
<li><p><strong>Function in Protein Quality Control</strong>: The N-end rule pathway serves as an important mechanism for maintaining protein quality control in cells. It targets misfolded or damaged proteins for degradation, thereby preventing the accumulation of potentially harmful protein aggregates and promoting cellular homeostasis.</p></li>
<li><p><strong>Regulation of Cellular Processes</strong>: The N-end rule pathway regulates various cellular processes, including cell cycle progression, DNA repair, apoptosis, and response to environmental stresses. By controlling the stability of key regulatory proteins, the N-end rule pathway helps orchestrate cellular responses to internal and external cues.</p></li>
</ol>
<p>Examples of destabilizing N-terminal residues include methionine (Met) followed by certain amino acids such as arginine (Arg), lysine (Lys), phenylalanine (Phe), and tryptophan (Trp). These residues are recognized by specific components of the N-end rule pathway and targeted for degradation.</p>
</section>
<section id="substrates-produced" class="level3" data-number="2.2.2">
<h3 data-number="2.2.2" class="anchored" data-anchor-id="substrates-produced"><span class="header-section-number">2.2.2</span> Substrates Produced</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/physiological_substrates.png" class="img-fluid figure-img" width="473"></p>
<p></p><figcaption class="figure-caption">Physiological Substrates of the N-End Rule Pathway</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Here are some examples of physiological substrates of the N-end rule pathway:</p>
<ol type="1">
<li><p><strong>Transcription Factors</strong>: Certain transcription factors are targeted by the N-end rule pathway for degradation, regulating gene expression and cellular responses to environmental cues. For example, the transcription factor MATα2 in yeast and the transcription factor Pd in mammals are degraded by the N-end rule pathway.</p></li>
<li><p><strong>Cell Cycle Regulators</strong>: Proteins involved in cell cycle regulation, such as cyclins and cyclin-dependent kinase inhibitors (CDKIs), are substrates of the N-end rule pathway. Degradation of these proteins by the N-end rule pathway helps control cell cycle progression and ensures proper cell division.</p></li>
<li><p><strong>DNA Repair Proteins</strong>: Components of DNA repair pathways, including proteins involved in base excision repair, nucleotide excision repair, and homologous recombination, can be targeted by the N-end rule pathway. Degradation of damaged or misfolded DNA repair proteins helps maintain genomic integrity and prevents the accumulation of DNA lesions.</p></li>
<li><p><strong>Apoptotic Regulators</strong>: Proteins involved in apoptosis, or programmed cell death, are regulated by the N-end rule pathway. For example, certain pro-apoptotic proteins, such as Bid in mammals, are degraded by the N-end rule pathway, contributing to the control of apoptosis.</p></li>
<li><p><strong>Cell Signaling Proteins</strong>: Components of cell signaling pathways, including kinases, phosphatases, and signaling adapters, can be targeted by the N-end rule pathway. Degradation of signaling proteins by the N-end rule pathway helps regulate cellular responses to extracellular signals and maintain signaling homeostasis.</p></li>
<li><p><strong>Proteins with Misfolded or Damaged Domains</strong>: Proteins that become misfolded or damaged due to cellular stresses or mutations can be recognized and degraded by the N-end rule pathway. Degradation of misfolded or damaged proteins prevents the accumulation of toxic protein aggregates and promotes cellular health.</p></li>
</ol>
</section>
</section>
<section id="protein-quality-control-by-ups" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="protein-quality-control-by-ups"><span class="header-section-number">2.3</span> Protein Quality Control by UPS</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/cytosolic_protein_production.png" class="img-fluid figure-img" width="512"></p>
<p></p><figcaption class="figure-caption">Protein Quality Control by the UPS</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The above figure shows how denatured or misfolded proteins are targeted for destruction by the 26s proteasome. Ubiquitin conjugation marks structures for degradation as previously mentioned.</p>
<section id="protein-secretory-pathway" class="level3" data-number="2.3.1">
<h3 data-number="2.3.1" class="anchored" data-anchor-id="protein-secretory-pathway"><span class="header-section-number">2.3.1</span> Protein Secretory Pathway</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/protein_secretory_pathway.png" class="img-fluid figure-img" width="452"></p>
<p></p><figcaption class="figure-caption">Protein Secretory Pathway</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>protein secretory pathway</strong> is a fundamental cellular process responsible for the synthesis, modification, and transport of proteins destined for secretion or localization to various cellular compartments. It involves a series of highly coordinated steps that occur within different organelles and membrane-bound compartments of the cell. Here’s an overview of the protein secretory pathway:</p>
<ol type="1">
<li><p><strong>Protein Synthesis in the Cytoplasm</strong>: Protein synthesis begins on ribosomes in the cytoplasm, where messenger RNA (mRNA) molecules are translated into polypeptide chains. These polypeptide chains contain signal sequences, often located at the N-terminus, which direct them to the endoplasmic reticulum (ER) for further processing and secretion.</p></li>
<li><p><strong>Translocation into the Endoplasmic Reticulum (ER)</strong>: Proteins targeted for secretion or membrane insertion are co-translationally translocated into the lumen of the ER. This process involves the recognition of signal sequences by the signal recognition particle (SRP) and its receptor, which direct the ribosome-nascent chain complex to the ER membrane. Translocation occurs through proteinaceous channels called translocons.</p></li>
<li><p><strong>Protein Folding and Modification in the Endoplasmic Reticulum (ER)</strong>: Once inside the ER lumen, newly synthesized proteins undergo folding, assembly, and post-translational modifications such as glycosylation, disulfide bond formation, and proteolytic cleavage. These processes ensure proper protein conformation and functionality.</p></li>
<li><p><strong>Quality Control in the Endoplasmic Reticulum (ER)</strong>: The ER contains mechanisms for quality control to ensure that only properly folded and assembled proteins are transported further along the secretory pathway. Misfolded or unassembled proteins are recognized and targeted for degradation by the ER-associated degradation (ERAD) pathway.</p></li>
<li><p><strong>Export from the Endoplasmic Reticulum (ER)</strong>: Proteins that pass quality control in the ER are packaged into transport vesicles derived from the ER membrane. These vesicles bud off and fuse with the Golgi apparatus, a central organelle in the secretory pathway.</p></li>
<li><p><strong>Processing and Sorting in the Golgi Apparatus</strong>: Within the Golgi apparatus, proteins undergo further post-translational modifications, including glycosylation, sulfation, phosphorylation, and proteolytic cleavage. The Golgi also acts as a sorting station, directing proteins to their final destinations, such as the plasma membrane, lysosomes, or secretion outside the cell.</p></li>
<li><p><strong>Secretion and Exocytosis</strong>: Proteins destined for secretion are transported in vesicles from the trans-Golgi network to the cell surface. Upon reaching the plasma membrane, these vesicles fuse with the membrane through a process called exocytosis, releasing their contents into the extracellular space.</p></li>
</ol>
</section>
<section id="er-associated-degradation" class="level3" data-number="2.3.2">
<h3 data-number="2.3.2" class="anchored" data-anchor-id="er-associated-degradation"><span class="header-section-number">2.3.2</span> ER-Associated Degradation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/erad.png" class="img-fluid figure-img" width="443"></p>
<p></p><figcaption class="figure-caption">Depiction of ERAD</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/atpase.png" class="img-fluid figure-img" width="492"></p>
<p></p><figcaption class="figure-caption">ATPase in Protein Degradation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>ERAD</strong>, or <strong>ER-associated degradation</strong>, is a quality control mechanism in eukaryotic cells responsible for identifying and targeting misfolded or unassembled proteins in the endoplasmic reticulum (ER) for degradation. The ERAD pathway ensures that only properly folded and functional proteins are allowed to proceed through the secretory pathway and reach their intended destinations within the cell. Here’s how ERAD works:</p>
<ol type="1">
<li><p><strong>Recognition of Misfolded Proteins</strong>: During protein synthesis, the ER serves as the primary site for the folding and maturation of secretory and membrane proteins. However, not all proteins fold correctly, and misfolded or improperly assembled proteins may accumulate in the ER lumen or membrane. These misfolded proteins are recognized by ER quality control systems, which include chaperone proteins and lectins that monitor protein folding status.</p></li>
<li><p><strong>Ubiquitination</strong>: Upon recognition of misfolded proteins, they are targeted for degradation through the covalent attachment of ubiquitin molecules. Ubiquitination is mediated by a series of enzymes, including ubiquitin-activating enzymes (E1s), ubiquitin-conjugating enzymes (E2s), and ubiquitin ligases (E3s). In the case of ERAD, specific E3 ubiquitin ligases recognize misfolded proteins and facilitate their ubiquitination.</p></li>
<li><p><strong>Extraction from the ER Membrane</strong>: Misfolded membrane proteins must be extracted from the ER membrane before they can be degraded by the proteasome, a large proteolytic complex found in the cytoplasm. This extraction process involves the action of protein complexes such as the ATPase-containing Cdc48/p97 complex (also known as valosin-containing protein, VCP, in mammals), which extracts the misfolded proteins from the ER membrane into the cytoplasm.</p></li>
<li><p><strong>Degradation by the Proteasome</strong>: Once extracted from the ER membrane, ubiquitinated misfolded proteins are targeted to the proteasome for degradation. The proteasome recognizes polyubiquitinated proteins and proteolytically cleaves them into short peptides. These peptides can then be further degraded into amino acids by cytoplasmic peptidases.</p></li>
<li><p><strong>Regulation and Coordination</strong>: The ERAD pathway is tightly regulated to ensure efficient removal of misfolded proteins while minimizing the degradation of properly folded proteins. It involves coordination between various components, including chaperones, ubiquitin ligases, and the proteasome, as well as post-translational modifications and signaling pathways.</p></li>
</ol>
<section id="cdc48-p97-aaa-atpase-in-protein-degradation" class="level4" data-number="2.3.2.1">
<h4 data-number="2.3.2.1" class="anchored" data-anchor-id="cdc48-p97-aaa-atpase-in-protein-degradation"><span class="header-section-number">2.3.2.1</span> Cdc48 / p97: AAA<sup>+</sup> ATPase in Protein Degradation</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/atpase.png" class="img-fluid figure-img" width="492"></p>
<p></p><figcaption class="figure-caption">ATPase Types in Organisms and Activities</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/atpase_activities.png" class="img-fluid figure-img" width="460"></p>
<p></p><figcaption class="figure-caption">ATPase Types in Organisms and Activities</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/atpase_mechanism.png" class="img-fluid figure-img" width="244"></p>
<p></p><figcaption class="figure-caption">ATPase Activities in ERAD</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>AAA<sup>+</sup> (ATPases Associated with diverse cellular Activities) family of proteins</strong> is a large and diverse group of ATPases found in all domains of life. These proteins share a common structural feature known as the AAA+ domain, which is characterized by conserved sequence motifs and is responsible for ATP binding and hydrolysis. AAA<sup>+</sup> proteins participate in a wide range of cellular processes, including protein degradation, DNA replication, DNA repair, membrane fusion, and vesicle trafficking. Here’s an overview of the general mechanism of AAA<sup>+</sup> family proteins:</p>
<ol type="1">
<li><p><strong>Oligomerization</strong>: AAA+ proteins typically function as oligomers, forming complex structures composed of multiple subunits. These subunits may assemble into hexameric rings, double rings, or other higher-order oligomeric structures depending on the specific protein and its function.</p></li>
<li><p><strong>ATP Binding and Hydrolysis</strong>: The AAA+ domain contains conserved Walker A and Walker B motifs, which are responsible for ATP binding and hydrolysis, respectively. Upon ATP binding, conformational changes occur within the AAA+ domain, resulting in the formation of a catalytically active ATPase site. ATP hydrolysis fuels the mechanical work performed by AAA+ proteins, driving conformational changes that are often essential for their biological functions.</p></li>
<li><p><strong>Substrate Recognition and Binding</strong>: AAA+ proteins interact with specific substrate molecules through specialized substrate-binding domains or motifs located adjacent to the AAA+ domain. These substrate-binding domains recognize and bind to target proteins, nucleic acids, or other substrates, facilitating their processing, modification, or degradation.</p></li>
<li><p><strong>Mechanical Work and Substrate Remodeling</strong>: ATP hydrolysis by AAA+ proteins generates mechanical force that drives conformational changes in the protein complex. These conformational changes can result in the remodeling, unfolding, or translocation of substrates, depending on the specific function of the AAA+ protein. For example, AAA+ proteins involved in protein degradation may unfold and translocate substrate proteins through a central pore formed by the AAA+ oligomer.</p></li>
<li><p><strong>Regulation and Coordination</strong>: The activity of AAA+ proteins is tightly regulated to ensure proper timing and coordination of cellular processes. Regulation may involve factors such as post-translational modifications, cofactors, allosteric regulators, and interactions with other proteins or cellular components. For example, ATP binding and hydrolysis kinetics, as well as interactions with regulatory proteins, can modulate the activity of AAA<sup>+</sup> proteins involved in DNA replication or protein degradation.</p></li>
</ol>
</section>
</section>
<section id="cdc48-p97-function-in-ubs" class="level3" data-number="2.3.3">
<h3 data-number="2.3.3" class="anchored" data-anchor-id="cdc48-p97-function-in-ubs"><span class="header-section-number">2.3.3</span> Cdc48 / p97 Function in UBS</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/cdc_function.png" class="img-fluid figure-img" width="396"></p>
<p></p><figcaption class="figure-caption">Role of Cdc48 / p97 in the UBS</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Cdc48, also known as p97 in mammals, is a highly conserved AAA+ (ATPases Associated with diverse cellular Activities) ATPase protein that plays a crucial role in various cellular processes, including the ubiquitin-proteasome system (UPS). Its function in the UPS is diverse and multifaceted, contributing to protein quality control, protein degradation, and various cellular pathways. Here’s an overview of the functions of Cdc48/p97 in the ubiquitin-proteasome system:</p>
<ol type="1">
<li><p><strong>Protein Degradation and ERAD (ER-Associated Degradation)</strong>: Cdc48/p97 is a central player in ERAD, a quality control mechanism responsible for identifying and targeting misfolded or unassembled proteins in the endoplasmic reticulum (ER) for degradation by the proteasome. Cdc48/p97 extracts ubiquitinated substrates from the ER membrane and facilitates their delivery to the proteasome for degradation. It collaborates with other ERAD components, including E3 ubiquitin ligases and ubiquitin-conjugating enzymes, to ensure efficient protein degradation and maintain ER homeostasis.</p></li>
<li><p><strong>Ubiquitin-Dependent Protein Degradation</strong>: Cdc48/p97 participates in the degradation of ubiquitinated proteins in the cytoplasm and nucleus. It collaborates with specific ubiquitin ligases to extract ubiquitinated proteins from cellular structures, protein complexes, or membranes, facilitating their degradation by the proteasome. Cdc48/p97 also plays a role in disassembling protein complexes or aggregates targeted for degradation, promoting the recycling of ubiquitin and preventing the accumulation of toxic protein aggregates.</p></li>
<li><p><strong>Cell Cycle Regulation</strong>: Cdc48/p97 is involved in various aspects of cell cycle regulation, including mitotic progression, DNA replication, and DNA repair. It participates in the turnover of cell cycle regulators, DNA replication factors, and checkpoint proteins by facilitating their ubiquitin-dependent degradation. Cdc48/p97-mediated protein degradation ensures proper cell cycle progression and maintenance of genomic integrity.</p></li>
<li><p><strong>Protein Quality Control and Stress Response</strong>: Cdc48/p97 contributes to protein quality control and stress response pathways by eliminating misfolded, damaged, or aggregated proteins. It collaborates with molecular chaperones, proteases, and other protein quality control factors to recognize, extract, and degrade aberrant proteins, thereby maintaining cellular homeostasis and viability under stress conditions.</p></li>
<li><p><strong>Membrane Dynamics and Vesicle Trafficking</strong>: Cdc48/p97 participates in membrane dynamics, vesicle trafficking, and organelle biogenesis by regulating the turnover of membrane-associated proteins and lipid-modifying enzymes. It mediates the extraction and degradation of membrane proteins involved in membrane fusion, vesicle formation, and organelle assembly, contributing to the maintenance of cellular membrane integrity and function.</p></li>
</ol>
</section>
</section>
<section id="degradation-by-the-proteasome" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="degradation-by-the-proteasome"><span class="header-section-number">2.4</span> Degradation by the Proteasome</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/proteasome_structure.png" class="img-fluid figure-img" width="521"></p>
<p></p><figcaption class="figure-caption">Structure of the Proteasome</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>proteasome</strong>, a critical component of cellular protein degradation machinery, is a large, multi-subunit protease complex found within eukaryotic cells. It plays a fundamental role in maintaining cellular homeostasis by selectively degrading proteins marked for destruction by ubiquitin. Structurally, the proteasome consists of a core particle, known as the <strong>20S proteasome</strong>, which serves as the central catalytic unit responsible for protein degradation. This cylindrical core particle comprises four stacked rings, with two outer rings composed of seven α subunits each and two inner rings made up of seven β subunits each.</p>
<p>Each β subunit within the 20S proteasome possesses catalytic activity, allowing the proteasome to cleave peptide bonds within target proteins. The β subunits are categorized into three distinct types: β1, β2, and β5, each exhibiting specific peptidase activities. The β1 subunits display caspase-like activity, cleaving after acidic residues; the β2 subunits exhibit trypsin-like activity, cleaving after basic residues; and the β5 subunits demonstrate chymotrypsin-like activity, cleaving after hydrophobic residues.</p>
<p>Proteins targeted for degradation are tagged with a small protein called ubiquitin, marking them for recognition and processing by the proteasome. The recognition and degradation of ubiquitinated proteins involve a complex series of events coordinated by various cellular components, including E1 ubiquitin-activating enzymes, E2 ubiquitin-conjugating enzymes, and E3 ubiquitin ligases. Once recognized, ubiquitinated proteins are unfolded and translocated into the central chamber of the 20S proteasome, where they undergo proteolytic degradation into short peptide fragments.</p>
<section id="multiple-myeloma-and-proteasome-inhibitor" class="level3" data-number="2.4.1">
<h3 data-number="2.4.1" class="anchored" data-anchor-id="multiple-myeloma-and-proteasome-inhibitor"><span class="header-section-number">2.4.1</span> Multiple Myeloma and Proteasome Inhibitor</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/inhibitor.png" class="img-fluid figure-img" width="346"></p>
<p></p><figcaption class="figure-caption">Inhibitors of the Proteasome and Multiple Myeloma</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Multiple myeloma</strong> is a kind of cancer of the plasma cells. One’s risk of developing multiple myeloma increases as they become older - most people who have this kind of cancer get diagnosed in their mid-60s.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/b_cell.png" class="img-fluid figure-img" width="470"></p>
<p></p><figcaption class="figure-caption">Comparison Between a B Cell and a Plasma Cell</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Some symptoms of this cancer include:</p>
<ol type="1">
<li>Anemia (i.e., a shortage of red blood cells) resulting in fatigue</li>
<li>Thrombocytopenia (i.e., a lack of platelets) resulting in easy bleeding and bruising</li>
<li>Leukopenia (i.e., a lack of white blood cells) resulting in frequent infections</li>
</ol>
</section>
<section id="aneuploidy-and-toxic-stress" class="level3" data-number="2.4.2">
<h3 data-number="2.4.2" class="anchored" data-anchor-id="aneuploidy-and-toxic-stress"><span class="header-section-number">2.4.2</span> Aneuploidy and Toxic Stress</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/an_and_toxic.png" class="img-fluid figure-img" width="390"></p>
<p></p><figcaption class="figure-caption">Aneuploidy and Toxic Stress</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Aneuploidy</strong> refers to an abnormal number of chromosomes in a cell, typically resulting from errors in cell division processes such as mitosis or meiosis. This condition leads to cells having either fewer or more chromosomes than the normal diploid number characteristic of the species. Aneuploidy can occur in various organisms, including humans, and it is associated with several health conditions, including developmental disorders and cancer.</p>
<p><strong>Proteotoxic stress</strong>, on the other hand, refers to the cellular stress caused by an imbalance in protein homeostasis, leading to the accumulation of misfolded or aggregated proteins. Proteotoxic stress can result from various factors such as environmental stressors, genetic mutations, or disruptions in protein quality control mechanisms. The accumulation of misfolded proteins can overwhelm the cellular protein degradation systems, including the ubiquitin-proteasome system and autophagy pathways, leading to cellular dysfunction and disease.</p>
<p>Aneuploid cells often experience proteotoxic stress due to alterations in gene dosage and expression resulting from chromosome imbalances. The abnormal expression levels of genes encoding chaperones, proteases, and components of the protein degradation machinery can disrupt protein folding, assembly, and turnover processes, contributing to proteotoxic stress in aneuploid cells.</p>
<p>Conversely, proteotoxic stress can also promote aneuploidy by inducing genomic instability and impairing cell division processes. Misfolded or aggregated proteins can interfere with the function of key cellular pathways involved in DNA replication, repair, and chromosome segregation, leading to the accumulation of genetic abnormalities and aneuploidy over time.</p>
</section>
<section id="problems-with-proteasomal-degradation" class="level3" data-number="2.4.3">
<h3 data-number="2.4.3" class="anchored" data-anchor-id="problems-with-proteasomal-degradation"><span class="header-section-number">2.4.3</span> Problems with Proteasomal Degradation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/problem_1.png" class="img-fluid figure-img" width="520"></p>
<p></p><figcaption class="figure-caption">Problems Associated with Proteasomal Degradation</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/problem_2.png" class="img-fluid figure-img" width="518"></p>
<p></p><figcaption class="figure-caption">Problems Associated with Proteasomal Degradation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Several problems can arise when proteasomal degradation is impaired or dysregulated:</p>
<ol type="1">
<li><p><strong>Accumulation of Misfolded or Damaged Proteins</strong>: Impaired proteasomal degradation can lead to the accumulation of misfolded or damaged proteins within cells. These proteins may form aggregates or toxic oligomers, disrupting cellular function and contributing to the pathogenesis of neurodegenerative diseases such as Alzheimer’s, Parkinson’s, and Huntington’s disease.</p></li>
<li><p><strong>Dysregulation of Protein Turnover</strong>: Proper protein turnover is essential for maintaining cellular function and responding to dynamic changes in cellular environments. Dysregulation of proteasomal degradation can disrupt protein turnover rates, leading to the accumulation of specific proteins that are not degraded when they should be, or the premature degradation of proteins essential for cellular processes.</p></li>
<li><p><strong>Protein Aggregation and Neurodegeneration</strong>: Impaired proteasomal degradation is implicated in the pathogenesis of neurodegenerative diseases characterized by the accumulation of protein aggregates, such as tau tangles in Alzheimer’s disease and α-synuclein aggregates in Parkinson’s disease. The failure to clear these aggregates can lead to neuronal dysfunction and cell death.</p></li>
<li><p><strong>Cell Cycle Dysregulation</strong>: The proteasome plays a crucial role in regulating cell cycle progression by degrading key cell cycle regulators, such as cyclins and cyclin-dependent kinase inhibitors. Dysregulation of proteasomal degradation can lead to aberrant cell cycle progression, uncontrolled cell proliferation, and genomic instability, contributing to cancer development and progression.</p></li>
<li><p><strong>Immune System Dysfunction</strong>: The proteasome is involved in the processing of antigens for presentation on major histocompatibility complex (MHC) molecules, a critical step in the activation of the immune response. Impaired proteasomal degradation can lead to defects in antigen presentation, compromising immune surveillance and response to pathogens.</p></li>
<li><p><strong>Drug Resistance in Cancer</strong>: Cancer cells often develop resistance to chemotherapy drugs by upregulating proteasomal activity, leading to increased degradation of anti-cancer proteins. Targeting the proteasome with proteasome inhibitors can overcome drug resistance in some cases by inducing apoptosis in cancer cells.</p></li>
<li><p><strong>Aging and Age-Related Diseases</strong>: Dysregulation of proteasomal degradation is associated with aging and age-related diseases. As cells age, the efficiency of proteasomal degradation declines, leading to the accumulation of damaged proteins and contributing to the aging process and age-related diseases.</p></li>
</ol>
</section>
</section>
<section id="ups-and-neurodegeneration" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="ups-and-neurodegeneration"><span class="header-section-number">2.5</span> UPS and Neurodegeneration</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week2/ups_aggregates.png" class="img-fluid figure-img" width="338"></p>
<p></p><figcaption class="figure-caption">Impact of Protein Aggregates on the UPS</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Protein aggregates, which result from the misfolding and aggregation of proteins, can have significant impacts on the Ubiquitin-Proteasome System (UPS), the primary mechanism responsible for protein degradation in eukaryotic cells. Here are some key impacts of protein aggregates on the UPS:</p>
<ol type="1">
<li><p><strong>Overload of the UPS</strong>: Protein aggregates can overwhelm the capacity of the UPS, leading to a backlog of ubiquitinated proteins awaiting degradation. The UPS relies on the coordinated action of E1 ubiquitin-activating enzymes, E2 ubiquitin-conjugating enzymes, and E3 ubiquitin ligases to tag proteins with ubiquitin for degradation by the proteasome. However, the excessive accumulation of misfolded or aggregated proteins can saturate the UPS, impairing its ability to degrade other cellular proteins.</p></li>
<li><p><strong>Inhibition of Proteasomal Activity</strong>: Protein aggregates can directly inhibit proteasomal activity by physically obstructing the proteasome’s access to substrate proteins or by interfering with the unfolding and translocation of substrates into the proteasomal core particle. Additionally, certain types of protein aggregates, such as amyloid fibrils, can bind to and sequester proteasomal subunits, further impairing proteasome function.</p></li>
<li><p><strong>Induction of Proteasome Stress</strong>: The presence of protein aggregates within cells can induce a cellular stress response known as proteasome stress. Proteasome stress is characterized by the upregulation of genes encoding proteasome subunits, as well as components of the UPS and protein quality control machinery. While proteasome stress initially serves to enhance the cell’s capacity to degrade misfolded proteins, chronic or severe proteasome stress can lead to cellular dysfunction and apoptosis.</p></li>
<li><p><strong>Formation of Resistant Aggregates</strong>: Some protein aggregates, particularly those associated with neurodegenerative diseases such as Alzheimer’s and Parkinson’s disease, are highly stable and resistant to degradation by the UPS. These aggregates may contain cross-linked or ubiquitinated proteins that are difficult for the proteasome to recognize and process. As a result, the accumulation of resistant protein aggregates can exacerbate proteotoxicity and contribute to disease progression.</p></li>
<li><p><strong>Impairment of Protein Quality Control Pathways</strong>: The presence of protein aggregates can impair other protein quality control pathways, such as chaperone-mediated autophagy (CMA) and macroautophagy. While the UPS primarily degrades short-lived and soluble proteins, autophagy pathways are responsible for the degradation of larger protein aggregates and damaged organelles. However, the chronic accumulation of protein aggregates can overwhelm autophagy pathways, further exacerbating proteotoxicity and cellular dysfunction.</p></li>
</ol>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week1.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week3.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>