<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 4&nbsp; Chaperones and Protein Homeostasis</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week5.html" rel="next">
<link href="../chapters/week3.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Alzheimer’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Parkinson’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">ALS and Huntington’s Disease</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#protein-folding-pathways-in-bacteria" id="toc-protein-folding-pathways-in-bacteria" class="nav-link active" data-scroll-target="#protein-folding-pathways-in-bacteria"><span class="toc-section-number">4.1</span>  Protein Folding Pathways in Bacteria</a>
  <ul class="collapse">
  <li><a href="#chaperonin" id="toc-chaperonin" class="nav-link" data-scroll-target="#chaperonin"><span class="toc-section-number">4.1.1</span>  Chaperonin</a></li>
  </ul></li>
  <li><a href="#chaperone-families-in-eukaryotes-and-their-functions" id="toc-chaperone-families-in-eukaryotes-and-their-functions" class="nav-link" data-scroll-target="#chaperone-families-in-eukaryotes-and-their-functions"><span class="toc-section-number">4.2</span>  Chaperone Families in Eukaryotes and Their Functions</a>
  <ul class="collapse">
  <li><a href="#functions-of-hsp70" id="toc-functions-of-hsp70" class="nav-link" data-scroll-target="#functions-of-hsp70"><span class="toc-section-number">4.2.1</span>  Functions of Hsp70</a></li>
  <li><a href="#tric" id="toc-tric" class="nav-link" data-scroll-target="#tric"><span class="toc-section-number">4.2.2</span>  TRiC</a></li>
  <li><a href="#protein-folding-in-eukaryotes" id="toc-protein-folding-in-eukaryotes" class="nav-link" data-scroll-target="#protein-folding-in-eukaryotes"><span class="toc-section-number">4.2.3</span>  Protein Folding in Eukaryotes</a></li>
  <li><a href="#polyribosomes" id="toc-polyribosomes" class="nav-link" data-scroll-target="#polyribosomes"><span class="toc-section-number">4.2.4</span>  Polyribosomes</a></li>
  <li><a href="#secretory-protein-binding" id="toc-secretory-protein-binding" class="nav-link" data-scroll-target="#secretory-protein-binding"><span class="toc-section-number">4.2.5</span>  Secretory Protein Binding</a></li>
  <li><a href="#protein-glycosylation" id="toc-protein-glycosylation" class="nav-link" data-scroll-target="#protein-glycosylation"><span class="toc-section-number">4.2.6</span>  Protein Glycosylation</a></li>
  <li><a href="#erad-substrate-recognition" id="toc-erad-substrate-recognition" class="nav-link" data-scroll-target="#erad-substrate-recognition"><span class="toc-section-number">4.2.7</span>  ERAD Substrate Recognition</a></li>
  </ul></li>
  <li><a href="#cellular-responses-to-proteotoxic-stresses" id="toc-cellular-responses-to-proteotoxic-stresses" class="nav-link" data-scroll-target="#cellular-responses-to-proteotoxic-stresses"><span class="toc-section-number">4.3</span>  Cellular Responses to Proteotoxic Stresses</a>
  <ul class="collapse">
  <li><a href="#proteostasis-network" id="toc-proteostasis-network" class="nav-link" data-scroll-target="#proteostasis-network"><span class="toc-section-number">4.3.1</span>  Proteostasis Network</a></li>
  <li><a href="#heat-shock-response" id="toc-heat-shock-response" class="nav-link" data-scroll-target="#heat-shock-response"><span class="toc-section-number">4.3.2</span>  Heat Shock Response</a></li>
  <li><a href="#unfolded-protein-response" id="toc-unfolded-protein-response" class="nav-link" data-scroll-target="#unfolded-protein-response"><span class="toc-section-number">4.3.3</span>  Unfolded Protein Response</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week4.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<p>Christian Anfinsen, winner of the 1972 Nobel Prize in Chemistry, once pondered the questions of who is right and what is true. These questions are fundamental to understanding the world around us. In his research, Anfinsen explored how proteins fold, a process essential for their function. His work shed light on the complex mechanisms governing life at a molecular level.</p>
<p>Similarly, scientists F. Ulrich Hartl and Arthur Horwich have made significant contributions to our understanding of biology. Their research delves into the intricate world of protein folding, unraveling the mysteries of how cells function and how diseases like Alzheimer’s and Parkinson’s develop.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/protein_folding.png" class="img-fluid figure-img" width="516"></p>
<p></p><figcaption class="figure-caption">Illustration of Protein Folding in a Crowded Environment</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Protein folding</strong>, the process by which proteins assume their functional three-dimensional shapes, is crucial for their proper functioning within living organisms. Imagine a bustling city with crowded streets; similarly, within our cells, proteins fold in a crowded environment filled with other molecules.</p>
<p>This crowded environment presents challenges for proteins as they fold. They must navigate through a sea of other molecules, jostling and interacting with them along the way. Sometimes, this crowded space can hinder the folding process, leading to misfolded proteins that may not function correctly.</p>
<p>Scientists like F. Ulrich Hartl and Arthur Horwich have dedicated their research to understanding how proteins fold in such crowded conditions. By studying the intricate dance of molecules within cells, they have uncovered mechanisms that assist proteins in folding properly despite the crowded environment.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/chaperones.png" class="img-fluid figure-img" width="508"></p>
<p></p><figcaption class="figure-caption">Illustration of Chaperones in Protein Folding</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Chaperones</strong> play vital roles in the proteostasis network, which is the cellular system responsible for maintaining the proper balance of proteins. Think of chaperones as the guardians of protein health within our cells.</p>
<p>One of the primary functions of chaperones is to assist proteins in folding correctly. Proteins start as long chains of amino acids and need to fold into specific shapes to perform their functions effectively. Chaperones help guide this folding process, ensuring that proteins adopt their proper three-dimensional structures.</p>
<p>Additionally, chaperones prevent protein misfolding and aggregation. Sometimes, proteins can misfold or clump together, which can be harmful to the cell. Chaperones act as quality control agents, identifying misfolded or aggregated proteins and either refolding them into their correct shapes or targeting them for degradation.</p>
<p>Chaperones also play a role in protein transport within the cell. They help ferry proteins to their designated locations, ensuring they reach the right organelles or compartments where they are needed.</p>
<p>Furthermore, chaperones assist in protein degradation. When proteins are damaged or no longer needed, they must be broken down and recycled. Chaperones help facilitate this process by guiding proteins to degradation machinery where they can be dismantled into their component parts for reuse.</p>
<section id="protein-folding-pathways-in-bacteria" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="protein-folding-pathways-in-bacteria"><span class="header-section-number">4.1</span> Protein Folding Pathways in Bacteria</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/bacteria_pathway.png" class="img-fluid figure-img" width="186"></p>
<p></p><figcaption class="figure-caption">Protein Folding Pathway in Bacteria</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>One of the primary pathways involved in protein folding in bacteria is the <strong>chaperone-mediated folding pathway</strong>. Chaperones are specialized proteins that assist other proteins in folding correctly. Within this pathway, chaperones interact with newly synthesized proteins as they emerge from ribosomes, guiding them through the folding process to ensure they attain their functional shapes.</p>
<p>Another important pathway in bacterial protein folding is the <strong>co-translational folding pathway</strong>. In this pathway, proteins begin to fold even as they are being synthesized by ribosomes. As the protein chain emerges from the ribosome, it starts to fold into its native structure with the help of chaperones and other folding factors. This simultaneous folding and synthesis process ensure that the protein is properly folded as soon as it is synthesized.</p>
<p>Furthermore, bacteria utilize <strong>protein degradation pathways</strong> to maintain protein homeostasis. If a protein fails to fold correctly despite chaperone assistance or becomes damaged or obsolete, it may be targeted for degradation. Bacterial cells have proteases, enzymes specialized in breaking down proteins, which dismantle misfolded or unwanted proteins into smaller peptides or amino acids for recycling.</p>
<section id="hsp70-hsp40-system" class="level4" data-number="4.1.0.1">
<h4 data-number="4.1.0.1" class="anchored" data-anchor-id="hsp70-hsp40-system"><span class="header-section-number">4.1.0.1</span> Hsp70-Hsp40 System</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/hsp70.png" class="img-fluid figure-img" width="510"></p>
<p></p><figcaption class="figure-caption">Illustration of the Hsp70-Hsp40 System</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>Hsp70-Hsp40 system</strong> is a critical molecular machinery found in cells, including bacteria, that helps in protein folding and quality control. These components are often referred to as chaperones because they assist other proteins in folding correctly and prevent misfolding.</p>
<p>At the heart of this system is the Hsp70 protein, which acts as a molecular chaperone. Hsp70 binds to exposed hydrophobic regions on proteins, which are common during the folding process. By doing so, Hsp70 prevents these regions from sticking together inappropriately, thus helping the protein fold properly into its functional shape. However, Hsp70 alone can be slow in its action.</p>
<p>This is where the Hsp40 co-chaperone comes into play. Hsp40 interacts with Hsp70 and helps accelerate its activity. It does this by stimulating the ATPase activity of Hsp70, which is crucial for its chaperone function. By enhancing the ATPase activity, Hsp40 increases the efficiency of Hsp70 in binding to and releasing its client proteins, thereby facilitating the protein folding process.</p>
<p>Together, the Hsp70-Hsp40 system forms a dynamic duo in the cell, working hand in hand to ensure that proteins fold correctly and maintain their proper structures. This system is essential for cellular health and is involved in various cellular processes, including protein synthesis, protein trafficking, and response to stress.</p>
</section>
<section id="chaperonin" class="level3" data-number="4.1.1">
<h3 data-number="4.1.1" class="anchored" data-anchor-id="chaperonin"><span class="header-section-number">4.1.1</span> Chaperonin</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/chaperonin.png" class="img-fluid figure-img" width="514"></p>
<p></p><figcaption class="figure-caption">Illustration of Chaperonin</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Chaperonins</strong> are like tiny cages within the cell that provide a protected environment for protein folding. These remarkable structures play a crucial role in ensuring that proteins fold correctly, without interference from other molecules in the crowded cellular environment.</p>
<p>The key feature of chaperonins is their hollow, barrel-like structure, which forms a chamber where proteins can fold undisturbed. One of the most well-known types of chaperonins is called <strong>GroEL-GroES</strong>, found in bacteria.</p>
<p>When a protein needs to fold, it enters the central cavity of the chaperonin. Once inside, the chaperonin closes like a lid, encapsulating the protein within its chamber. This isolation shields the folding protein from potential disruptions by other cellular components, allowing it to fold properly.</p>
<p>Inside the chaperonin chamber, the folding protein is provided with the ideal conditions for folding. The enclosed space allows the protein to explore different conformations without interference, helping it find its correct three-dimensional structure.</p>
<p>To further assist in the folding process, chaperonins often work in conjunction with co-chaperones, such as GroES in the case of GroEL. GroES acts as a lid, sealing the top of the chaperonin chamber and creating an even more secluded environment for protein folding.</p>
<p>Once the folding process is complete, the chaperonin releases the folded protein, which can then go on to perform its specific function within the cell.</p>
</section>
</section>
<section id="chaperone-families-in-eukaryotes-and-their-functions" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="chaperone-families-in-eukaryotes-and-their-functions"><span class="header-section-number">4.2</span> Chaperone Families in Eukaryotes and Their Functions</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/nac_and_rac.png" class="img-fluid figure-img" width="504"></p>
<p></p><figcaption class="figure-caption">NAC and RAC</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In eukaryotic cells, there are specialized chaperones that assist in protein folding as soon as the proteins are synthesized by ribosomes. Two prominent examples of these ribosome-binding chaperones are <strong>RAC (Ribosome-associated complex)</strong> and <strong>NAC (Nascent polypeptide-associated complex)</strong>.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/nac_and_rac2.png" class="img-fluid figure-img" width="488"></p>
<p></p><figcaption class="figure-caption">Illustration of NAC and RAC Mechanism</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>RAC serves as a cofactor of Hsp70, another important chaperone protein, facilitating its activity in protein folding. As proteins are being synthesized by ribosomes, RAC interacts with the ribosome and recruits Hsp70 to the nascent polypeptide chain as it emerges from the ribosome. This interaction helps to prevent premature folding or aggregation of the nascent protein, ensuring that it attains its proper three-dimensional structure.</p>
<p>NAC, on the other hand, directly binds to the nascent polypeptide chain as it emerges from the ribosome. This association helps to shield the newly synthesized protein from interactions with other cellular components and provides a conducive environment for proper folding. NAC also assists in targeting the nascent protein to its correct cellular location, ensuring that it reaches its designated organelle or compartment.</p>
<section id="functions-of-hsp70" class="level3" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="functions-of-hsp70"><span class="header-section-number">4.2.1</span> Functions of Hsp70</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/hsp70_functions.png" class="img-fluid figure-img" width="452"></p>
<p></p><figcaption class="figure-caption">Cellular Functions of Hsp70</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Firstly, Hsp70 assists in protein folding. It interacts with newly synthesized or misfolded proteins, preventing them from aggregating and guiding them along the correct folding pathways. By doing so, Hsp70 ensures that proteins adopt their proper three-dimensional structures, which is essential for their biological activity.</p>
<p>Secondly, Hsp70 plays a role in protein transport within the cell. It helps to ferry proteins across membranes or guide them to specific cellular compartments where they are needed for various functions. This transport function ensures that proteins reach their correct destinations within the cell, allowing them to perform their designated roles.</p>
<p>Additionally, Hsp70 is involved in protein degradation. It interacts with misfolded or damaged proteins, targeting them for degradation by cellular machinery such as proteasomes. This process helps to remove aberrant proteins from the cell, preventing them from interfering with normal cellular processes and maintaining protein quality control.</p>
<p>Moreover, Hsp70 plays a crucial role in cellular stress responses. When cells are exposed to stressful conditions such as heat, oxidative stress, or nutrient deprivation, the expression of Hsp70 is often upregulated. Hsp70 helps cells cope with these stresses by stabilizing proteins, preventing their denaturation, and promoting cell survival under adverse conditions.</p>
</section>
<section id="tric" class="level3" data-number="4.2.2">
<h3 data-number="4.2.2" class="anchored" data-anchor-id="tric"><span class="header-section-number">4.2.2</span> TRiC</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/tric.png" class="img-fluid figure-img" width="458"></p>
<p></p><figcaption class="figure-caption">TRiC Illustration</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>TRiC</strong>, also known as TCP-1 Ring Complex, is a type II chaperonin found in the cytosol of eukaryotic cells. It is composed of two stacked rings, each containing eight different subunits. TRiC plays a crucial role in facilitating the folding of diverse cytosolic proteins, ensuring their proper structure and function.</p>
<p>One of the key functions of TRiC is to assist in the folding of newly synthesized proteins as they emerge from ribosomes. As proteins are synthesized, they often require assistance to attain their correct three-dimensional shapes. TRiC provides a protected environment within its chamber-like structure, allowing proteins to fold properly without interference from other cellular components. This function is particularly important for proteins with complex folding patterns or those prone to misfolding.</p>
<p>Additionally, TRiC is involved in the folding of proteins that are crucial for cellular processes such as cell cycle regulation, cytoskeletal dynamics, and signal transduction. By facilitating the folding of these essential proteins, TRiC contributes to the proper functioning of the cell.</p>
<p>Moreover, TRiC is implicated in the maintenance of protein homeostasis and cellular integrity under stress conditions. When cells are exposed to environmental stressors such as heat or toxins, TRiC helps to prevent protein misfolding and aggregation, thereby promoting cell survival.</p>
</section>
<section id="protein-folding-in-eukaryotes" class="level3" data-number="4.2.3">
<h3 data-number="4.2.3" class="anchored" data-anchor-id="protein-folding-in-eukaryotes"><span class="header-section-number">4.2.3</span> Protein Folding in Eukaryotes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/protein_folding_eu.png" class="img-fluid figure-img" width="238"></p>
<p></p><figcaption class="figure-caption">Protein Folding in Eukaryotes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In eukaryotic cells, cytosolic protein folding involves a series of coordinated pathways and molecular chaperones that ensure newly synthesized proteins attain their correct three-dimensional structures. Here’s an overview of the key players and steps involved in cytosolic protein folding:</p>
<ol type="1">
<li><p><strong>RAC (Ribosome-associated complex)</strong> and <strong>NAC (Nascent polypeptide-associated complex)</strong>: These complexes interact with ribosomes and nascent polypeptide chains emerging from ribosomes, respectively. RAC recruits Hsp70 to the nascent chain, while NAC directly binds to the nascent polypeptide, protecting it and facilitating its proper folding.</p></li>
<li><p><strong>Hsp70</strong>: Heat Shock Protein 70 is a major molecular chaperone involved in protein folding. It interacts with exposed hydrophobic regions on proteins, preventing misfolding and aggregation. Hsp70’s activity is regulated by ATP and co-chaperones such as Hsp40 (also known as DnaJ).</p></li>
<li><p><strong>Hsp40 (DnaJ)</strong>: Hsp40 acts as a co-chaperone to Hsp70, stimulating its ATPase activity and enhancing its chaperone function. It helps to deliver substrate proteins to Hsp70 and stabilize their binding.</p></li>
<li><p><strong>NEF (Nucleotide Exchange Factor)</strong>: NEF assists in the dissociation of ADP from Hsp70, allowing for the binding of fresh ATP. This ATP hydrolysis and exchange cycle is essential for the chaperone activity of Hsp70.</p></li>
<li><p><strong>HOP (Hsp70-Hsp90 Organizing Protein)</strong>: HOP acts as a co-chaperone that bridges the interaction between Hsp70 and Hsp90. This interaction is crucial for the transfer of client proteins from Hsp70 to Hsp90 during their folding process.</p></li>
<li><p><strong>Hsp90</strong>: Heat Shock Protein 90 is another important molecular chaperone involved in protein folding. It acts downstream of Hsp70 and plays a role in the maturation and stabilization of client proteins. Hsp90’s activity is also ATP-dependent.</p></li>
<li><p><strong>TRiC (TCP-1 Ring Complex)</strong>: TRiC is a large multi-subunit chaperonin complex that provides a protected environment for the folding of diverse cytosolic proteins. It assists in the folding of a wide range of substrates, particularly those with complex folding patterns.</p></li>
</ol>
</section>
<section id="polyribosomes" class="level3" data-number="4.2.4">
<h3 data-number="4.2.4" class="anchored" data-anchor-id="polyribosomes"><span class="header-section-number">4.2.4</span> Polyribosomes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/polyribosomes.png" class="img-fluid figure-img" width="460"></p>
<p></p><figcaption class="figure-caption">Illustration of Polyribosomes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Polyribosomes</strong>, also known as polysomes, are clusters of ribosomes found within eukaryotic cells that are actively translating mRNA molecules into proteins. These structures play a crucial role in protein synthesis, allowing multiple ribosomes to simultaneously translate a single mRNA strand. Within eukaryotic cells, polyribosomes can exist in two primary forms: free polyribosomes and membrane-bound polyribosomes.</p>
<p>Free polyribosomes are dispersed throughout the cytoplasm, existing independently of any membrane-bound structures such as the endoplasmic reticulum (ER). They are responsible for synthesizing proteins that are intended to remain within the cytosol or to be transported to various organelles such as the nucleus or mitochondria. Proteins synthesized by free polyribosomes often fulfill essential cellular functions related to metabolism, cytoskeletal organization, and signaling pathways.</p>
<p>On the other hand, membrane-bound polyribosomes are attached to the endoplasmic reticulum (ER) membrane. These polyribosomes are typically found in close association with the rough ER, contributing to its characteristic rough appearance observed under electron microscopy. Proteins synthesized by membrane-bound polyribosomes are translocated into the lumen of the ER as they are being synthesized. These proteins undergo further processing, such as folding and glycosylation, within the ER before being transported to their final destinations. Membrane-bound polyribosomes primarily synthesize proteins that are destined for secretion, incorporation into the plasma membrane, or localization within the endomembrane system, including lysosomes and the Golgi apparatus.</p>
</section>
<section id="secretory-protein-binding" class="level3" data-number="4.2.5">
<h3 data-number="4.2.5" class="anchored" data-anchor-id="secretory-protein-binding"><span class="header-section-number">4.2.5</span> Secretory Protein Binding</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/secretory.png" class="img-fluid figure-img" width="386"></p>
<p></p><figcaption class="figure-caption">Secretory Protein Binding Illustration</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Secretory protein folding</strong> is a complex process that occurs within the endoplasmic reticulum (ER) of eukaryotic cells. As proteins destined for secretion are synthesized by ribosomes attached to the rough ER, they undergo a series of folding and modification steps to attain their native conformations. Several key molecular players are involved in this process, including BiP (binding protein), Hsp70 chaperone, and Protein disulfide isomerase (PDI).</p>
<p><strong>BiP</strong>, also known as GRP78, is an ER-resident chaperone protein that plays a central role in protein folding and quality control within the ER lumen. It interacts with newly synthesized polypeptide chains to prevent misfolding and aggregation, guiding them along the folding pathway. BiP also assists in the translocation of proteins across the ER membrane and facilitates the assembly of multiprotein complexes.</p>
<p>Hsp70 chaperone, similar to BiP, is involved in the folding of secretory proteins within the ER. It interacts with exposed hydrophobic regions on nascent polypeptide chains, preventing nonproductive interactions and promoting correct folding. Hsp70’s activity is regulated by ATP hydrolysis and is coordinated with other ER-resident chaperones to ensure efficient protein folding.</p>
<p><strong>Protein disulfide isomerase (PDI)</strong> is an enzyme that catalyzes the formation and breakage of disulfide (S-S) bonds within secretory proteins. Disulfide bonds play a crucial role in stabilizing protein structures, particularly in secretory and membrane proteins. PDI helps to ensure the proper formation of disulfide bonds, which are essential for protein stability and function. Additionally, PDI can catalyze the rearrangement of incorrect disulfide bonds, ensuring that secretory proteins attain their correct native conformations.</p>
</section>
<section id="protein-glycosylation" class="level3" data-number="4.2.6">
<h3 data-number="4.2.6" class="anchored" data-anchor-id="protein-glycosylation"><span class="header-section-number">4.2.6</span> Protein Glycosylation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/protein_glyco.png" class="img-fluid figure-img" width="308"></p>
<p></p><figcaption class="figure-caption">Protein Glycosylation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Protein glycosylation is a common post-translational modification that occurs in the endoplasmic reticulum (ER) of eukaryotic cells. This process involves the addition of sugar molecules, such as glucose, mannose, and N-acetylglucosamine, to specific amino acid residues on nascent polypeptide chains as they are being synthesized by ribosomes attached to the ER membrane. The process of protein glycosylation in the ER involves several steps:</p>
<ol type="1">
<li><p><strong>Enzymatic addition of a pre-formed oligosaccharide</strong>: The initial step of protein glycosylation involves the transfer of a pre-formed oligosaccharide (a short sugar chain) from a lipid carrier molecule to specific asparagine residues on the nascent polypeptide chain. This process is catalyzed by an enzyme complex known as the oligosaccharyltransferase (OST) complex, which is embedded in the ER membrane.</p></li>
<li><p><strong>Modification and processing of the glycan chain</strong>: After the oligosaccharide is transferred to the protein, it undergoes further modifications and processing steps within the ER lumen. These modifications include trimming of the sugar chain by specific enzymes and the addition of additional sugar residues to form complex glycan structures.</p></li>
<li><p><strong>Quality control and protein folding</strong>: During or after glycosylation, the newly synthesized protein undergoes quality control checks to ensure proper folding and assembly. Chaperone proteins such as BiP and lectin-like chaperones assist in protein folding and recognize misfolded or improperly glycosylated proteins, targeting them for degradation if necessary.</p></li>
<li><p><strong>Glycoprotein trafficking</strong>: Once properly folded and glycosylated, the glycoproteins are transported from the ER to the Golgi apparatus for further processing and sorting. In the Golgi, additional modifications may occur, and glycoproteins are sorted for secretion or delivery to specific cellular compartments.</p></li>
</ol>
<section id="role-of-glycosylation-during-er-protein-folding" class="level4" data-number="4.2.6.1">
<h4 data-number="4.2.6.1" class="anchored" data-anchor-id="role-of-glycosylation-during-er-protein-folding"><span class="header-section-number">4.2.6.1</span> Role of Glycosylation During ER Protein Folding</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/calnexin.png" class="img-fluid figure-img" width="402"></p>
<p></p><figcaption class="figure-caption">Role of Glycosylation in ER Protein Folding</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Glycosylation plays a crucial role in protein folding within the endoplasmic reticulum (ER), and calnexin is a key player in this process. <strong>Calnexin</strong> is a type I integral membrane protein located in the ER membrane and functions as a lectin, specifically a carbohydrate-binding protein.</p>
<p>During protein synthesis, nascent polypeptide chains are co-translationally glycosylated by the addition of a pre-formed oligosaccharide chain. This glycosylation is essential for proper protein folding and quality control. Calnexin recognizes specific monosaccharides, such as glucose and mannose, present on the glycosylated proteins.</p>
<p>Calnexin interacts with newly synthesized glycoproteins that have not yet folded correctly. By binding to the oligosaccharide chains on these proteins, calnexin helps to stabilize them and prevent premature degradation. This interaction provides a quality control mechanism, allowing the glycoproteins to undergo proper folding and assembly.</p>
<p>Furthermore, calnexin collaborates with other chaperones and folding enzymes within the ER, such as BiP (binding immunoglobulin protein) and protein disulfide isomerase (PDI), to facilitate protein folding. Together, these ER-resident proteins ensure that glycosylated proteins attain their native conformations and are properly folded before being transported to their final destinations within the cell.</p>
</section>
</section>
<section id="erad-substrate-recognition" class="level3" data-number="4.2.7">
<h3 data-number="4.2.7" class="anchored" data-anchor-id="erad-substrate-recognition"><span class="header-section-number">4.2.7</span> ERAD Substrate Recognition</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/erad.png" class="img-fluid figure-img" width="416"></p>
<p></p><figcaption class="figure-caption">ERAD Pathway Mechanism</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>ERAD (Endoplasmic Reticulum-Associated Degradation)</strong> is a cellular quality control mechanism responsible for identifying and targeting misfolded or unassembled proteins in the endoplasmic reticulum (ER) for degradation. Substrate recognition is a crucial step in the ERAD process, ensuring that only aberrant proteins are targeted for degradation while properly folded proteins are spared.</p>
<p>The recognition of ERAD substrates involves multiple checkpoints and mechanisms:</p>
<ol type="1">
<li><p><strong>Quality control during protein folding</strong>: The ER contains a network of chaperones and folding enzymes that assist in the folding of nascent proteins. Properly folded proteins are able to escape recognition by ERAD machinery and proceed to their intended destinations. However, misfolded or unassembled proteins are retained in the ER and subjected to further scrutiny.</p></li>
<li><p><strong>Glycan-dependent recognition</strong>: Many ERAD substrates bear characteristic glycan modifications that serve as signals for recognition by ERAD machinery. Lectins, such as calnexin and calreticulin, recognize these glycan structures and facilitate the interaction between misfolded proteins and ERAD components.</p></li>
<li><p><strong>Ubiquitin-dependent recognition</strong>: Misfolded or unassembled proteins are tagged with ubiquitin, a small protein that serves as a signal for degradation by the proteasome. This ubiquitination process is mediated by ER-resident E3 ubiquitin ligases, which specifically recognize and target ERAD substrates for ubiquitination.</p></li>
<li><p><strong>ERAD receptors</strong>: ERAD substrates are recognized and bound by ER-resident receptors that bridge the interaction between the misfolded protein and the ERAD machinery. These receptors contain specific domains that recognize either the glycan modifications or the ubiquitin chains on ERAD substrates. Examples of ERAD receptors include Derlin proteins and Hrd1 complex components.</p></li>
<li><p><strong>ER quality control checkpoints</strong>: The ER contains quality control checkpoints that monitor the folding status of proteins. If a protein fails to fold properly or assemble into its functional form, it is recognized by ER quality control sensors, leading to its retrotranslocation from the ER lumen to the cytosol for degradation via ERAD.</p></li>
</ol>
</section>
</section>
<section id="cellular-responses-to-proteotoxic-stresses" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="cellular-responses-to-proteotoxic-stresses"><span class="header-section-number">4.3</span> Cellular Responses to Proteotoxic Stresses</h2>
<section id="proteostasis-network" class="level3" data-number="4.3.1">
<h3 data-number="4.3.1" class="anchored" data-anchor-id="proteostasis-network"><span class="header-section-number">4.3.1</span> Proteostasis Network</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/network.png" class="img-fluid figure-img" width="442"></p>
<p></p><figcaption class="figure-caption">Proteostasis Network</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>Proteostasis Network</strong> refers to the intricate system within cells that regulates protein homeostasis, ensuring that proteins are properly folded, assembled, trafficked, and degraded. Maintaining proteostasis is essential for cellular health and function, as proteins are involved in virtually every aspect of cellular processes, from structural support to enzymatic reactions and signaling pathways.</p>
<p>The Proteostasis Network consists of various molecular components and pathways that work together to maintain protein quality control:</p>
<ol type="1">
<li><p><strong>Chaperones and Folding Factors</strong>: Molecular chaperones, such as Hsp70 and Hsp90, assist in protein folding, preventing misfolding and aggregation. They guide nascent polypeptide chains to their proper three-dimensional structures and help proteins navigate through folding intermediates. Folding factors, such as protein disulfide isomerases (PDIs), aid in the formation of disulfide bonds, a crucial step in protein folding.</p></li>
<li><p><strong>Protein Degradation Pathways</strong>: Misfolded or damaged proteins are targeted for degradation by proteolytic systems such as the ubiquitin-proteasome system (UPS) and autophagy-lysosome pathway. The UPS involves the tagging of proteins with ubiquitin molecules, marking them for degradation by the proteasome. Autophagy sequesters cytoplasmic components, including protein aggregates, into autophagosomes for delivery to lysosomes, where they are degraded.</p></li>
<li><p><strong>Quality Control Mechanisms</strong>: Cells have quality control mechanisms that monitor protein folding and assembly. If a protein fails to fold properly or assemble correctly, it may be targeted for degradation or undergo retrotranslocation from the ER (endoplasmic reticulum) to the cytosol for ER-associated degradation (ERAD).</p></li>
<li><p><strong>Regulatory Pathways</strong>: Various signaling pathways and transcriptional regulators modulate the expression of chaperones, folding factors, and components of protein degradation pathways in response to cellular stress or changes in protein homeostasis. These regulatory mechanisms help cells adapt to changing conditions and maintain proteostasis under different physiological states.</p></li>
<li><p><strong>Organelle-Specific Proteostasis</strong>: Different organelles within the cell, such as the ER, mitochondria, and nucleus, have specialized proteostasis networks tailored to their specific needs. For example, the ER is responsible for folding and processing secretory and membrane proteins, while the mitochondria have their own chaperones and quality control mechanisms to maintain mitochondrial protein homeostasis.</p></li>
</ol>
</section>
<section id="heat-shock-response" class="level3" data-number="4.3.2">
<h3 data-number="4.3.2" class="anchored" data-anchor-id="heat-shock-response"><span class="header-section-number">4.3.2</span> Heat Shock Response</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/heat_shock.png" class="img-fluid figure-img" width="466"></p>
<p></p><figcaption class="figure-caption">Heat Shock Protein Illustration</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>Heat Shock Response (HSR)</strong> is a cellular defense mechanism that is activated in response to elevated temperatures and other stressors, such as oxidative stress, heavy metals, and toxins. The primary function of the HSR is to protect cells from damage caused by these stressors and to restore cellular homeostasis.</p>
<p>When cells are exposed to stress, heat shock transcription factors (HSFs), particularly <strong>HSF1</strong>, are activated. HSF1 undergoes a conformational change, leading to its trimerization and translocation to the nucleus. Once in the nucleus, HSF1 binds to specific DNA sequences known as heat shock elements (HSEs) located in the promoters of heat shock genes.</p>
<p>Activation of HSF1 leads to the transcriptional upregulation of heat shock proteins (HSPs), a group of molecular chaperones that play essential roles in protein folding, stabilization, and degradation. HSPs help to protect cells from stress-induced damage by assisting in the refolding of denatured proteins, preventing protein aggregation, and facilitating the degradation of irreversibly damaged proteins.</p>
<p>The induction of HSP expression by the HSR is a highly conserved and tightly regulated process across species, from bacteria to humans. HSPs are classified into families based on their molecular weight, including HSP70, HSP90, HSP60, and small HSPs (sHSPs).</p>
<p>In addition to their roles in protein homeostasis, HSPs have been implicated in various cellular processes, including signal transduction, cell cycle regulation, and apoptosis. They also play important roles in development, immunity, and disease.</p>
<section id="hsf1-in-cancer-and-neurodegenerative-diseases" class="level4" data-number="4.3.2.1">
<h4 data-number="4.3.2.1" class="anchored" data-anchor-id="hsf1-in-cancer-and-neurodegenerative-diseases"><span class="header-section-number">4.3.2.1</span> HSF1 in Cancer and Neurodegenerative Diseases?</h4>
<p><strong>Tumorigenesis:</strong> HSF1 contributes to tumorigenesis by promoting cancer cell survival and proliferation through the regulation of heat shock proteins (HSPs). Elevated levels of HSPs induced by HSF1 activation confer a survival advantage to cancer cells by facilitating protein folding, inhibiting apoptosis, and enhancing cell migration and invasion.</p>
<p><strong>Chemo- and Radiotherapy Resistance:</strong> HSF1 activation can lead to resistance to chemotherapy and radiation therapy in cancer cells. The upregulation of HSPs induced by HSF1 activation can protect cancer cells from the cytotoxic effects of these treatments, contributing to treatment failure and disease recurrence.</p>
<p><strong>Metastasis:</strong> HSF1 may also play a role in cancer metastasis by promoting epithelial-to-mesenchymal transition (EMT), a process that enhances cancer cell migration and invasion. HSF1-mediated upregulation of HSPs can facilitate EMT-associated changes in gene expression and cellular morphology, promoting metastatic spread.</p>
<p><strong>Protein Misfolding and Aggregation:</strong> HSF1 is activated in response to protein misfolding and aggregation, characteristic features of neurodegenerative diseases such as Alzheimer’s, Parkinson’s, and Huntington’s diseases. HSF1 activation leads to the upregulation of HSPs, which can help to refold or degrade misfolded proteins, thereby mitigating their toxic effects.</p>
<p><strong>Neuronal Protection:</strong> Activation of HSF1 and induction of HSP expression provide neuroprotection in experimental models of neurodegenerative diseases. By promoting protein homeostasis and reducing cellular stress, HSPs can help to preserve neuronal function and viability in the face of disease-related insults.</p>
<p><strong>Potential Therapeutic Target:</strong> Targeting HSF1 or its downstream effectors, such as HSPs, has emerged as a potential therapeutic strategy for neurodegenerative diseases. Modulation of HSF1 activity may help to enhance cellular proteostasis and alleviate disease-associated pathology.</p>
</section>
</section>
<section id="unfolded-protein-response" class="level3" data-number="4.3.3">
<h3 data-number="4.3.3" class="anchored" data-anchor-id="unfolded-protein-response"><span class="header-section-number">4.3.3</span> Unfolded Protein Response</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/upr.png" class="img-fluid figure-img" width="348"></p>
<p></p><figcaption class="figure-caption">Illustration of the UPR</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The Unfolded Protein Response (UPR) is a cellular stress response pathway activated in response to the accumulation of unfolded or misfolded proteins within the endoplasmic reticulum (ER), a cellular organelle involved in protein synthesis and folding. The UPR plays a critical role in maintaining protein homeostasis, or proteostasis, by alleviating ER stress and restoring normal cellular function.</p>
<p>When unfolded or misfolded proteins accumulate in the ER, they trigger the activation of three main transmembrane proteins: inositol-requiring enzyme 1 (IRE1), protein kinase RNA-like ER kinase (PERK), and activating transcription factor 6 (ATF6). These sensors initiate signaling cascades that culminate in a coordinated cellular response aimed at reducing ER stress and promoting cell survival.</p>
<ol type="1">
<li><p><strong>IRE1 Pathway</strong>: Upon activation, IRE1 splices an mRNA encoding the transcription factor X-box binding protein 1 (XBP1), leading to the production of an active transcription factor called spliced XBP1 (XBP1s). XBP1s translocates to the nucleus and activates the transcription of genes involved in ER-associated degradation (ERAD), protein folding, and lipid biosynthesis, promoting ER function and protein folding capacity.</p></li>
<li><p><strong>PERK Pathway</strong>: PERK activation leads to the phosphorylation of eukaryotic translation initiation factor 2 alpha (eIF2α), resulting in global attenuation of protein synthesis. This temporary reduction in protein synthesis alleviates the burden on the ER, allowing cells to cope with ER stress. Additionally, PERK activation induces the expression of ATF4, a transcription factor that regulates the expression of genes involved in antioxidant defense, amino acid metabolism, and apoptosis.</p></li>
<li><p><strong>ATF6 Pathway</strong>: ATF6 is transported to the Golgi apparatus upon ER stress, where it undergoes proteolytic cleavage to generate an active transcription factor fragment. This fragment translocates to the nucleus and activates the expression of genes encoding ER chaperones, such as BiP/GRP78, and other proteins involved in protein folding and ERAD.</p></li>
</ol>
<p>The combined actions of these UPR pathways help to restore ER homeostasis by increasing the capacity of the ER to fold and degrade proteins, reducing the load of unfolded proteins, and promoting cell survival. However, if ER stress is severe or prolonged, the UPR can also induce apoptosis to eliminate irreversibly damaged cells and maintain overall tissue integrity.</p>
<section id="splicing-of-xbp1-mrna" class="level4" data-number="4.3.3.1">
<h4 data-number="4.3.3.1" class="anchored" data-anchor-id="splicing-of-xbp1-mrna"><span class="header-section-number">4.3.3.1</span> Splicing of XBP1 mRNA</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/xbp1_mrna.png" class="img-fluid figure-img" width="456"></p>
<p></p><figcaption class="figure-caption">Regulating the Splicing of XBP1 mRNA</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The splicing of X-box binding protein 1 (XBP1) mRNA is a critical step in the activation of the unfolded protein response (UPR), particularly the IRE1 pathway. This process, known as <strong>regulated splicing</strong>, is tightly regulated and plays a key role in the cellular response to endoplasmic reticulum (ER) stress.</p>
<p>When unfolded or misfolded proteins accumulate in the ER, the ER stress sensor protein inositol-requiring enzyme 1 (IRE1) becomes activated. IRE1 is a transmembrane protein located in the ER membrane. Upon activation, IRE1 undergoes oligomerization and autophosphorylation, leading to its activation as an endoribonuclease.</p>
<p>IRE1 then catalyzes the unconventional splicing of XBP1 mRNA, a process that removes a 26-nucleotide intron from the mRNA. This splicing event generates a shorter mRNA transcript, referred to as spliced XBP1 (XBP1s). XBP1s encodes a transcription factor that is a potent activator of genes involved in ER-associated degradation (ERAD), protein folding, and lipid biosynthesis.</p>
<p>The splicing of XBP1 mRNA is a highly regulated process that is specific to ER stress conditions. Under normal conditions, XBP1 mRNA remains unspliced, and the protein encoded by unspliced XBP1 (XBP1u) does not have transcriptional activity. However, upon ER stress, IRE1-mediated splicing of XBP1 mRNA is rapidly induced, leading to the production of XBP1s and subsequent activation of UPR target genes.</p>
<p>The activation of XBP1s and its downstream target genes is critical for restoring ER homeostasis and promoting cell survival during ER stress. XBP1s regulates the expression of genes involved in protein folding, ERAD, and lipid metabolism, thereby increasing the capacity of the ER to handle the influx of misfolded proteins and alleviate ER stress.</p>
</section>
<section id="perk-activation" class="level4" data-number="4.3.3.2">
<h4 data-number="4.3.3.2" class="anchored" data-anchor-id="perk-activation"><span class="header-section-number">4.3.3.2</span> PERK Activation</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week4/perk_activation.png" class="img-fluid figure-img" width="197"></p>
<p></p><figcaption class="figure-caption">Activation of PERK</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Protein kinase RNA-like ER kinase (PERK)</strong> is a key component of the unfolded protein response (UPR) pathway, activated in response to endoplasmic reticulum (ER) stress. One of the main functions of PERK activation is to suppress global translation initiation, a process essential for protein synthesis. This translational attenuation helps to alleviate the burden on the ER by reducing the influx of newly synthesized proteins into the ER lumen, where protein folding occurs.</p>
<p>Upon activation, PERK phosphorylates the alpha subunit of eukaryotic translation initiation factor 2 (eIF2α), a critical component of the translation initiation complex. Phosphorylation of eIF2α inhibits its activity, preventing the formation of the ternary complex required for translation initiation. As a result, global protein synthesis is attenuated, reducing the overall protein load on the ER and allowing cells to cope with ER stress.</p>
<p>While global translation is suppressed, the translation of specific mRNAs is selectively enhanced or maintained during PERK activation. For example, the mRNA encoding activating transcription factor 4 (ATF4) contains specific regulatory elements in its 5’ untranslated region (UTR), known as upstream open reading frames (uORFs), which allow for preferential translation even under conditions of eIF2α phosphorylation. ATF4 is a transcription factor that regulates the expression of genes involved in various cellular processes, including antioxidant defense, amino acid metabolism, and apoptosis.</p>


</section>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week3.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week5.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>