<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 6&nbsp; Autophagy of Protein Aggregates and Mitochondria</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week7.html" rel="next">
<link href="../chapters/week5.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Plant Genome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Bioenergy and Biomaterials</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Secondary Metabolism in Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Plants as Medicine</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#autophagy-mechanisms" id="toc-autophagy-mechanisms" class="nav-link active" data-scroll-target="#autophagy-mechanisms"><span class="toc-section-number">6.1</span>  Autophagy Mechanisms</a>
  <ul class="collapse">
  <li><a href="#from-cytosol-to-lysosome" id="toc-from-cytosol-to-lysosome" class="nav-link" data-scroll-target="#from-cytosol-to-lysosome"><span class="toc-section-number">6.1.1</span>  From Cytosol to Lysosome</a></li>
  <li><a href="#autophagy-in-yeast" id="toc-autophagy-in-yeast" class="nav-link" data-scroll-target="#autophagy-in-yeast"><span class="toc-section-number">6.1.2</span>  Autophagy in Yeast</a></li>
  <li><a href="#autophagosome" id="toc-autophagosome" class="nav-link" data-scroll-target="#autophagosome"><span class="toc-section-number">6.1.3</span>  Autophagosome</a></li>
  <li><a href="#selective-and-non-selective-autophagy" id="toc-selective-and-non-selective-autophagy" class="nav-link" data-scroll-target="#selective-and-non-selective-autophagy"><span class="toc-section-number">6.1.4</span>  Selective and Non-Selective Autophagy</a></li>
  <li><a href="#phospholipids-in-bilayer" id="toc-phospholipids-in-bilayer" class="nav-link" data-scroll-target="#phospholipids-in-bilayer"><span class="toc-section-number">6.1.5</span>  Phospholipids in Bilayer</a></li>
  <li><a href="#autophagy-regulation" id="toc-autophagy-regulation" class="nav-link" data-scroll-target="#autophagy-regulation"><span class="toc-section-number">6.1.6</span>  Autophagy Regulation</a></li>
  </ul></li>
  <li><a href="#autophagy-and-neurodegeneration" id="toc-autophagy-and-neurodegeneration" class="nav-link" data-scroll-target="#autophagy-and-neurodegeneration"><span class="toc-section-number">6.2</span>  Autophagy and Neurodegeneration</a>
  <ul class="collapse">
  <li><a href="#if-mitochondria-do-not-fuse-or-divide" id="toc-if-mitochondria-do-not-fuse-or-divide" class="nav-link" data-scroll-target="#if-mitochondria-do-not-fuse-or-divide"><span class="toc-section-number">6.2.1</span>  If Mitochondria Do Not Fuse or Divide</a></li>
  <li><a href="#drp1-mediated-fission" id="toc-drp1-mediated-fission" class="nav-link" data-scroll-target="#drp1-mediated-fission"><span class="toc-section-number">6.2.2</span>  Drp1-Mediated Fission</a></li>
  <li><a href="#mitochondrial-fusion" id="toc-mitochondrial-fusion" class="nav-link" data-scroll-target="#mitochondrial-fusion"><span class="toc-section-number">6.2.3</span>  Mitochondrial Fusion</a></li>
  <li><a href="#pink1-parkin-and-mitophagy" id="toc-pink1-parkin-and-mitophagy" class="nav-link" data-scroll-target="#pink1-parkin-and-mitophagy"><span class="toc-section-number">6.2.4</span>  PINK1, Parkin, and Mitophagy</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week6.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<section id="autophagy-mechanisms" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="autophagy-mechanisms"><span class="header-section-number">6.1</span> Autophagy Mechanisms</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/autophagy.png" class="img-fluid figure-img" width="365"></p>
<p></p><figcaption class="figure-caption">Autophagy Mechanism Illustrated</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Autophagy</strong> is a cellular process responsible for the degradation and recycling of damaged or unnecessary cellular components, including protein aggregates and dysfunctional organelles such as mitochondria. Here’s how autophagy mechanisms work in directing degradation targets to lysosomes:</p>
<ol type="1">
<li><p><strong>Initiation of Autophagy:</strong> Autophagy is initiated in response to various cellular stresses, such as nutrient deprivation, oxidative stress, or protein aggregation. The initiation phase involves the activation of key autophagy-related proteins, including the ULK1 (unc-51 like autophagy activating kinase 1) complex and the class III phosphatidylinositol 3-kinase (PI3K) complex.</p></li>
<li><p><strong>Formation of Autophagosome:</strong> Once initiated, autophagy progresses through several steps, including the formation of a specialized double-membrane structure called the autophagosome. The autophagosome sequesters cellular components targeted for degradation, including protein aggregates and dysfunctional organelles such as mitochondria.</p></li>
<li><p><strong>Targeting Degradation Targets:</strong> Selective autophagy mechanisms enable the specific targeting of degradation targets to autophagosomes. In the case of protein aggregates, selective autophagy receptors, such as p62/SQSTM1 (sequestosome 1), recognize ubiquitinated proteins and bind to both the cargo and the autophagosomal membrane, facilitating their engulfment into autophagosomes. Similarly, dysfunctional mitochondria are targeted for degradation through selective autophagy receptors, such as NIX (also known as BNIP3L) and FUNDC1, which recognize damaged mitochondria and recruit them to autophagosomes.</p></li>
<li><p><strong>Fusion with Lysosomes:</strong> Following sequestration of cargo, autophagosomes undergo fusion with lysosomes to form autolysosomes. Lysosomes contain acidic hydrolases that degrade the engulfed cargo, resulting in the breakdown of proteins, lipids, nucleic acids, and other cellular components. This process releases the breakdown products, such as amino acids and fatty acids, for reuse by the cell.</p></li>
<li><p><strong>Degradation and Recycling:</strong> Within autolysosomes, the cargo undergoes degradation by lysosomal enzymes. Proteins are degraded into amino acids, which can be recycled for protein synthesis or used as an energy source. Lipids are broken down into fatty acids, while nucleic acids are degraded into nucleotides. This degradation process ensures the removal of damaged or unnecessary cellular components and contributes to cellular homeostasis and survival.</p></li>
</ol>
<section id="from-cytosol-to-lysosome" class="level3" data-number="6.1.1">
<h3 data-number="6.1.1" class="anchored" data-anchor-id="from-cytosol-to-lysosome"><span class="header-section-number">6.1.1</span> From Cytosol to Lysosome</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/cytosol.png" class="img-fluid figure-img" width="412"></p>
<p></p><figcaption class="figure-caption">From Cytosol to Lysosome</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The journey of autophagy from the cytosol to the lysosome involves several key steps:</p>
<ol type="1">
<li><p><strong>Initiation:</strong> Autophagy is initiated in response to various cellular stresses, such as nutrient deprivation, oxidative stress, or protein aggregation. The initiation phase involves the activation of autophagy-related proteins, including the ULK1 (unc-51 like autophagy activating kinase 1) complex and the class III phosphatidylinositol 3-kinase (PI3K) complex.</p></li>
<li><p><strong>Formation of Phagophore:</strong> Once initiated, autophagy progresses with the formation of a phagophore, a crescent-shaped membrane structure derived from the endoplasmic reticulum (ER) or other membrane sources. The phagophore expands and engulfs cellular components targeted for degradation, including protein aggregates and dysfunctional organelles.</p></li>
<li><p><strong>Formation of Autophagosome:</strong> The edges of the phagophore seal to form a double-membrane vesicle called the autophagosome. The autophagosome encloses the cargo, sequestering it from the cytoplasm and creating a protective environment for degradation.</p></li>
<li><p><strong>Lysosomal Fusion:</strong> The autophagosome then undergoes fusion with lysosomes, which are membrane-bound organelles containing acidic hydrolases. This fusion forms a structure called the autolysosome, where degradation occurs. Lysosomes provide the enzymes necessary for breaking down the engulfed cargo.</p></li>
<li><p><strong>Cargo Degradation:</strong> Within the autolysosome, the cargo is exposed to the acidic pH and the enzymatic activity of lysosomal hydrolases. Proteins are degraded into amino acids, lipids into fatty acids, and nucleic acids into nucleotides. These breakdown products are then released back into the cytoplasm for reuse by the cell.</p></li>
<li><p><strong>Recycling and Reuse:</strong> The degraded components are recycled and can be used for various cellular processes, such as protein synthesis, energy production, or the maintenance of cellular membranes. This recycling process ensures the efficient utilization of cellular resources and contributes to cellular homeostasis.</p></li>
</ol>
</section>
<section id="autophagy-in-yeast" class="level3" data-number="6.1.2">
<h3 data-number="6.1.2" class="anchored" data-anchor-id="autophagy-in-yeast"><span class="header-section-number">6.1.2</span> Autophagy in Yeast</h3>
<section id="atg-genes" class="level4" data-number="6.1.2.1">
<h4 data-number="6.1.2.1" class="anchored" data-anchor-id="atg-genes"><span class="header-section-number">6.1.2.1</span> ATG Genes</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/atg_genes.png" class="img-fluid figure-img" width="510"></p>
<p></p><figcaption class="figure-caption">Discovery of <em>ATG</em> Genes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Autophagy was first discovered in yeast, particularly Saccharomyces cerevisiae, by Dr.&nbsp;Yoshinori Ohsumi, who was awarded the Nobel Prize in Physiology or Medicine in 2016 for his groundbreaking work. Dr.&nbsp;Ohsumi’s research led to the identification of a set of genes essential for autophagy, known as autophagy-related genes or ATG genes.</p>
<p>The discovery of ATG genes provided key insights into the molecular mechanisms underlying autophagy and paved the way for understanding the process in other organisms, including humans. By systematically studying yeast mutants defective in autophagy, Dr.&nbsp;Ohsumi and his team identified a series of genes involved in autophagosome formation, cargo recognition, and lysosomal degradation.</p>
<p>Some of the key ATG genes identified in yeast include ATG1, ATG5, ATG6 (also known as VPS30), ATG8, and ATG12. These genes encode proteins that play crucial roles in different stages of the autophagy pathway, from the initiation of autophagosome formation to the fusion of autophagosomes with lysosomes.</p>
<p><strong>ATG1</strong> is a protein kinase involved in the initiation of autophagy, while ATG5 and ATG12 are involved in the conjugation of Atg8 (LC3 in mammals) to phosphatidylethanolamine, a critical step in autophagosome formation. ATG6/VPS30 is a subunit of the phosphatidylinositol 3-kinase (PI3K) complex, which regulates the nucleation of autophagosomal membranes.</p>
</section>
</section>
<section id="autophagosome" class="level3" data-number="6.1.3">
<h3 data-number="6.1.3" class="anchored" data-anchor-id="autophagosome"><span class="header-section-number">6.1.3</span> Autophagosome</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/autophagosome.png" class="img-fluid figure-img" width="497"></p>
<p></p><figcaption class="figure-caption">Autophagosome Formation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>autophagosome</strong> is a double-membraned vesicle that plays a central role in the process of autophagy, serving as the primary structure for sequestering and transporting cellular components destined for degradation within the lysosome. Here’s a closer look at the structure and function of the autophagosome:</p>
<ol type="1">
<li><p><strong>Double-Membraned Structure:</strong> The autophagosome is characterized by its double-membraned structure, comprising an inner membrane and an outer membrane. These membranes are derived from various cellular sources, including the endoplasmic reticulum (ER), Golgi apparatus, plasma membrane, and mitochondria, among others.</p></li>
<li><p><strong>Formation:</strong> The formation of autophagosomes begins with the nucleation of a small membrane structure, known as the phagophore or isolation membrane. The phagophore expands and engulfs cytoplasmic components, including protein aggregates, damaged organelles, and invading pathogens, to form the autophagosome.</p></li>
<li><p><strong>Sequestration of Cargo:</strong> As the phagophore expands, it sequesters cytoplasmic cargo within its lumen, encapsulating it within the double-membraned vesicle. This sequestration process allows for the selective targeting and isolation of cellular components destined for degradation, while excluding essential cellular structures from degradation.</p></li>
<li><p><strong>Maturation:</strong> Once formed, autophagosomes undergo maturation through a series of fusion events. They fuse with endosomes to form amphisomes, which subsequently fuse with lysosomes to form autolysosomes. These fusion events enable the delivery of autophagosomal cargo to lysosomes for degradation.</p></li>
<li><p><strong>Degradation of Cargo:</strong> Within autolysosomes, the acidic environment and the presence of lysosomal hydrolases facilitate the degradation of the sequestered cargo. Proteins, lipids, nucleic acids, and other cellular components are broken down into their constituent molecules, which can then be recycled by the cell.</p></li>
<li><p><strong>Role in Cellular Homeostasis:</strong> The autophagosome serves as a critical component of the cellular quality control system, allowing cells to eliminate damaged or unnecessary cellular components and maintain cellular homeostasis. Dysregulation of autophagosome formation and function has been implicated in various diseases, including neurodegenerative disorders, cancer, and metabolic diseases.</p></li>
</ol>
</section>
<section id="selective-and-non-selective-autophagy" class="level3" data-number="6.1.4">
<h3 data-number="6.1.4" class="anchored" data-anchor-id="selective-and-non-selective-autophagy"><span class="header-section-number">6.1.4</span> Selective and Non-Selective Autophagy</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/selective.png" class="img-fluid figure-img" width="507"></p>
<p></p><figcaption class="figure-caption">Selective and Non-Selective Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Autophagy is a highly regulated process that can be categorized into two main types based on the specificity of the cargo targeted for degradation: selective autophagy and non-selective autophagy.</p>
<ol type="1">
<li><strong>Selective Autophagy:</strong> Selective autophagy involves the targeted degradation of specific cellular components, such as damaged organelles, protein aggregates, or invading pathogens. Selective autophagy receptors recognize and bind to cargo destined for degradation, facilitating their engulfment into autophagosomes. Examples of selective autophagy include:
<ul>
<li><strong>Mitophagy:</strong> Selective degradation of damaged or dysfunctional mitochondria.</li>
<li><strong>Aggrephagy:</strong> Clearance of protein aggregates and inclusion bodies.</li>
<li><strong>Pexophagy:</strong> Elimination of peroxisomes.</li>
<li><strong>Lipophagy:</strong> Degradation of lipid droplets.</li>
<li><strong>Xenophagy:</strong> Removal of intracellular pathogens, such as bacteria or viruses.</li>
</ul></li>
<li><strong>Non-Selective Autophagy (Bulk Autophagy):</strong> Non-selective autophagy, also known as bulk autophagy, involves the indiscriminate degradation of cytoplasmic components to maintain cellular homeostasis during nutrient deprivation or other stress conditions. In non-selective autophagy, autophagosomes engulf cytoplasmic material randomly without specific cargo recognition. This form of autophagy is essential for recycling cellular constituents and generating nutrients and energy for cell survival during periods of stress.</li>
</ol>
<section id="types-of-autophagy" class="level4" data-number="6.1.4.1">
<h4 data-number="6.1.4.1" class="anchored" data-anchor-id="types-of-autophagy"><span class="header-section-number">6.1.4.1</span> Types of Autophagy</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/3_types.png" class="img-fluid figure-img" width="514"></p>
<p></p><figcaption class="figure-caption">Three Different Kinds of Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The three different types of autophagy are:</p>
<ol type="1">
<li><p><strong>Macroautophagy:</strong> Macroautophagy, often simply referred to as autophagy, involves the sequestration of cytosolic components within double-membraned organelles called autophagosomes. These autophagosomes then fuse with lysosomes to form autolysosomes, where the engulfed cargo is degraded by lysosomal enzymes. Macroautophagy plays a fundamental role in the turnover of cellular components and the maintenance of cellular homeostasis.</p></li>
<li><p><strong>Microautophagy:</strong> Microautophagy involves the direct engulfment of cytosolic components by lysosomes through invagination of the lysosomal membrane. Unlike macroautophagy, which involves the formation of autophagosomes, microautophagy occurs through direct membrane rearrangements at the lysosomal surface. Microautophagy contributes to the selective degradation of cytoplasmic constituents and plays a role in nutrient recycling and cellular quality control.</p></li>
<li><p><strong>Chaperone-Mediated Autophagy (CMA):</strong> Chaperone-Mediated Autophagy (CMA) is a selective form of autophagy that targets specific cytosolic proteins for degradation. In CMA, chaperone proteins such as heat shock cognate 70 (HSC-70) recognize and bind to target proteins bearing a specific targeting motif. The chaperone-protein complex then interacts with lysosomal-associated membrane protein-2A (LAMP2A) on the lysosomal membrane, leading to the translocation of the unfolded target protein into the lysosome for degradation. CMA is essential for the removal of damaged or misfolded proteins and plays a role in cellular quality control and protein homeostasis.</p></li>
</ol>
</section>
</section>
<section id="phospholipids-in-bilayer" class="level3" data-number="6.1.5">
<h3 data-number="6.1.5" class="anchored" data-anchor-id="phospholipids-in-bilayer"><span class="header-section-number">6.1.5</span> Phospholipids in Bilayer</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/phospholipid.png" class="img-fluid figure-img" width="414"></p>
<p></p><figcaption class="figure-caption">Phospholipid Structure</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Phospholipids</strong> are a crucial component of the lipid bilayer, the fundamental structure of cell membranes. Here’s how phospholipids contribute to the structure and function of the lipid bilayer:</p>
<ol type="1">
<li><p><strong>Basic Structure:</strong> Phospholipids consist of a hydrophilic (water-attracting) head region and two hydrophobic (water-repelling) tail regions. The hydrophilic head contains a phosphate group, which is polar and interacts with water molecules, while the hydrophobic tails consist of fatty acid chains that are nonpolar and repel water.</p></li>
<li><p><strong>Arrangement in the Bilayer:</strong> In the lipid bilayer, phospholipids arrange themselves in a double layer with their hydrophilic heads facing outward toward the aqueous environment (extracellular space or cytoplasm) and their hydrophobic tails facing inward, away from water. This arrangement creates a hydrophobic core within the membrane, which acts as a barrier to the passage of water-soluble molecules.</p></li>
<li><p><strong>Fluidity and Flexibility:</strong> Phospholipids confer fluidity and flexibility to the lipid bilayer. The presence of unsaturated fatty acid chains in some phospholipids introduces kinks in the hydrophobic tails, preventing them from packing tightly together. This results in a more fluid and flexible membrane that allows for lateral movement of phospholipids and membrane proteins.</p></li>
<li><p><strong>Selective Permeability:</strong> The lipid bilayer, composed primarily of phospholipids, exhibits selective permeability, allowing certain molecules to pass through while restricting the passage of others. Small, nonpolar molecules such as oxygen and carbon dioxide can diffuse freely through the lipid bilayer, while larger or charged molecules require specific transport proteins or channels for passage.</p></li>
</ol>
<section id="phosphatidylinositol" class="level4" data-number="6.1.5.1">
<h4 data-number="6.1.5.1" class="anchored" data-anchor-id="phosphatidylinositol"><span class="header-section-number">6.1.5.1</span> Phosphatidylinositol</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/phospho.png" class="img-fluid figure-img" width="466"></p>
<p></p><figcaption class="figure-caption">Figures Pertaining to Phosphatidylinositol</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Phosphatidylinositol (PI)</strong> is a type of phospholipid that plays essential roles in cell signaling, membrane structure, and various cellular processes. Here are some key aspects of phosphatidylinositol:</p>
<ol type="1">
<li><p><strong>Structure:</strong> Phosphatidylinositol consists of a glycerol backbone, two fatty acid chains, and a phosphate group linked to the inositol head group. The inositol head group can be phosphorylated at different positions, leading to the formation of various phosphoinositides.</p></li>
<li><p><strong>Cell Signaling:</strong> Phosphatidylinositol lipids are crucial components of cell signaling pathways. Phosphorylation of the inositol head group at the 3, 4, and 5 positions generates specific phosphoinositides, such as phosphatidylinositol 3-phosphate (PI3P), phosphatidylinositol 4,5-bisphosphate (PI(4,5)P2), and phosphatidylinositol 3,4,5-trisphosphate (PIP3). These phosphoinositides serve as signaling molecules that regulate diverse cellular processes, including cell growth, proliferation, survival, and vesicle trafficking.</p></li>
<li><p><strong>Membrane Trafficking:</strong> Phosphatidylinositol lipids play critical roles in membrane trafficking and vesicle formation. They act as docking sites for proteins involved in vesicle budding, fusion, and trafficking between cellular compartments, such as the endoplasmic reticulum (ER), Golgi apparatus, endosomes, and lysosomes.</p></li>
<li><p><strong>Cytoskeleton Regulation:</strong> Phosphatidylinositol lipids contribute to the regulation of the cytoskeleton, which is essential for cell shape, movement, and intracellular transport. Phosphoinositides interact with cytoskeletal proteins, such as actin-binding proteins and microtubule-associated proteins, to coordinate cytoskeletal dynamics and cellular processes such as cell migration and cytokinesis.</p></li>
<li><p><strong>Membrane Identity and Function:</strong> Phosphatidylinositol lipids help define the identity and function of cellular membranes. By interacting with specific proteins and other lipids, phosphoinositides contribute to membrane curvature, organization, and protein recruitment, thereby regulating membrane structure and function.</p></li>
<li><p><strong>Implications in Disease:</strong> Dysregulation of phosphatidylinositol signaling has been implicated in various human diseases, including cancer, neurodegenerative disorders, and metabolic diseases. Mutations in enzymes involved in phosphatidylinositol metabolism and signaling pathways can disrupt cellular homeostasis and contribute to disease pathogenesis.</p></li>
</ol>
</section>
</section>
<section id="autophagy-regulation" class="level3" data-number="6.1.6">
<h3 data-number="6.1.6" class="anchored" data-anchor-id="autophagy-regulation"><span class="header-section-number">6.1.6</span> Autophagy Regulation</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/autophagy_regulation.png" class="img-fluid figure-img" width="442"></p>
<p></p><figcaption class="figure-caption">Regulating Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The initiation of autophagy involves a complex series of molecular events orchestrated by several key proteins and signaling pathways. Here’s an overview of the mechanism of autophagy initiation:</p>
<ol type="1">
<li><p><strong>Activation of Autophagy-Related Proteins:</strong> Autophagy initiation is regulated by a conserved protein kinase complex called the ULK1 (unc-51 like autophagy activating kinase 1) complex, which consists of ULK1, FIP200, ATG13, and ATG101. Under nutrient-rich conditions, the mammalian target of rapamycin complex 1 (mTORC1) phosphorylates ULK1 and inhibits its activity, thereby suppressing autophagy initiation. However, during nutrient deprivation or other stress conditions, mTORC1 is inhibited, leading to the activation of the ULK1 complex.</p></li>
<li><p><strong>Nucleation of the Phagophore:</strong> Upon activation, the ULK1 complex phosphorylates and activates the class III phosphatidylinositol 3-kinase (PI3K) complex, which includes BECN1 (Beclin 1), VPS34 (vacuolar protein sorting 34), VPS15 (vacuolar protein sorting 15), and ATG14L. The activated PI3K complex generates phosphatidylinositol 3-phosphate (PI3P) on the endoplasmic reticulum (ER) membrane, promoting the nucleation of the phagophore or isolation membrane.</p></li>
<li><p><strong>Phagophore Expansion and Cargo Sequestration:</strong> The phagophore expands and engulfs cytoplasmic components, including protein aggregates, damaged organelles, and invading pathogens. This process involves the recruitment of ATG proteins, such as ATG12-ATG5-ATG16L1 complex, and the lipidation of LC3 (microtubule-associated protein 1A/1B-light chain 3) to form LC3-II, which is essential for phagophore elongation and cargo sequestration.</p></li>
<li><p><strong>Closure of the Autophagosome:</strong> As the phagophore expands and engulfs cytoplasmic cargo, it eventually closes to form a double-membraned vesicle called the autophagosome. The outer membrane of the autophagosome fuses with vesicles derived from the ER or other membrane sources, while the inner membrane sequesters the cargo destined for degradation.</p></li>
<li><p><strong>Fusion with Lysosomes:</strong> After closure, autophagosomes undergo fusion with lysosomes, forming autolysosomes. This fusion event is mediated by proteins such as Rab7 and SNARE proteins. The lysosomal hydrolases within autolysosomes degrade the sequestered cargo, releasing breakdown products for recycling and reuse by the cell.</p></li>
</ol>
<section id="pe-modification-during-autophagy" class="level4" data-number="6.1.6.1">
<h4 data-number="6.1.6.1" class="anchored" data-anchor-id="pe-modification-during-autophagy"><span class="header-section-number">6.1.6.1</span> PE Modification During Autophagy</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/pe_modification.png" class="img-fluid figure-img" width="498"></p>
<p></p><figcaption class="figure-caption">Modifying PE During Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The modification of Atg8 in yeast or LC3 in humans with phosphatidylethanolamine (PE) is a crucial step in the autophagy process. This modification plays a key role in autophagosome formation and cargo sequestration. Here’s how the PE modification of Atg8/LC3 occurs and its significance in autophagy:</p>
<ol type="1">
<li><p><strong>Conjugation Process:</strong> Atg8 in yeast or LC3 in humans undergo a conjugation process to become lipidated with phosphatidylethanolamine (PE). This process involves a series of enzymatic reactions mediated by several autophagy-related proteins. First, Atg8/LC3 is cleaved at its C-terminus by the cysteine protease Atg4, exposing a glycine residue. Then, the exposed glycine residue is conjugated to the amino group of PE through a ubiquitin-like conjugation system. The conjugation of Atg8/LC3 to PE is catalyzed by the sequential action of Atg7 (E1-like enzyme) and Atg3 (E2-like enzyme). This results in the formation of the lipidated form of Atg8/LC3, known as Atg8-PE or LC3-II, which is associated with the autophagosomal membrane.</p></li>
<li><p><strong>Membrane Association:</strong> The lipidation of Atg8/LC3 with PE facilitates its association with the autophagosomal membrane. Atg8-PE or LC3-II is recruited to the expanding phagophore, the precursor structure of the autophagosome, where it plays a critical role in membrane elongation and closure. The lipidated form of Atg8/LC3 anchors the autophagosomal membrane to the cytosolic cargo, facilitating cargo sequestration within the autophagosome.</p></li>
<li><p><strong>Cargo Sequestration:</strong> Atg8/LC3 lipidation with PE is essential for cargo sequestration during autophagy. The lipidated form of Atg8/LC3 acts as a molecular scaffold that binds to specific autophagy receptors, which in turn recognize and tether cargo molecules (such as protein aggregates, damaged organelles, or pathogens) to the autophagosomal membrane. This selective targeting and sequestration of cargo within the autophagosome are crucial for the degradation and recycling of cellular components.</p></li>
<li><p><strong>Autophagosome Maturation:</strong> The lipidation of Atg8/LC3 with PE also plays a role in autophagosome maturation and fusion with lysosomes. Once formed, autophagosomes undergo fusion with lysosomes to form autolysosomes, where the sequestered cargo is degraded by lysosomal enzymes. Atg8-PE or LC3-II remains associated with the autophagosomal membrane during this process and is eventually recycled upon autolysosome formation.</p></li>
</ol>
</section>
<section id="mechanism" class="level4" data-number="6.1.6.2">
<h4 data-number="6.1.6.2" class="anchored" data-anchor-id="mechanism"><span class="header-section-number">6.1.6.2</span> Mechanism</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/mechanism.png" class="img-fluid figure-img" width="449"></p>
<p></p><figcaption class="figure-caption">Mechanism of Autophagy Initiation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The initiation of autophagy involves various signaling pathways and regulatory proteins, one of which is the phosphatidylinositol 3-kinase (PI3K) complex. Here’s an overview of how PI3K contributes to autophagy initiation:</p>
<ol type="1">
<li><p><strong>Formation of the PI3K Complex:</strong> The PI3K complex involved in autophagy initiation consists of several subunits, including the class III PI3K, also known as VPS34 (vacuolar protein sorting 34), and its regulatory subunits such as VPS15 (vacuolar protein sorting 15) and Beclin 1 (BECN1). Under basal conditions or during nutrient-rich states, the activity of the PI3K complex is inhibited.</p></li>
<li><p><strong>Activation by Stress Signals:</strong> Autophagy initiation is often triggered by various stress signals, such as nutrient deprivation, oxidative stress, or accumulation of damaged proteins or organelles. These stress signals lead to the inhibition of the mTOR (mechanistic target of rapamycin) pathway, a negative regulator of autophagy, which in turn relieves the suppression of the PI3K complex.</p></li>
<li><p><strong>Phosphatidylinositol 3-Phosphate (PI3P) Production:</strong> Upon activation, the PI3K complex catalyzes the phosphorylation of phosphatidylinositol (PI) to generate phosphatidylinositol 3-phosphate (PI3P) on the membranes of various cellular compartments, including the endoplasmic reticulum (ER), Golgi apparatus, and endosomes. PI3P serves as a critical signaling lipid that recruits downstream autophagy-related proteins to initiate autophagosome formation.</p></li>
<li><p><strong>Recruitment of Autophagy Machinery:</strong> PI3P-enriched membranes act as nucleation sites for the assembly of the autophagy machinery. Autophagy-related proteins containing PI3P-binding domains, such as WIPI proteins (WD-repeat protein interacting with phosphoinositides), DFCP1 (double FYVE-containing protein 1), and FYCO1 (FYVE and coiled-coil domain-containing protein 1), are recruited to these membranes, where they initiate the formation of the phagophore, the precursor structure of the autophagosome.</p></li>
<li><p><strong>Phagophore Elongation and Cargo Sequestration:</strong> The recruitment of autophagy-related proteins to PI3P-enriched membranes promotes the elongation and expansion of the phagophore, leading to the formation of the autophagosomal membrane. Meanwhile, cargo molecules such as protein aggregates, damaged organelles, or intracellular pathogens are selectively recognized and sequestered within the growing autophagosomal membrane.</p></li>
<li><p><strong>Autophagosome Maturation and Fusion with Lysosomes:</strong> Following cargo sequestration, the autophagosome matures through membrane fusion events and eventually fuses with lysosomes, forming autolysosomes. Within autolysosomes, the sequestered cargo is degraded by lysosomal hydrolases, and the breakdown products are recycled back into the cytoplasm for reuse.</p></li>
</ol>
</section>
<section id="cargo-selection" class="level4" data-number="6.1.6.3">
<h4 data-number="6.1.6.3" class="anchored" data-anchor-id="cargo-selection"><span class="header-section-number">6.1.6.3</span> Cargo Selection</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/cargo_selection.png" class="img-fluid figure-img" width="323"></p>
<p></p><figcaption class="figure-caption">Cargo Selection</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Cargo selection</strong> in autophagy is a highly regulated process that ensures the selective degradation of specific cellular components, such as damaged organelles, protein aggregates, and intracellular pathogens, while preserving essential cellular structures. Here’s how cargo selection occurs in autophagy:</p>
<ol type="1">
<li><p><strong>Recognition by Autophagy Receptors:</strong> Cargo selection begins with the recognition of specific cellular components by autophagy receptors. These receptors contain specific domains that recognize and bind to the cargo molecules destined for degradation. Autophagy receptors often contain LC3-interacting region (LIR) motifs, which allow them to interact with Atg8/LC3 proteins on the autophagosomal membrane.</p></li>
<li><p><strong>Targeting Signals and Tags:</strong> Cargo molecules destined for degradation may contain specific targeting signals or post-translational modifications that mark them for recognition by autophagy receptors. For example, damaged organelles may expose specific surface markers or undergo modifications that make them recognizable by autophagy receptors.</p></li>
<li><p><strong>Interaction with Autophagy Machinery:</strong> Once bound to autophagy receptors, cargo molecules are targeted to the expanding phagophore, the precursor structure of the autophagosome. The autophagy machinery, including Atg proteins and the lipidated form of Atg8/LC3 (such as Atg8-PE or LC3-II), facilitates the sequestration of cargo within the growing autophagosomal membrane.</p></li>
<li><p><strong>Cargo Sequestration and Encapsulation:</strong> The autophagosomal membrane engulfs the cargo molecules, forming a double-membraned vesicle known as the autophagosome. The lipidated form of Atg8/LC3 anchors the autophagosomal membrane to the cargo, facilitating cargo sequestration within the autophagosome. This process ensures that only selected cargo molecules are enclosed within the autophagosomal vesicle.</p></li>
<li><p><strong>Selective Degradation in Autolysosomes:</strong> Autophagosomes subsequently fuse with lysosomes, forming autolysosomes, where the sequestered cargo is degraded by lysosomal hydrolases. The degradation products are then recycled back into the cytoplasm for reuse by the cell. Selective cargo degradation ensures the removal of damaged or unnecessary cellular components while preserving essential cellular structures and functions.</p></li>
</ol>
</section>
<section id="receptors" class="level4" data-number="6.1.6.4">
<h4 data-number="6.1.6.4" class="anchored" data-anchor-id="receptors"><span class="header-section-number">6.1.6.4</span> Receptors</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/receptors.png" class="img-fluid figure-img" width="430"></p>
<p></p><figcaption class="figure-caption">Receptors in Autophagy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Aggrephagy and mitophagy are specialized forms of autophagy that target protein aggregates and damaged mitochondria, respectively, for degradation. The selective recognition and targeting of cargo in these processes involve specific receptors that play crucial roles in cargo recognition and autophagosome formation. Here are the receptors involved in aggrephagy and mitophagy:</p>
<ol type="1">
<li><p><strong>Receptors in Aggrephagy:</strong> Aggrephagy refers to the selective degradation of protein aggregates by autophagy. Several receptors have been identified to mediate the recognition and targeting of protein aggregates for autophagic degradation. Some of the key receptors involved in aggrephagy include:</p>
<ul>
<li><p><strong>p62/SQSTM1 (Sequestosome 1):</strong> p62 is a multifunctional protein that contains several domains, including an LC3-interacting region (LIR) motif. p62 binds to ubiquitinated protein aggregates via its ubiquitin-binding domain (UBA) and interacts with LC3 on the autophagosomal membrane, facilitating the sequestration of protein aggregates into autophagosomes.</p></li>
<li><p><strong>NBR1 (Neighbor of BRCA1 Gene 1):</strong> NBR1 is another autophagy receptor that contains LIR motifs and UBA domains. Similar to p62, NBR1 binds to ubiquitinated protein aggregates and recruits autophagy machinery components to promote their degradation by autophagy.</p></li>
<li><p><strong>Optineurin:</strong> Optineurin is a protein involved in various cellular processes, including autophagy. It contains LIR motifs and interacts with both ubiquitin and LC3, facilitating the selective targeting of protein aggregates for autophagic degradation.</p></li>
</ul></li>
<li><p><strong>Receptors in Mitophagy:</strong> Mitophagy specifically targets damaged or dysfunctional mitochondria for degradation to maintain cellular homeostasis. The selective recognition and targeting of mitochondria for degradation involve specific receptors that recognize mitochondrial damage or stress signals. Some of the key receptors involved in mitophagy include:</p>
<ul>
<li><p><strong>PINK1 (PTEN-induced putative kinase 1):</strong> PINK1 is a mitochondrial kinase that accumulates on the outer mitochondrial membrane (OMM) of damaged mitochondria. Upon mitochondrial depolarization, PINK1 is stabilized and phosphorylates both Parkin and mitochondrial substrates. This recruits Parkin to the depolarized mitochondria, where it ubiquitinates mitochondrial proteins, marking them for selective degradation.</p></li>
<li><p><strong>Parkin:</strong> Parkin is an E3 ubiquitin ligase that is recruited to depolarized mitochondria by PINK1. Once recruited, Parkin ubiquitinates proteins on the OMM, leading to the recruitment of autophagy receptors such as p62/SQSTM1 and optineurin. These receptors bind to ubiquitinated mitochondrial proteins and interact with LC3 on the autophagosomal membrane, facilitating the engulfment of damaged mitochondria into autophagosomes for degradation.</p></li>
<li><p><strong>BNIP3 (BCL2/adenovirus E1B 19 kDa protein-interacting protein 3):</strong> BNIP3 is a mitochondrial outer membrane protein that contains an LC3-interacting region (LIR) motif. During hypoxia or other stress conditions, BNIP3 binds to LC3 on the autophagosomal membrane, mediating the selective engulfment of damaged mitochondria for degradation.</p></li>
</ul></li>
</ol>
</section>
</section>
</section>
<section id="autophagy-and-neurodegeneration" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="autophagy-and-neurodegeneration"><span class="header-section-number">6.2</span> Autophagy and Neurodegeneration</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/mito_things.png" class="img-fluid figure-img" width="384"></p>
<p></p><figcaption class="figure-caption">Mitochondrial Dynamics</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Mitochondrial dynamics</strong>, which include processes such as fusion, fission, and mitochondrial movement, play a crucial role in maintaining mitochondrial function and cellular homeostasis. Here’s how mitochondrial dynamics contribute to protecting mitochondrial function:</p>
<ol type="1">
<li><p><strong>Mitochondrial Quality Control:</strong> Mitochondrial dynamics participate in quality control mechanisms by regulating the turnover of damaged or dysfunctional mitochondria. Fusion facilitates the mixing of mitochondrial contents, allowing damaged mitochondria to exchange components with healthy ones, which may help dilute damaged proteins or DNA. Fission, on the other hand, segregates damaged portions of mitochondria, facilitating their selective removal by mitophagy.</p></li>
<li><p><strong>Energy Production and Metabolism:</strong> Mitochondrial fusion promotes the exchange of mitochondrial DNA, proteins, and metabolites, optimizing mitochondrial function and energy production. By creating interconnected mitochondrial networks, fusion helps to maintain efficient electron transport chain complexes, ensuring proper ATP production and metabolic homeostasis. Additionally, fusion can rescue partially dysfunctional mitochondria by complementing missing or defective components.</p></li>
<li><p><strong>Distribution of Mitochondria:</strong> Mitochondrial movement and distribution are essential for ensuring a balanced distribution of mitochondria throughout the cell, particularly in cells with high energy demands or polarized structures. Dynamic positioning of mitochondria allows them to respond rapidly to local energy needs, such as ATP production near sites of high metabolic activity or calcium buffering in regions with elevated calcium levels.</p></li>
<li><p><strong>Mitochondrial Adaptation to Stress:</strong> Mitochondrial dynamics enable rapid responses to cellular stressors by modulating mitochondrial morphology and function. Under stress conditions such as nutrient deprivation, oxidative stress, or changes in energy demand, mitochondrial dynamics can shift towards fusion or fission to optimize mitochondrial function and promote cellular survival. For example, fusion can mitigate stress-induced mitochondrial fragmentation and maintain mitochondrial integrity.</p></li>
<li><p><strong>Apoptosis Regulation:</strong> Mitochondrial dynamics also play a role in regulating apoptosis, a programmed cell death process. Alterations in mitochondrial fusion and fission dynamics can influence mitochondrial morphology and membrane permeability, affecting the release of apoptotic factors such as cytochrome c.&nbsp;Balanced mitochondrial dynamics contribute to cell survival by preventing excessive mitochondrial fragmentation and apoptosis induction.</p></li>
</ol>
<section id="if-mitochondria-do-not-fuse-or-divide" class="level3" data-number="6.2.1">
<h3 data-number="6.2.1" class="anchored" data-anchor-id="if-mitochondria-do-not-fuse-or-divide"><span class="header-section-number">6.2.1</span> If Mitochondria Do Not Fuse or Divide</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/no_fusion.png" class="img-fluid figure-img" width="420"></p>
<p></p><figcaption class="figure-caption">Mitochondria Without Fusion or Division</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>If mitochondria are unable to fuse or divide properly, it can lead to various cellular dysfunctions and pathologies due to the disruption of mitochondrial dynamics. Here are some consequences of impaired mitochondrial fusion or fission:</p>
<ol type="1">
<li><p><strong>Accumulation of Damaged Mitochondria:</strong> Without fusion, damaged mitochondria cannot mix their contents with healthy mitochondria, leading to the accumulation of dysfunctional components such as damaged proteins, lipids, and DNA. Similarly, impaired fission prevents the segregation and removal of damaged portions of mitochondria, further exacerbating mitochondrial dysfunction.</p></li>
<li><p><strong>Mitochondrial Fragmentation:</strong> In the absence of fusion, mitochondria may undergo excessive fission, leading to mitochondrial fragmentation and the formation of smaller, dysfunctional organelles. Fragmented mitochondria have reduced capacity for ATP production, compromised membrane potential, and increased susceptibility to oxidative stress and apoptosis.</p></li>
<li><p><strong>Energy Deficiency:</strong> Mitochondrial fusion is essential for maintaining interconnected mitochondrial networks that facilitate efficient ATP production and metabolic coordination. Impaired fusion results in fragmented mitochondria with reduced respiratory capacity, leading to cellular energy deficits and metabolic dysfunction.</p></li>
<li><p><strong>Cellular Stress Sensitivity:</strong> Proper mitochondrial dynamics play a crucial role in cellular stress responses by modulating mitochondrial morphology and function. Cells with impaired fusion or fission may exhibit heightened sensitivity to various stressors such as nutrient deprivation, oxidative stress, or changes in energy demand, leading to increased susceptibility to cell death or dysfunction.</p></li>
<li><p><strong>Neurological Disorders:</strong> Dysfunction in mitochondrial dynamics has been implicated in the pathogenesis of neurodegenerative diseases such as Parkinson’s disease, Alzheimer’s disease, and Huntington’s disease. Impaired mitochondrial fusion or fission can disrupt neuronal function, synaptic transmission, and neuronal survival, contributing to the progression of neurological disorders.</p></li>
</ol>
</section>
<section id="drp1-mediated-fission" class="level3" data-number="6.2.2">
<h3 data-number="6.2.2" class="anchored" data-anchor-id="drp1-mediated-fission"><span class="header-section-number">6.2.2</span> Drp1-Mediated Fission</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/drp1.png" class="img-fluid figure-img" width="432"></p>
<p></p><figcaption class="figure-caption">Mechanism of Drp1</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Dynamin-related protein 1 (Drp1)</strong> is a key regulator of mitochondrial fission, a process that involves the division of mitochondria into smaller organelles. Here’s how Drp1-mediated fission of mitochondria occurs:</p>
<ol type="1">
<li><p><strong>Localization to Mitochondria:</strong> Drp1 is primarily a cytosolic protein, but it translocates to the outer mitochondrial membrane (OMM) in response to signals that trigger mitochondrial fission. The recruitment and activation of Drp1 at the OMM are tightly regulated by various factors, including post-translational modifications and binding partners.</p></li>
<li><p><strong>Formation of Oligomeric Rings:</strong> Upon translocation to the OMM, Drp1 assembles into higher-order structures, forming spiral or ring-like oligomers around constricted regions of mitochondria. This oligomerization process is facilitated by the GTPase activity of Drp1 and is essential for the subsequent scission of mitochondria.</p></li>
<li><p><strong>Constriction of Mitochondria:</strong> As Drp1 oligomers assemble on the OMM, they constrict the mitochondrial membrane at specific sites, often at the points where ER-mitochondria contact sites are located. This constriction process involves the coordinated action of Drp1 and other proteins, such as mitochondrial fission factor (Mff) and mitochondrial dynamics proteins of 49 and 51 kDa (MiD49/51), which serve as receptors for Drp1 recruitment.</p></li>
<li><p><strong>Fission Machinery Activation:</strong> Once the mitochondria are constricted by Drp1 oligomers, the fission machinery is activated to complete the division process. GTP hydrolysis by Drp1 induces a conformational change that triggers the mechanical force required for mitochondrial scission. This leads to the division of the mitochondria into two daughter organelles.</p></li>
<li><p><strong>Release of Daughter Mitochondria:</strong> Following scission, the daughter mitochondria are released into the cytosol as individual organelles. Drp1 remains associated with the mitochondrial fragments for a short period before being recycled back to the cytosol. The daughter mitochondria undergo further remodeling and fusion with other mitochondria to maintain mitochondrial homeostasis and function.</p></li>
</ol>
</section>
<section id="mitochondrial-fusion" class="level3" data-number="6.2.3">
<h3 data-number="6.2.3" class="anchored" data-anchor-id="mitochondrial-fusion"><span class="header-section-number">6.2.3</span> Mitochondrial Fusion</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/fusion.png" class="img-fluid figure-img" width="518"></p>
<p></p><figcaption class="figure-caption">Schemaitc of Mitochondria Fusion</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Mitochondrial fusion</strong> is a complex process involving the merger of two individual mitochondria to form a single, interconnected organelle. This process is crucial for maintaining mitochondrial function, morphology, and cellular homeostasis. Here’s an overview of the mechanism of mitochondrial fusion:</p>
<ol type="1">
<li><p><strong>Outer Membrane Fusion:</strong> Mitochondrial fusion begins with the fusion of the outer mitochondrial membranes (OMMs) of two adjacent mitochondria. The initial step involves tethering and docking of the outer membranes through protein-protein interactions. Several protein complexes facilitate this process, including mitofusins (Mfn1 and Mfn2) in mammals and Fzo1 in yeast. These dynamin-like GTPases are anchored in the OMM and mediate the physical interaction between adjacent mitochondria.</p></li>
<li><p><strong>GTP Hydrolysis and Membrane Fusion:</strong> After tethering, mitofusins/Fzo1 proteins undergo GTP hydrolysis, leading to a conformational change that promotes the fusion of the outer membranes. This process involves the formation of hemifusion intermediates, where the outer leaflets of the lipid bilayers merge, followed by the fusion of the inner leaflets, resulting in the complete fusion of the outer membranes. The exact mechanism by which mitofusins/Fzo1 facilitate membrane fusion is still under investigation but likely involves changes in protein conformation and lipid rearrangement.</p></li>
<li><p><strong>Intermembrane Space Fusion:</strong> Following outer membrane fusion, the inner mitochondrial membranes (IMMs) of the fused mitochondria come into close proximity. Proteins such as OPA1 (optic atrophy 1) in mammals and Mgm1 in yeast are responsible for mediating fusion of the inner membranes. OPA1 is a dynamin-like GTPase that exists in multiple isoforms, some of which are anchored in the IMM and regulate cristae morphology, while others are released into the intermembrane space (IMS) during fusion. The exact role of OPA1/Mgm1 in inner membrane fusion is not fully understood but likely involves tethering, hemifusion, and lipid mixing similar to outer membrane fusion.</p></li>
<li><p><strong>Matrix Content Mixing:</strong> Once the outer and inner membranes are fused, the contents of the matrix, including proteins, lipids, and mitochondrial DNA, can mix between the fused mitochondria. This allows for the exchange of metabolites and mitochondrial components, contributing to the maintenance of mitochondrial function, genome integrity, and metabolic homeostasis.</p></li>
<li><p><strong>Regulation of Fusion Machinery:</strong> Mitochondrial fusion is tightly regulated by various factors, including post-translational modifications of fusion proteins, calcium signaling, mitochondrial dynamics proteins, and cellular energy status. These regulatory mechanisms ensure the proper coordination of fusion events and prevent excessive or aberrant fusion, which could lead to mitochondrial dysfunction and cellular pathology.</p></li>
</ol>
</section>
<section id="pink1-parkin-and-mitophagy" class="level3" data-number="6.2.4">
<h3 data-number="6.2.4" class="anchored" data-anchor-id="pink1-parkin-and-mitophagy"><span class="header-section-number">6.2.4</span> PINK1, Parkin, and Mitophagy</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week6/pink1.png" class="img-fluid figure-img" width="519"></p>
<p></p><figcaption class="figure-caption">PINK1 and Parkin Mode of Action</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>PINK1 (PTEN-induced putative kinase 1)</strong> and <strong>Parkin</strong> are two key proteins involved in the regulation of mitophagy, a selective form of autophagy that targets damaged or dysfunctional mitochondria for degradation. Here’s how PINK1 and Parkin cooperate to initiate and execute mitophagy:</p>
<ol type="1">
<li><p><strong>PINK1 Activation:</strong> Under normal conditions, PINK1 is continuously imported into healthy mitochondria through the translocase of the outer membrane (TOM) complex and the translocase of the inner membrane (TIM)23 complex. Once inside the mitochondria, PINK1 undergoes proteolytic processing by mitochondrial processing peptidase (MPP) and presenilin-associated rhomboid-like protease (PARL), leading to its degradation in the inner mitochondrial membrane (IMM). However, upon mitochondrial damage or depolarization, PINK1 import is impaired, leading to its accumulation on the outer mitochondrial membrane (OMM).</p></li>
<li><p><strong>PINK1 Stabilization and Phosphorylation:</strong> Accumulated PINK1 on the OMM becomes stabilized and undergoes autophosphorylation. This phosphorylation activates PINK1’s kinase activity, allowing it to phosphorylate various substrates on the OMM, including Miro and ubiquitin.</p></li>
<li><p><strong>Recruitment of Parkin:</strong> Phosphorylated ubiquitin on the OMM serves as a signal for Parkin recruitment. Parkin is an E3 ubiquitin ligase that is normally cytosolic but translocates to damaged mitochondria upon activation by PINK1. Parkin is recruited to mitochondria through its interaction with phosphorylated ubiquitin and other OMM proteins.</p></li>
<li><p><strong>Ubiquitination of Mitochondrial Proteins:</strong> Once recruited to the mitochondria, Parkin catalyzes the ubiquitination of various OMM proteins, including voltage-dependent anion channel (VDAC) and mitofusins (Mfn1 and Mfn2). Ubiquitination of these proteins serves as a signal for the targeting of damaged mitochondria for degradation.</p></li>
<li><p><strong>Recognition by Autophagy Machinery:</strong> Ubiquitinated proteins on the surface of damaged mitochondria are recognized by autophagy receptors such as p62/SQSTM1 and optineurin. These receptors bind to ubiquitin chains via their ubiquitin-binding domains (UBDs) and interact with LC3 on the forming autophagosomal membrane, thereby linking the damaged mitochondria to the autophagy machinery.</p></li>
<li><p><strong>Sequestration and Degradation:</strong> The autophagosome engulfs the damaged mitochondria, forming an autophagosome-enclosed structure called the mitophagosome. The mitophagosome subsequently fuses with lysosomes to form an autolysosome, where the engulfed mitochondria are degraded by lysosomal enzymes, leading to the removal of damaged components and recycling of nutrients.</p></li>
</ol>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week5.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week7.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>