<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS4018: Biology of Aging and Neurodegeneration - 3&nbsp; Regulating Protein Synthesis</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week4.html" rel="next">
<link href="../chapters/week2.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS4018: Biology of Aging and Neurodegeneration</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS4018_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Mitochondria in Aging and Neurodegeneration</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Autophagy of Protein Aggregates and Mitochondria</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week7.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Singalling Pathways and Telomeres in Aging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week8.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Mechanism of Intracellular Protein Deposition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week9.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Alzheimer’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week10.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Parkinson’s Disease</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Secondary Metabolism in Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Plants as Medicine</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#structure-and-synthesis-of-the-ribosome" id="toc-structure-and-synthesis-of-the-ribosome" class="nav-link active" data-scroll-target="#structure-and-synthesis-of-the-ribosome"><span class="toc-section-number">3.1</span>  Structure and Synthesis of the Ribosome</a>
  <ul class="collapse">
  <li><a href="#ribosome-synthesis" id="toc-ribosome-synthesis" class="nav-link" data-scroll-target="#ribosome-synthesis"><span class="toc-section-number">3.1.1</span>  Ribosome Synthesis</a></li>
  <li><a href="#rna-binding-sites" id="toc-rna-binding-sites" class="nav-link" data-scroll-target="#rna-binding-sites"><span class="toc-section-number">3.1.2</span>  RNA Binding Sites</a></li>
  </ul></li>
  <li><a href="#initiating-translation" id="toc-initiating-translation" class="nav-link" data-scroll-target="#initiating-translation"><span class="toc-section-number">3.2</span>  Initiating Translation</a>
  <ul class="collapse">
  <li><a href="#initiation-factors-in-bacteria" id="toc-initiation-factors-in-bacteria" class="nav-link" data-scroll-target="#initiation-factors-in-bacteria"><span class="toc-section-number">3.2.1</span>  Initiation Factors in Bacteria</a></li>
  <li><a href="#in-eukaryotes" id="toc-in-eukaryotes" class="nav-link" data-scroll-target="#in-eukaryotes"><span class="toc-section-number">3.2.2</span>  In Eukaryotes</a></li>
  <li><a href="#utr-in-eukaryotes" id="toc-utr-in-eukaryotes" class="nav-link" data-scroll-target="#utr-in-eukaryotes"><span class="toc-section-number">3.2.3</span>  5’ UTR in Eukaryotes</a></li>
  <li><a href="#prokaryotic-and-eukaryotic-mrna" id="toc-prokaryotic-and-eukaryotic-mrna" class="nav-link" data-scroll-target="#prokaryotic-and-eukaryotic-mrna"><span class="toc-section-number">3.2.4</span>  Prokaryotic and Eukaryotic mRNA</a></li>
  </ul></li>
  <li><a href="#regulating-translation-initiation" id="toc-regulating-translation-initiation" class="nav-link" data-scroll-target="#regulating-translation-initiation"><span class="toc-section-number">3.3</span>  Regulating Translation Initiation</a>
  <ul class="collapse">
  <li><a href="#eif2-cycle" id="toc-eif2-cycle" class="nav-link" data-scroll-target="#eif2-cycle"><span class="toc-section-number">3.3.1</span>  eIF2 Cycle</a></li>
  <li><a href="#integrated-stress-response" id="toc-integrated-stress-response" class="nav-link" data-scroll-target="#integrated-stress-response"><span class="toc-section-number">3.3.2</span>  Integrated Stress Response</a></li>
  </ul></li>
  <li><a href="#elongating-translation" id="toc-elongating-translation" class="nav-link" data-scroll-target="#elongating-translation"><span class="toc-section-number">3.4</span>  Elongating Translation</a>
  <ul class="collapse">
  <li><a href="#trna-charging" id="toc-trna-charging" class="nav-link" data-scroll-target="#trna-charging"><span class="toc-section-number">3.4.1</span>  tRNA Charging</a></li>
  <li><a href="#elongation-by-eef2" id="toc-elongation-by-eef2" class="nav-link" data-scroll-target="#elongation-by-eef2"><span class="toc-section-number">3.4.2</span>  Elongation by eEF2</a></li>
  <li><a href="#profiling" id="toc-profiling" class="nav-link" data-scroll-target="#profiling"><span class="toc-section-number">3.4.3</span>  Profiling</a></li>
  </ul></li>
  <li><a href="#terminating-translation" id="toc-terminating-translation" class="nav-link" data-scroll-target="#terminating-translation"><span class="toc-section-number">3.5</span>  Terminating Translation</a>
  <ul class="collapse">
  <li><a href="#recycling-ribosomes" id="toc-recycling-ribosomes" class="nav-link" data-scroll-target="#recycling-ribosomes"><span class="toc-section-number">3.5.1</span>  Recycling Ribosomes</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS4018_Notes/edit/main/chapters/week3.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS4018_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Regulating Protein Synthesis</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/central_dogma.png" class="img-fluid figure-img" width="494"></p>
<p></p><figcaption class="figure-caption">Outline of the Central Dogma</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>central dogma</strong> of molecular biology is a fundamental principle that describes the flow of genetic information within a biological system, particularly in cells. Proposed by Francis Crick in 1958, the central dogma outlines the sequential processes by which genetic information encoded in DNA is transferred to RNA and eventually translated into proteins. The central dogma can be summarized in three main steps:</p>
<ol type="1">
<li><p><strong>DNA Replication</strong>: The process by which DNA molecules are duplicated to produce two identical copies of the genetic material. DNA replication occurs during cell division and ensures that each daughter cell receives a complete set of genetic instructions.</p></li>
<li><p><strong>Transcription</strong>: The process by which genetic information encoded in DNA is transcribed into RNA molecules. During transcription, an enzyme called RNA polymerase binds to a specific region of DNA known as a gene and synthesizes a complementary RNA strand using one of the DNA strands as a template. The resulting RNA molecule, known as messenger RNA (mRNA), carries the genetic information from the DNA to the ribosomes, where it serves as a template for protein synthesis.</p></li>
<li><p><strong>Translation</strong>: The process by which the genetic information carried by mRNA is translated into proteins. Translation occurs on ribosomes, cellular organelles composed of ribosomal RNA (rRNA) and proteins. During translation, transfer RNA (tRNA) molecules deliver amino acids to the ribosome in response to the sequence of codons (three-nucleotide sequences) on the mRNA. The ribosome reads the mRNA sequence and catalyzes the formation of peptide bonds between the amino acids, resulting in the synthesis of a polypeptide chain, which folds into a functional protein.</p></li>
</ol>
<section id="structure-and-synthesis-of-the-ribosome" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="structure-and-synthesis-of-the-ribosome"><span class="header-section-number">3.1</span> Structure and Synthesis of the Ribosome</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/ribosomes.png" class="img-fluid figure-img" width="435"></p>
<p></p><figcaption class="figure-caption">Differences Between Eukaryotic and Prokaryotic Ribosomes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Prokaryotic ribosomes are composed of two subunits: a large 50S subunit and a small 30S subunit, which together form the 70S ribosome. The 70S ribosome is the site of protein synthesis in prokaryotic cells. Prokaryotic ribosomes are smaller than eukaryotic ribosomes, with the 50S subunit having a sedimentation coefficient of 50S and the 30S subunit having a sedimentation coefficient of 30S. They contain ribosomal RNA (rRNA) and protein components, with the 50S subunit containing 23S and 5S rRNA molecules, and the 30S subunit containing 16S rRNA.</p>
<p>Eukaryotic ribosomes are larger and more complex than prokaryotic ribosomes, consisting of a large 60S subunit and a small 40S subunit, which together form the 80S ribosome. The 80S ribosome is the site of protein synthesis in eukaryotic cells. Eukaryotic ribosomes are larger than prokaryotic ribosomes, with the 60S subunit having a sedimentation coefficient of 60S and the 40S subunit having a sedimentation coefficient of 40S. They also contain rRNA and protein components, with the 60S subunit containing 28S, 5.8S, and 5S rRNA molecules, and the 40S subunit containing 18S rRNA.</p>
<section id="ribosome-synthesis" class="level3" data-number="3.1.1">
<h3 data-number="3.1.1" class="anchored" data-anchor-id="ribosome-synthesis"><span class="header-section-number">3.1.1</span> Ribosome Synthesis</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/ribosome_synthesis.png" class="img-fluid figure-img" width="510"></p>
<p></p><figcaption class="figure-caption">Ribosome Synthesis Outline</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Ribosome synthesis primarily occurs within the nucleolus, a distinct subcompartment of the cell nucleus responsible for ribosome biogenesis. The nucleolus is the site where ribosomal RNA (rRNA) transcription, processing, and assembly with ribosomal proteins take place, leading to the formation of functional ribosomes. Here’s an overview of ribosome synthesis in the nucleolus:</p>
<ol type="1">
<li><p><strong>rRNA Transcription</strong>: The process of ribosome synthesis begins with the transcription of rRNA genes by RNA polymerase I (Pol I) within the nucleolus. These rRNA genes are organized into tandem repeats known as nucleolar organizer regions (NORs) located on specific chromosomes. The transcription of rRNA genes generates a large precursor molecule called pre-rRNA.</p></li>
<li><p><strong>Pre-rRNA Processing</strong>: The pre-rRNA transcript undergoes extensive processing to generate mature rRNAs, including the 18S, 5.8S, and 28S rRNAs found in eukaryotic ribosomes. Processing involves the removal of spacer sequences and the cleavage of pre-rRNA intermediates by a complex machinery of small nucleolar ribonucleoproteins (snoRNPs) and endonucleases. This processing occurs in distinct subcompartments of the nucleolus, known as the fibrillar center and the dense fibrillar component.</p></li>
<li><p><strong>Assembly of Ribosomal Subunits</strong>: Once processed, the mature rRNAs associate with ribosomal proteins to form the small (40S) and large (60S) ribosomal subunits. The assembly of ribosomal subunits occurs in a stepwise manner, involving the coordinated binding of ribosomal proteins to specific regions of the rRNA molecules. The nucleolus provides the necessary environment and factors for efficient assembly, ensuring the proper folding and stability of the ribosomal subunits.</p></li>
<li><p><strong>Export of Ribosomal Subunits</strong>: After assembly, the mature ribosomal subunits are exported from the nucleolus to the cytoplasm, where they participate in protein synthesis. This export process involves the recognition of specific export signals within the ribosomal subunits by nuclear export receptors and the translocation of ribosomal subunits through nuclear pores. Once in the cytoplasm, the small and large ribosomal subunits associate with messenger RNA (mRNA) and transfer RNA (tRNA) molecules to form functional ribosomes capable of protein synthesis.</p></li>
</ol>
</section>
<section id="rna-binding-sites" class="level3" data-number="3.1.2">
<h3 data-number="3.1.2" class="anchored" data-anchor-id="rna-binding-sites"><span class="header-section-number">3.1.2</span> RNA Binding Sites</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/rna_binding_sites.png" class="img-fluid figure-img" width="418"></p>
<p></p><figcaption class="figure-caption">The Four RNA Binding Sites</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Ribosomes possess three primary binding sites crucial for the translation process: the <strong>A (Aminoacyl) site</strong>, the <strong>P (Peptidyl) site</strong>, and the <strong>E (Exit) site</strong>. These sites facilitate the interaction between mRNA, tRNA, and ribosomal subunits during protein synthesis. In the A site, incoming aminoacyl-tRNA molecules bearing amino acids align with the mRNA codon, ensuring the correct addition of amino acids to the growing polypeptide chain. The P site harbors the peptidyl-tRNA, where the nascent polypeptide chain is elongated by the addition of amino acids. Finally, the E site accommodates the exit of deacylated tRNA molecules following peptide bond formation, enabling the recycling of tRNA for subsequent rounds of translation.</p>
<p>Translation proceeds through four main steps: initiation, elongation, termination, and ribosome recycling. During <strong>initiation</strong>, the small ribosomal subunit binds to the mRNA molecule at the start codon, guided by initiation factors and the initiator tRNA. Following initiation, the large ribosomal subunit joins the complex, forming the functional ribosome ready for protein synthesis. In the <strong>elongation phase</strong>, aminoacyl-tRNA molecules enter the A site, and peptide bond formation occurs between the amino acids carried by the tRNAs in the A and P sites. The ribosome moves along the mRNA in a 5’ to 3’ direction, synthesizing the polypeptide chain one amino acid at a time.</p>
<p><strong>Termination</strong> occurs when the ribosome encounters a stop codon (UAA, UAG, or UGA) on the mRNA. Release factors bind to the A site, promoting the hydrolysis of the bond between the completed polypeptide chain and the tRNA in the P site. This leads to the release of the polypeptide chain from the ribosome and the dissociation of the ribosomal subunits from the mRNA. Finally, in the ribosome recycling phase, ribosome recycling factors facilitate the disassembly of the ribosome complex, allowing the small and large subunits to be reused for subsequent rounds of translation. This cycle ensures the continuous synthesis of proteins in the cell, crucial for various cellular functions and processes.</p>
</section>
</section>
<section id="initiating-translation" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="initiating-translation"><span class="header-section-number">3.2</span> Initiating Translation</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/shine_dalgarno.png" class="img-fluid figure-img" width="430"></p>
<p></p><figcaption class="figure-caption">Depiction of the Shine-Dalgarno Sequence</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In prokaryotes, the <strong>Shine-Dalgarno sequence</strong> is a conserved nucleotide sequence found in the mRNA, typically positioned upstream of the start codon (AUG) within the ribosome binding site (RBS). The Shine-Dalgarno sequence facilitates the recognition and binding of the mRNA by the small ribosomal subunit during the initiation of translation.</p>
<p>The Shine-Dalgarno sequence is characterized by the presence of a purine-rich region, often containing the consensus sequence AGGAGG. This sequence is complementary to a region of the 16S ribosomal RNA (rRNA) found in the small ribosomal subunit. The interaction between the Shine-Dalgarno sequence in the mRNA and the complementary sequence in the 16S rRNA helps position the ribosome correctly on the mRNA, ensuring accurate initiation of protein synthesis.</p>
<p>During translation initiation in prokaryotes, the small ribosomal subunit, along with initiation factors, recognizes and binds to the Shine-Dalgarno sequence on the mRNA. This binding facilitates the scanning of the mRNA by the ribosome until it reaches the start codon (usually AUG). Once the start codon is encountered, the large ribosomal subunit joins the complex, and protein synthesis proceeds along the mRNA.</p>
<section id="initiation-factors-in-bacteria" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="initiation-factors-in-bacteria"><span class="header-section-number">3.2.1</span> Initiation Factors in Bacteria</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/initiation_factors.png" class="img-fluid figure-img" width="416"></p>
<p></p><figcaption class="figure-caption">Some Bacterial Initiation Factors</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In bacterial translation initiation, several initiation factors play critical roles in orchestrating the assembly of the translation initiation complex and ensuring accurate start codon recognition. Two key initiation factors in bacteria are <strong>IF1</strong> and <strong>IF3</strong>, which prevent premature association of the large 50S ribosomal subunit with the small 30S subunit, thereby promoting the formation of the initiation complex on the mRNA.</p>
<ol type="1">
<li><p><strong>IF1</strong>: IF1 is an initiation factor that helps prevent the premature binding of the 50S ribosomal subunit to the 30S subunit, ensuring that translation initiation occurs accurately at the start codon. IF1 also stabilizes the 30S subunit and helps maintain its integrity during the initiation process.</p></li>
<li><p><strong>IF3</strong>: IF3 is another initiation factor that contributes to the fidelity of translation initiation by preventing the association of the 50S subunit with the 30S subunit until the initiation complex is properly assembled on the mRNA. IF3 also helps dissociate ribosomes at the termination of translation.</p></li>
</ol>
<p>In addition to IF1 and IF3, IF2 is a critical initiation factor responsible for delivering the initiator tRNA (fMet-tRNAfMet) to the ribosomal complex during translation initiation. IF2 is a GTPase, meaning it hydrolyzes GTP to GDP during its function in the initiation process. The binding of IF2-GTP to the initiator tRNA stabilizes its interaction with the 30S ribosomal subunit and helps position it correctly within the ribosomal P site, ensuring accurate start codon recognition and initiation of protein synthesis.</p>
</section>
<section id="in-eukaryotes" class="level3" data-number="3.2.2">
<h3 data-number="3.2.2" class="anchored" data-anchor-id="in-eukaryotes"><span class="header-section-number">3.2.2</span> In Eukaryotes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/eukaryotes.png" class="img-fluid figure-img" width="436"></p>
<p></p><figcaption class="figure-caption">Initiating Translation in Eukaryotes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Here’s an overview of the key steps involved in translation initiation in eukaryotic cells:</p>
<ol type="1">
<li><p><strong>Recognition of the 5’ Cap Structure</strong>: Translation initiation in eukaryotes begins with the recognition of the 5’ cap structure present at the 5’ end of the mRNA molecule. The 5’ cap is recognized by the eukaryotic initiation factor 4E (eIF4E), which forms part of the eIF4F complex along with eIF4A (helicase) and eIF4G (scaffolding protein).</p></li>
<li><p><strong>Recruitment of the 40S Ribosomal Subunit</strong>: The eIF4F complex, bound to the 5’ cap structure, recruits the 40S small ribosomal subunit. This recruitment is facilitated by the interaction between eIF4G and the poly(A)-binding protein (PABP) bound to the poly(A) tail of the mRNA, forming a closed-loop structure that enhances translation efficiency.</p></li>
<li><p><strong>Scanning for the Start Codon</strong>: The 40S subunit, together with initiation factors (eIF1, eIF1A, and eIF3), scans along the mRNA in a 5’ to 3’ direction until it encounters the start codon (AUG) within a favorable Kozak sequence context. The Kozak sequence (consensus sequence: GCCRCCAUGG) surrounding the start codon helps ensure accurate start codon recognition.</p></li>
<li><p><strong>Initiator tRNA Binding</strong>: Upon recognition of the start codon, the initiator tRNA (Met-tRNAiMet) bound to eukaryotic initiation factor 2 (eIF2) and GTP forms a ternary complex. This ternary complex is then recruited to the ribosome, where it base pairs with the start codon in the mRNA.</p></li>
<li><p><strong>Assembly of the 80S Initiation Complex</strong>: Once the initiator tRNA is properly positioned at the start codon, GTP bound to eIF2 is hydrolyzed to GDP and Pi, resulting in the release of eIF2-GDP from the ribosome. The 60S large ribosomal subunit then joins the 40S subunit to form the 80S initiation complex, with the initiator tRNA occupying the P site of the ribosome.</p></li>
<li><p><strong>Initiation Factor Dissociation and Start of Elongation</strong>: Following the assembly of the 80S initiation complex, initiation factors eIF1, eIF1A, and eIF3 are released from the ribosome, allowing for the start of elongation. Elongation factors, along with charged tRNAs and GTP, facilitate the stepwise addition of amino acids to the growing polypeptide chain.</p></li>
</ol>
</section>
<section id="utr-in-eukaryotes" class="level3" data-number="3.2.3">
<h3 data-number="3.2.3" class="anchored" data-anchor-id="utr-in-eukaryotes"><span class="header-section-number">3.2.3</span> 5’ UTR in Eukaryotes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/5utr.png" class="img-fluid figure-img" width="182"></p>
<p></p><figcaption class="figure-caption">Illustration of the 5’ UTR in Eukaryotes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In eukaryotic translation initiation, the 5’ untranslated region (5’ UTR) of mRNA plays a crucial role in directing ribosomes to the correct start codon for protein synthesis. The process of 5’ UTR scanning involves the stepwise movement of ribosomes along the 5’ UTR until they identify and initiate translation at the appropriate start codon. Here’s how 5’ UTR scanning occurs in eukaryotes:</p>
<ol type="1">
<li><p><strong>Recognition of the 5’ Cap Structure</strong>: Translation initiation begins with the recognition of the 5’ cap structure present at the 5’ end of the mRNA molecule. The 5’ cap consists of a modified guanosine nucleotide (7-methylguanosine) added during mRNA transcription. The cap structure is recognized by eukaryotic initiation factors (eIFs), particularly eIF4E, which is a component of the eIF4F complex.</p></li>
<li><p><strong>Recruitment of the 43S Pre-initiation Complex</strong>: The eIF4F complex, bound to the 5’ cap structure, recruits the 43S pre-initiation complex, which comprises the 40S small ribosomal subunit, eIF3, eIF1, eIF1A, and the ternary complex consisting of initiator tRNA (Met-tRNAiMet) and eIF2-GTP (eukaryotic initiation factor 2 bound to guanosine triphosphate).</p></li>
<li><p><strong>Scanning of the 5’ UTR</strong>: The 43S pre-initiation complex, guided by eIF4F, scans along the 5’ UTR of the mRNA in a 5’ to 3’ direction. The scanning process is facilitated by the interaction between eIF4G, a component of the eIF4F complex, and the poly(A)-binding protein (PABP) bound to the poly(A) tail of the mRNA. This interaction helps maintain the circular structure of the mRNA, allowing the ribosome to scan efficiently.</p></li>
<li><p><strong>Recognition of the Start Codon</strong>: As the 43S pre-initiation complex scans along the 5’ UTR, it encounters the start codon (AUG) embedded within the Kozak consensus sequence. The Kozak sequence (GCC(A/G)CCAUGG) surrounding the start codon helps facilitate accurate start codon recognition by the ribosome.</p></li>
<li><p><strong>Formation of the 48S Initiation Complex</strong>: Upon recognition of the start codon, base pairing occurs between the initiator tRNA (Met-tRNAiMet) and the start codon. GTP bound to eIF2 is hydrolyzed to GDP and Pi, leading to the release of eIF2-GDP and other initiation factors. The 60S large ribosomal subunit then joins the 40S subunit to form the 80S initiation complex, ready for translation elongation.</p></li>
</ol>
</section>
<section id="prokaryotic-and-eukaryotic-mrna" class="level3" data-number="3.2.4">
<h3 data-number="3.2.4" class="anchored" data-anchor-id="prokaryotic-and-eukaryotic-mrna"><span class="header-section-number">3.2.4</span> Prokaryotic and Eukaryotic mRNA</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/mrna_types.png" class="img-fluid figure-img" width="424"></p>
<p></p><figcaption class="figure-caption">Different Kinds of mRNA</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Prokaryotic and eukaryotic mRNA serve as intermediaries between DNA and protein synthesis, yet they exhibit notable differences in structure, processing, and regulation. Prokaryotic mRNA tends to have a simpler structure, lacking extensive post-transcriptional modifications commonly found in eukaryotic mRNA. In prokaryotes, mRNA molecules are often polycistronic, capable of encoding multiple proteins within a single transcript. Conversely, eukaryotic mRNA is typically monocistronic, encoding only one protein per transcript, adding to the complexity of gene expression regulation.</p>
<p>Eukaryotic mRNA undergoes comprehensive post-transcriptional processing, including 5’ capping, splicing, and polyadenylation, which stabilize the molecule and optimize translation initiation. This processing ensures the production of mature mRNA molecules ready for efficient translation. In contrast, prokaryotic mRNA generally lacks such extensive post-transcriptional modifications, being synthesized as mature transcripts directly by RNA polymerase.</p>
<p>Regulatory mechanisms governing mRNA expression differ significantly between prokaryotes and eukaryotes. In eukaryotic cells, gene expression is tightly regulated at various levels, including chromatin remodeling, transcription initiation, mRNA processing, nuclear export, and mRNA stability. Prokaryotic gene expression, on the other hand, primarily occurs at the transcriptional level, although regulatory RNA elements and proteins can also modulate mRNA stability and translation efficiency.</p>
<p>Prokaryotic mRNA features a Shine-Dalgarno sequence upstream of the start codon, facilitating ribosome binding and translation initiation. In contrast, eukaryotic mRNA lacks a Shine-Dalgarno sequence and relies on the recognition of the 5’ cap structure by initiation factors to initiate translation. The ribosome then scans along the 5’ untranslated region until it encounters the start codon.</p>
</section>
</section>
<section id="regulating-translation-initiation" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="regulating-translation-initiation"><span class="header-section-number">3.3</span> Regulating Translation Initiation</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/mtorc1.png" class="img-fluid figure-img" width="480"></p>
<p></p><figcaption class="figure-caption">mTORC1 Phosphorylating eIF4E-Binding Protein</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The phosphorylation of <strong>eIF4E-binding proteins (4E-BPs)</strong> by the mechanistic target of <strong>rapamycin complex 1 (mTORC1)</strong> is a critical regulatory event in the control of protein synthesis in eukaryotic cells. 4E-BPs are a family of proteins that bind to eIF4E, preventing its interaction with the mRNA 5’ cap structure and thereby inhibiting translation initiation. Here’s an overview of how mTORC1 regulates protein synthesis through the phosphorylation of 4E-BPs:</p>
<ol type="1">
<li><p><strong>mTORC1 Activation</strong>: The mechanistic target of rapamycin complex 1 (mTORC1) is a key regulator of cell growth and metabolism that integrates various signals, including growth factors, nutrients, energy status, and stress, to control protein synthesis and cell proliferation. When activated, mTORC1 phosphorylates downstream targets involved in protein synthesis, including 4E-BPs.</p></li>
<li><p><strong>Phosphorylation of 4E-BPs</strong>: In response to favorable growth conditions and signaling cues, mTORC1 phosphorylates 4E-BPs at multiple serine and threonine residues. Phosphorylation of 4E-BPs induces a conformational change that reduces their affinity for eIF4E, allowing eIF4E to interact with eIF4G and other initiation factors to form the eIF4F complex. The eIF4F complex then associates with the mRNA 5’ cap structure, promoting translation initiation.</p></li>
<li><p><strong>Release of eIF4E</strong>: Upon phosphorylation by mTORC1, 4E-BPs dissociate from eIF4E, freeing eIF4E to participate in the assembly of the translation initiation complex. By releasing eIF4E from its inhibitory interaction with 4E-BPs, mTORC1 activation promotes the recruitment of ribosomes to mRNA transcripts and enhances protein synthesis rates.</p></li>
<li><p><strong>Regulation of Gene Expression</strong>: The phosphorylation of 4E-BPs by mTORC1 represents a key regulatory mechanism that controls the translation of specific mRNAs involved in cell growth, proliferation, and metabolism. By modulating the activity of 4E-BPs, mTORC1 fine-tunes the translational output of cells in response to extracellular signals and nutrient availability, ensuring efficient protein synthesis under conditions favorable for cell growth and survival.</p></li>
</ol>
<section id="eif2-cycle" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="eif2-cycle"><span class="header-section-number">3.3.1</span> eIF2 Cycle</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/eif2_cycle.png" class="img-fluid figure-img" width="502"></p>
<p></p><figcaption class="figure-caption">Cycle of eIF2</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>eukaryotic translation initiation factor 2 (eIF2) cycle</strong> is a fundamental process in protein synthesis regulation, controlling the initiation of translation in response to various cellular stresses and environmental cues. Here’s an overview of the eIF2 cycle:</p>
<ol type="1">
<li><p><strong>GTP-bound eIF2 Complex Formation</strong>: In its inactive state, eIF2 is bound to guanosine triphosphate (GTP) and forms a complex with initiator tRNA (Met-tRNAiMet), termed the ternary complex (eIF2-GTP-Met-tRNAiMet). This ternary complex plays a crucial role in delivering the initiator tRNA to the ribosome during translation initiation.</p></li>
<li><p><strong>Recognition of Start Codon</strong>: The eIF2-GTP-Met-tRNAiMet complex associates with the small ribosomal subunit (40S) and scans the mRNA transcript for the start codon (AUG). Once the start codon is recognized, the ternary complex positions the initiator tRNA at the ribosomal P site, ready to initiate translation.</p></li>
<li><p><strong>GTP Hydrolysis</strong>: Upon accurate recognition of the start codon, GTP bound to eIF2 is hydrolyzed to guanosine diphosphate (GDP) and inorganic phosphate (Pi) by the GTPase activity of eIF2. This hydrolysis reaction is facilitated by eIF5, a GTPase-activating protein (GAP) that stimulates the intrinsic GTPase activity of eIF2.</p></li>
<li><p><strong>Release of eIF2-GDP</strong>: Following GTP hydrolysis, eIF2-GDP is released from the ribosome and recycled back to its active form through the exchange of GDP for GTP. This exchange is catalyzed by eIF2B, a guanine nucleotide exchange factor (GEF) that promotes the reactivation of eIF2 by loading GTP onto eIF2, allowing it to participate in subsequent rounds of translation initiation.</p></li>
<li><p><strong>Regulation by Phosphorylation</strong>: The activity of eIF2 is tightly regulated by phosphorylation of its α-subunit at a conserved serine residue (Ser51) by specific kinases in response to various stress signals. Phosphorylation of eIF2α inhibits the guanine nucleotide exchange activity of eIF2B, leading to a decrease in the formation of active eIF2-GTP complexes and a consequent reduction in global protein synthesis rates. This integrated stress response (ISR) helps cells adapt to adverse conditions by conserving energy and resources.</p></li>
</ol>
<section id="inactivating-eif2" class="level4" data-number="3.3.1.1">
<h4 data-number="3.3.1.1" class="anchored" data-anchor-id="inactivating-eif2"><span class="header-section-number">3.3.1.1</span> Inactivating eIF2</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/inactivating_eif2.png" class="img-fluid figure-img" width="492"></p>
<p></p><figcaption class="figure-caption">How eIF2 Can be Inactivated</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The inactivation of eukaryotic translation initiation factor 2B (eIF2B) by phosphorylated eIF2 is a key regulatory mechanism in the integrated stress response (ISR) pathway. eIF2B is a guanine nucleotide exchange factor (GEF) responsible for catalyzing the exchange of guanosine diphosphate (GDP) for guanosine triphosphate (GTP) on eukaryotic translation initiation factor 2 (eIF2), thereby promoting the formation of the active eIF2-GTP ternary complex essential for translation initiation. Here’s how the inactivation process occurs:</p>
<ol type="1">
<li><p><strong>Phosphorylation of eIF2</strong>: In response to various stress stimuli such as amino acid deprivation, viral infection, endoplasmic reticulum (ER) stress, or oxidative stress, specific kinases phosphorylate the alpha subunit of eIF2 (eIF2α) at a conserved serine residue (Ser51). These kinases include protein kinase R (PKR), heme-regulated inhibitor kinase (HRI), PKR-like endoplasmic reticulum kinase (PERK), and general control non-derepressible 2 (GCN2). Phosphorylation of eIF2α at Ser51 inhibits the GDP-GTP exchange activity of eIF2B, leading to the sequestration of eIF2 in its inactive GDP-bound form.</p></li>
<li><p><strong>Attenuation of Guanine Nucleotide Exchange Activity</strong>: Phosphorylated eIF2 (eIF2α-P) binds tightly to eIF2B, resulting in the sequestration and inhibition of eIF2B’s guanine nucleotide exchange activity. As a consequence, the recycling of eIF2-GDP to its active GTP-bound form is impaired, leading to a reduction in the levels of the active eIF2-GTP ternary complex available for translation initiation.</p></li>
<li><p><strong>Global Attenuation of Protein Synthesis</strong>: The inhibition of eIF2B by phosphorylated eIF2 results in a general attenuation of protein synthesis rates in the cell. Translation initiation is hindered due to the reduced availability of the active eIF2-GTP ternary complex, which is essential for the formation of the pre-initiation complex and the recruitment of ribosomes to mRNA transcripts.</p></li>
<li><p><strong>Induction of Stress-Responsive Gene Expression</strong>: The reduction in global protein synthesis rates during the ISR is coupled with the selective translation of specific stress-responsive mRNAs encoding proteins involved in adaptive cellular processes. This selective translational control allows cells to conserve energy and resources while prioritizing the synthesis of proteins required for stress adaptation and survival.</p></li>
</ol>
</section>
</section>
<section id="integrated-stress-response" class="level3" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="integrated-stress-response"><span class="header-section-number">3.3.2</span> Integrated Stress Response</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/integrated_stress_responses.png" class="img-fluid figure-img" width="494"></p>
<p></p><figcaption class="figure-caption">Integrated Stress Response Depictions</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The <strong>integrated stress response (ISR)</strong> is a conserved cellular signaling pathway that coordinates adaptive responses to various environmental stresses and perturbations. It serves as a crucial mechanism for maintaining cellular homeostasis and promoting cell survival under adverse conditions. Here’s an overview of the integrated stress response:</p>
<ol type="1">
<li><p><strong>Initiation of the Integrated Stress Response</strong>: The ISR is initiated by the phosphorylation of the alpha subunit of eukaryotic translation initiation factor 2 (eIF2α) at a conserved serine residue (Ser51) in response to diverse stress stimuli. Several stress-responsive protein kinases, including protein kinase R (PKR), heme-regulated inhibitor kinase (HRI), PKR-like endoplasmic reticulum kinase (PERK), and general control non-derepressible 2 (GCN2), phosphorylate eIF2α in a context-specific manner.</p></li>
<li><p><strong>Global Attenuation of Protein Synthesis</strong>: Phosphorylation of eIF2α inhibits the guanine nucleotide exchange activity of eIF2B, leading to a reduction in the formation of the active eIF2-GTP ternary complex required for translation initiation. As a result, global protein synthesis rates are attenuated, conserving cellular resources and energy during stress conditions.</p></li>
<li><p><strong>Selective Translation of Stress-Responsive mRNAs</strong>: Despite the general inhibition of protein synthesis, the ISR promotes the selective translation of a subset of mRNAs encoding stress-responsive proteins involved in adaptive cellular processes. These mRNAs often contain upstream open reading frames (uORFs) or specific regulatory elements in their untranslated regions (UTRs) that confer sensitivity to eIF2α phosphorylation and enable preferential translation under stress conditions.</p></li>
<li><p><strong>Induction of Stress-Responsive Genes</strong>: Activation of the ISR leads to the transcriptional upregulation of stress-responsive genes through the activation of transcription factors such as ATF4 (activating transcription factor 4). ATF4 induces the expression of genes involved in amino acid metabolism, redox regulation, autophagy, and apoptosis, facilitating cellular adaptation and survival in response to stress.</p></li>
<li><p><strong>Integration with Other Signaling Pathways</strong>: The ISR is integrated with other stress signaling pathways, including the unfolded protein response (UPR), oxidative stress response, and DNA damage response, to coordinate cellular stress responses and maintain homeostasis. Crosstalk between these pathways allows cells to mount appropriate adaptive responses tailored to specific stressors and environmental cues.</p></li>
</ol>
</section>
</section>
<section id="elongating-translation" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="elongating-translation"><span class="header-section-number">3.4</span> Elongating Translation</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/eukaryotic_elongation.png" class="img-fluid figure-img" width="512"></p>
<p></p><figcaption class="figure-caption">Elongating Translation via Eukaryotes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Eukaryotic translation elongation is the stage of protein synthesis where amino acids are added to the growing polypeptide chain according to the sequence encoded by the mRNA. It involves the movement of ribosomes along the mRNA template, decoding successive codons, and facilitating the transfer of amino acids from charged tRNAs to the growing peptide chain. Here’s an overview of eukaryotic translation elongation:</p>
<ol type="1">
<li><p><strong>Ribosome Movement</strong>: Once the ribosome is assembled on the mRNA and the initiator tRNA is positioned at the start codon (AUG), elongation begins with the binding of the second aminoacyl-tRNA to the A site of the ribosome. The ribosome then translocates along the mRNA in a 5’ to 3’ direction, shifting the peptidyl-tRNA from the A site to the P site and making space for the next incoming aminoacyl-tRNA.</p></li>
<li><p><strong>Codon Recognition and Peptide Bond Formation</strong>: During each cycle of elongation, the ribosome decodes the mRNA by recognizing the next codon in the A site. Aminoacyl-tRNAs carrying the corresponding amino acids bind to the A site through complementary base pairing between the anticodon of the tRNA and the mRNA codon. Peptide bond formation then occurs between the amino acid carried by the peptidyl-tRNA in the P site and the amino acid carried by the aminoacyl-tRNA in the A site, catalyzed by peptidyl transferase activity in the ribosome.</p></li>
<li><p><strong>Translocation</strong>: After peptide bond formation (which is formed via a nucleophilic attack by the N-terminus of an amino acid to the center carbon), the ribosome undergoes a translocation step, facilitated by elongation factor EF-G (eEF2 in eukaryotes) and GTP hydrolysis. This translocation process moves the ribosome along the mRNA by one codon, shifting the peptidyl-tRNA from the A site to the P site and the deacylated tRNA from the P site to the E (exit) site. The mRNA and the newly formed peptidyl-tRNA now occupy the A site, ready for the next cycle of elongation.</p></li>
<li><p><strong>Elongation Factors</strong>: Elongation factors play critical roles in facilitating the elongation process. In addition to EF-G, eukaryotic elongation factors include eEF1A, which delivers aminoacyl-tRNAs to the ribosome, and eEF1B, which promotes the binding of aminoacyl-tRNAs to eEF1A. These factors help ensure the accurate and efficient addition of amino acids to the growing polypeptide chain.</p></li>
<li><p><strong>Termination</strong>: Elongation continues until a stop codon (UAA, UAG, or UGA) is encountered in the A site of the ribosome. Termination factors, such as eRF1 and eRF3 in eukaryotes, recognize the stop codon and promote the release of the completed polypeptide chain from the ribosome. The ribosome dissociates into its subunits, and the newly synthesized protein is released into the cytoplasm for further processing or folding.</p></li>
</ol>
<section id="trna-charging" class="level3" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="trna-charging"><span class="header-section-number">3.4.1</span> tRNA Charging</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/trna_charging.png" class="img-fluid figure-img" width="452"></p>
<p></p><figcaption class="figure-caption">tRNA Charging</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>tRNA charging</strong>, also known as tRNA aminoacylation, is a crucial step in protein synthesis where specific amino acids are covalently attached to their corresponding transfer RNA (tRNA) molecules by <strong>aminoacyl-tRNA synthetases (aaRSs)</strong>. This process ensures that the correct amino acid is delivered to the ribosome during translation. Here’s an overview of tRNA charging by aminoacyl-tRNA synthetases:</p>
<ol type="1">
<li><p><strong>Recognition of Amino Acids</strong>: Each aminoacyl-tRNA synthetase is specific to a particular amino acid. Before aminoacylation, the amino acid to be attached to the tRNA is first recognized and selected by the corresponding aminoacyl-tRNA synthetase. This recognition process involves the binding of the amino acid to a specific binding pocket within the active site of the synthetase enzyme.</p></li>
<li><p><strong>Aminoacylation Reaction</strong>: Once the correct amino acid is bound to the synthetase enzyme, the aminoacylation reaction occurs. The enzyme catalyzes the transfer of the amino acid to the 3’ end of the appropriate tRNA molecule, forming an aminoacyl-tRNA complex. This reaction is driven by the hydrolysis of ATP, which provides the energy required for the formation of the high-energy aminoacyl-AMP intermediate.</p></li>
<li><p><strong>Specificity Determinants</strong>: Aminoacyl-tRNA synthetases possess mechanisms to ensure the fidelity and accuracy of tRNA charging. These enzymes contain specific recognition sites that interact with both the amino acid and the tRNA molecule, allowing them to distinguish between correct and incorrect substrates. The accuracy of tRNA charging is critical for maintaining the fidelity of protein synthesis and preventing errors in translation.</p></li>
<li><p><strong>Proofreading and Editing</strong>: Despite the high specificity of aminoacyl-tRNA synthetases, errors in tRNA charging can still occur. To minimize errors, some synthetases have proofreading or editing mechanisms that hydrolyze incorrectly charged aminoacyl-tRNA complexes. These editing activities help to correct mischarged tRNAs and maintain the integrity of the translation process.</p></li>
<li><p><strong>Role in Protein Synthesis</strong>: Once aminoacylated, the charged tRNA molecules serve as adapters between the mRNA codons and the corresponding amino acids during translation elongation. Each tRNA molecule carries a specific amino acid attached to its 3’ end and contains an anticodon sequence that base-pairs with the complementary mRNA codon. This ensures that the correct amino acid is incorporated into the growing polypeptide chain according to the genetic code.</p></li>
</ol>
<section id="eef1a" class="level4" data-number="3.4.1.1">
<h4 data-number="3.4.1.1" class="anchored" data-anchor-id="eef1a"><span class="header-section-number">3.4.1.1</span> eEF1A</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/eEF1A.png" class="img-fluid figure-img" width="377"></p>
<p></p><figcaption class="figure-caption">eEF1A Delivering Aminoacyl-tRNAs</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Eukaryotic elongation factor 1A (eEF1A)</strong> plays a crucial role in protein synthesis by delivering aminoacyl-tRNA (aa-tRNA) molecules to the ribosome during the elongation phase of translation. Here’s how eEF1A facilitates the delivery of aminoacyl-tRNAs:</p>
<ol type="1">
<li><p><strong>Binding to Aminoacyl-tRNA (aa-tRNA)</strong>: eEF1A binds to aminoacyl-tRNA molecules in a GTP-dependent manner. Each eEF1A molecule forms a complex with a single aa-tRNA, where the amino acid is attached to the 3’ end of the tRNA molecule. This binding occurs at the ribosomal A (aminoacyl) site during the elongation phase of translation.</p></li>
<li><p><strong>GTP Hydrolysis</strong>: The binding of eEF1A to aa-tRNA is accompanied by the hydrolysis of GTP bound to eEF1A. GTP hydrolysis triggers a conformational change in eEF1A, promoting the release of GDP and inorganic phosphate (Pi) from the protein. This conformational change is essential for the efficient delivery of aa-tRNA to the ribosome.</p></li>
<li><p><strong>Delivery to the Ribosome</strong>: Once eEF1A is bound to aa-tRNA and GTP is hydrolyzed, the eEF1A-aa-tRNA complex undergoes a structural rearrangement that allows it to interact with the ribosome. eEF1A delivers the aa-tRNA to the ribosomal A site, where the anticodon of the tRNA base-pairs with the complementary mRNA codon.</p></li>
<li><p><strong>Translocation and Peptide Bond Formation</strong>: After the aa-tRNA is delivered to the ribosome, the ribosome catalyzes the formation of a peptide bond between the amino acid carried by the aa-tRNA in the A site and the growing polypeptide chain attached to the peptidyl-tRNA in the P (peptidyl) site. This process is facilitated by elongation factors and ribosomal RNA (rRNA) within the ribosomal complex.</p></li>
<li><p><strong>GDP Dissociation and Recycling</strong>: Following peptide bond formation, eEF1A dissociates from the ribosome, releasing GDP from its active site. The released GDP is then exchanged for GTP by guanine nucleotide exchange factors (GEFs), such as eEF1B, regenerating the active form of eEF1A and allowing it to participate in subsequent rounds of aa-tRNA delivery.</p></li>
</ol>
</section>
</section>
<section id="elongation-by-eef2" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="elongation-by-eef2"><span class="header-section-number">3.4.2</span> Elongation by eEF2</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/eEF2.png" class="img-fluid figure-img" width="345"></p>
<p></p><figcaption class="figure-caption">eEF2 Promoting Translation Elongation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Eukaryotic elongation factor 2 (eEF2)</strong> is a GTPase protein that facilitates translocation by promoting the movement of the ribosome along the mRNA template. Here’s how eEF2 mediates translocation:</p>
<ol type="1">
<li><p><strong>Ribosome Movement</strong>: After the formation of the peptide bond between the amino acid attached to the tRNA in the A site and the growing polypeptide chain on the tRNA in the P site, the ribosome needs to shift to the next codon on the mRNA template to continue elongation.</p></li>
<li><p><strong>GTP Binding and Hydrolysis</strong>: eEF2 binds to the ribosome in a GTP-bound form. GTP hydrolysis by eEF2 is coupled with the translocation process. The hydrolysis of GTP into GDP and inorganic phosphate (Pi) provides the energy necessary for ribosome movement along the mRNA.</p></li>
<li><p><strong>Ribosome Translocation</strong>: Upon GTP hydrolysis, eEF2 undergoes a conformational change, which results in the movement of the ribosome along the mRNA in a 5’ to 3’ direction. This translocation step shifts the peptidyl-tRNA from the A site to the P site, and the deacylated tRNA from the P site to the E (exit) site of the ribosome.</p></li>
<li><p><strong>GDP Release and Recycling</strong>: After translocation, eEF2-GDP is released from the ribosome, and GDP is exchanged for GTP by guanine nucleotide exchange factors (GEFs), such as eEF2B. The recycling of eEF2 to its active GTP-bound form allows it to participate in subsequent rounds of translocation during translation elongation.</p></li>
<li><p><strong>Accuracy and Efficiency</strong>: The translocation process facilitated by eEF2 is essential for the accurate and efficient synthesis of proteins. It ensures that the ribosome moves along the mRNA in a stepwise manner, decoding each codon and adding the corresponding amino acid to the growing polypeptide chain.</p></li>
</ol>
</section>
<section id="profiling" class="level3" data-number="3.4.3">
<h3 data-number="3.4.3" class="anchored" data-anchor-id="profiling"><span class="header-section-number">3.4.3</span> Profiling</h3>
<section id="polysomes" class="level4" data-number="3.4.3.1">
<h4 data-number="3.4.3.1" class="anchored" data-anchor-id="polysomes"><span class="header-section-number">3.4.3.1</span> Polysomes</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/polysomes.png" class="img-fluid figure-img" width="424"></p>
<p></p><figcaption class="figure-caption">Steps Involved in Polysome Profiling</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Polysome profiling</strong> is a technique used to analyze the translational activity of mRNA molecules within cells. It involves subjecting cell lysates to centrifugation in a sucrose density gradient, which separates ribosomes and associated mRNA molecules based on their sedimentation rates. Polysomes, which consist of multiple ribosomes translating a single mRNA molecule, sediment at higher sucrose densities, while free ribosomes and untranslated mRNA sediment at lower densities.</p>
<p>By analyzing the distribution of ribosomes and mRNA along the gradient, polysome profiling provides valuable information about the efficiency of translation and the regulation of gene expression at the level of mRNA translation. This technique is widely used in molecular biology and cell biology research to study translational control mechanisms, cellular responses to stress, and changes in gene expression under different physiological conditions.</p>
</section>
<section id="ribosome-profiling" class="level4" data-number="3.4.3.2">
<h4 data-number="3.4.3.2" class="anchored" data-anchor-id="ribosome-profiling"><span class="header-section-number">3.4.3.2</span> Ribosome Profiling</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/ribosome_profiling.png" class="img-fluid figure-img" width="328"></p>
<p></p><figcaption class="figure-caption">Steps Involved in Ribosome Profiling</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Ribosome profiling</strong>, also known as ribosome footprinting or ribo-seq, is a powerful technique used to study the global landscape of translation at a genome-wide level. It involves the deep sequencing of ribosome-protected mRNA fragments (ribosome footprints) to identify the positions of ribosomes along mRNA molecules. The technique begins by treating cells with a translation inhibitor to arrest ribosomes at specific positions on mRNA transcripts. Subsequently, ribosomes are digested away, leaving behind ribosome-protected mRNA fragments. These fragments are then isolated, converted into cDNA libraries, and sequenced using high-throughput sequencing technologies.</p>
<p>Ribosome profiling provides detailed information about the density and position of ribosomes on mRNAs, allowing researchers to determine which mRNAs are actively translated, the efficiency of translation, and the locations of translation initiation and termination.</p>
</section>
</section>
</section>
<section id="terminating-translation" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="terminating-translation"><span class="header-section-number">3.5</span> Terminating Translation</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/erf1erf3.png" class="img-fluid figure-img" width="515"></p>
<p></p><figcaption class="figure-caption">eRF1 and eRF3 Terminating Translation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>At the termination stage of translation, eukaryotic release factor 1 (eRF1) and eukaryotic release factor 3 (eRF3) play crucial roles in recognizing and facilitating the termination of protein synthesis at stop codons. Here’s how they function at the stop codon:</p>
<ol type="1">
<li><p><strong>Recognition of Stop Codon</strong>: As the ribosome moves along the mRNA during translation elongation, it encounters stop codons (UAA, UAG, or UGA) that signal the end of protein synthesis. When a stop codon enters the ribosomal A (aminoacyl) site, it does not code for an amino acid but instead signals the termination of translation.</p></li>
<li><p><strong>Role of eRF1</strong>: eRF1 is responsible for recognizing stop codons and promoting the release of the nascent polypeptide chain from the ribosome. It binds to the stop codon in the A site of the ribosome, mimicking the structure of a tRNA molecule. However, instead of carrying an amino acid, eRF1 triggers the hydrolysis of the ester bond between the nascent polypeptide chain and the tRNA molecule in the P (peptidyl) site, resulting in the release of the polypeptide from the ribosome.</p></li>
<li><p><strong>Interaction with eRF3</strong>: eRF3 is a GTPase protein that interacts with eRF1 during translation termination. It enhances the efficiency of termination by stimulating the activity of eRF1 and promoting its release of the polypeptide chain. eRF3 also helps to recycle eRF1 by facilitating the dissociation of eRF1 from the ribosome after termination.</p></li>
<li><p><strong>GTP Hydrolysis</strong>: The interaction between eRF1 and eRF3 is GTP-dependent. eRF3 binds to GTP, and upon interaction with eRF1 at the ribosome, GTP hydrolysis occurs. This hydrolysis event is coupled with the termination process and facilitates the dissociation of eRF1 and eRF3 from the ribosome, allowing the ribosome to be recycled for another round of translation.</p></li>
</ol>
<section id="recycling-ribosomes" class="level3" data-number="3.5.1">
<h3 data-number="3.5.1" class="anchored" data-anchor-id="recycling-ribosomes"><span class="header-section-number">3.5.1</span> Recycling Ribosomes</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week3/ribosomal_recycling.png" class="img-fluid figure-img" width="516"></p>
<p></p><figcaption class="figure-caption">Ribosomes are Recycled After Translation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Ribosome recycling</strong> is a critical process in protein synthesis that ensures the efficient reuse of ribosomes after the termination of translation. It involves the dissociation of ribosomes from mRNA and the recycling of ribosomal subunits for subsequent rounds of translation. Here’s how ribosome recycling occurs:</p>
<ol type="1">
<li><p><strong>Termination of Translation</strong>: Ribosome recycling begins after the termination of translation, when a stop codon is encountered on the mRNA template. Release factors, such as eukaryotic release factor 1 (eRF1) and eukaryotic release factor 3 (eRF3), facilitate the release of the synthesized polypeptide chain from the ribosome.</p></li>
<li><p><strong>Dissociation of Ribosomes</strong>: Once translation is terminated, the ribosome complex consisting of the large and small subunits needs to be disassembled from the mRNA molecule. This dissociation is facilitated by ribosome recycling factors, such as ribosome recycling factor (RRF) and elongation factor G (EF-G) in bacteria, and Rli1/ABCE1 in eukaryotes. These factors promote the separation of the ribosomal subunits and disengage the ribosome from mRNA.</p></li>
<li><p><strong>Ribosome Splitting</strong>: In bacteria, RRF and EF-G work together to split the ribosome into its large and small subunits. RRF binds to the ribosomal A site, while EF-G hydrolyzes GTP and promotes the splitting of the ribosome complex. In eukaryotes, Rli1/ABCE1 acts as a ribosome-splitting factor by hydrolyzing ATP and disrupting the intersubunit bridges between the large and small ribosomal subunits.</p></li>
<li><p><strong>Recycling of Ribosomal Subunits</strong>: After the ribosome is split into its subunits, both bacterial and eukaryotic ribosomal subunits are recycled for further rounds of translation. The small ribosomal subunit is particularly important, as it recognizes the mRNA start codon and initiates translation. The recycling process ensures the efficient utilization of ribosomal components and enables the cell to maintain high rates of protein synthesis.</p></li>
</ol>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week2.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Ubiquitin Proteasome System</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week4.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Chaperones and Protein Homeostasis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>